
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.563202 - 56 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.563202 - 54 record(s) - 56 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-1e537d0b-f481-448d-9bbd-465b6d0a64f3/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.563202.zip">Download all results</a></li>
        
          <li><a href="6666666.563202.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-1e537d0b-f481-448d-9bbd-465b6d0a64f3.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton lanthipeptide-class-iii r2c1">
              <a href="#r2c1">2.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r4c1">
              <a href="#r4c1">4.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r11c1">
              <a href="#r11c1">11.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton melanin r23c1">
              <a href="#r23c1">23.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lassopeptide r24c1">
              <a href="#r24c1">24.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r27c1">
              <a href="#r27c1">27.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r30c1">
              <a href="#r30c1">30.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,T3PKS,terpene hybrid r38c1">
              <a href="#r38c1">38.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T3PKS r40c1">
              <a href="#r40c1">40.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS-like,T1PKS,prodigiosin hybrid r49c1">
              <a href="#r49c1">49.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r50c1">
              <a href="#r50c1">50.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r53c1">
              <a href="#r53c1">53.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton melanin r56c1">
              <a href="#r56c1">56.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r58c1">
              <a href="#r58c1">58.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r61c1">
              <a href="#r61c1">61.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like hybrid r70c1">
              <a href="#r70c1">70.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,T1PKS,T3PKS,betalactone hybrid r73c1">
              <a href="#r73c1">73.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T3PKS r77c1">
              <a href="#r77c1">77.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton other r81c1">
              <a href="#r81c1">81.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,ladderane hybrid r84c1">
              <a href="#r84c1">84.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r91c1">
              <a href="#r91c1">91.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r93c1">
              <a href="#r93c1">93.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,PKS-like,T1PKS,transAT-PKS hybrid r94c1">
              <a href="#r94c1">94.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton T1PKS,other hybrid r96c1">
              <a href="#r96c1">96.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r99c1">
              <a href="#r99c1">99.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-i r103c1">
              <a href="#r103c1">103.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r104c1">
              <a href="#r104c1">104.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r111c1">
              <a href="#r111c1">111.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r118c1">
              <a href="#r118c1">118.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r123c1">
              <a href="#r123c1">123.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r125c1">
              <a href="#r125c1">125.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r137c1">
              <a href="#r137c1">137.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,other hybrid r146c1">
              <a href="#r146c1">146.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r147c1">
              <a href="#r147c1">147.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T2PKS r151c1">
              <a href="#r151c1">151.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r155c1">
              <a href="#r155c1">155.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,terpene hybrid r157c1">
              <a href="#r157c1">157.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton redox-cofactor r162c1">
              <a href="#r162c1">162.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r162c2">
              <a href="#r162c2">162.2</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,other hybrid r167c1">
              <a href="#r167c1">167.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton T1PKS,terpene hybrid r170c1">
              <a href="#r170c1">170.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r179c1">
              <a href="#r179c1">179.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r197c1">
              <a href="#r197c1">197.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r201c1">
              <a href="#r201c1">201.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton T1PKS,lassopeptide hybrid r203c1">
              <a href="#r203c1">203.1</a>
            </div>
          
            
            
            <div class="regbutton phenazine r203c2">
              <a href="#r203c2">203.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r206c1">
              <a href="#r206c1">206.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r208c1">
              <a href="#r208c1">208.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like hybrid r215c1">
              <a href="#r215c1">215.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,RiPP-like,betalactone hybrid r217c1">
              <a href="#r217c1">217.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r222c1">
              <a href="#r222c1">222.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r226c1">
              <a href="#r226c1">226.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r227c1">
              <a href="#r227c1">227.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like hybrid r228c1">
              <a href="#r228c1">228.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r238c1">
              <a href="#r238c1">238.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r240c1">
              <a href="#r240c1">240.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00002_NODE_54..</strong> (original name was: NODE_54_length_64661_cov_44.8127) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="c00002_NODE_54.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r2c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r2c1">
      <a href="#r2c1">Region&nbsp2.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">8,116</td>
     <td class="digits table-split-left">33,620</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000874/1" target="_blank">blasticidin S</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00004_NODE_46..</strong> (original name was: NODE_46_length_68748_cov_44.2389) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="c00004_NODE_46.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r4c1">
     
     
     <td class="regbutton NRPS-like r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">25,168</td>
     <td class="digits table-split-left">67,975</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001406/1" target="_blank">telomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00011_NODE_49..</strong> (original name was: NODE_49_length_67870_cov_46.6939) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-11" data-filename="c00011_NODE_49.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-11" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r11c1">
     
     
     <td class="regbutton NRPS r11c1">
      <a href="#r11c1">Region&nbsp11.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">53,110</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001036/1" target="_blank">polyoxypeptin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00023_NODE_53..</strong> (original name was: NODE_53_length_65966_cov_65.1003) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-23" data-filename="c00023_NODE_53.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-23" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r23c1">
     
     
     <td class="regbutton melanin r23c1">
      <a href="#r23c1">Region&nbsp23.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">13,088</td>
     <td class="digits table-split-left">23,498</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000700/1" target="_blank">istamycin</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00024_NODE_19..</strong> (original name was: NODE_192_length_9307_cov_44.4) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-24" data-filename="c00024_NODE_19.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-24" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r24c1">
     
     
     <td class="regbutton lassopeptide r24c1">
      <a href="#r24c1">Region&nbsp24.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,307</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000579/1" target="_blank">SSV-2083</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 54%, #ffffff00 54%)">54%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00027_NODE_58..</strong> (original name was: NODE_58_length_62001_cov_45.861) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-27" data-filename="c00027_NODE_58.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-27" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r27c1">
     
     
     <td class="regbutton RiPP-like r27c1">
      <a href="#r27c1">Region&nbsp27.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">21,346</td>
     <td class="digits table-split-left">31,801</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00030_NODE_31..</strong> (original name was: NODE_31_length_85614_cov_46.6038) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-30" data-filename="c00030_NODE_31.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-30" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r30c1">
     
     
     <td class="regbutton terpene r30c1">
      <a href="#r30c1">Region&nbsp30.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">37,948</td>
     <td class="digits table-split-left">64,231</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 63%, #ffffff00 63%)">63%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00038_NODE_13..</strong> (original name was: NODE_13_length_106741_cov_45.7749) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-38" data-filename="c00038_NODE_13.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-38" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r38c1">
     
     
         
     
     <td class="regbutton NRPS,T3PKS,terpene hybrid r38c1">
      <a href="#r38c1">Region&nbsp38.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">56,937</td>
     <td class="digits table-split-left">106,741</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000378/1" target="_blank">kutzneride 2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00040_NODE_47..</strong> (original name was: NODE_47_length_68489_cov_44.956) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-40" data-filename="c00040_NODE_47.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-40" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r40c1">
     
     
     <td class="regbutton T3PKS r40c1">
      <a href="#r40c1">Region&nbsp40.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">5,549</td>
     <td class="digits table-split-left">46,589</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000029/1" target="_blank">BE-14106</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00049_NODE_82..</strong> (original name was: NODE_82_length_41000_cov_44.8257) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-49" data-filename="c00049_NODE_82.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-49" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r49c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS,prodigiosin hybrid r49c1">
      <a href="#r49c1">Region&nbsp49.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#prodigiosin" target="_blank">prodigiosin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">41,000</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000091/1" target="_blank">marineosin A / marineosin B</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 77%, #ffffff00 77%)">77%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00050_NODE_9_..</strong> (original name was: NODE_9_length_121235_cov_47.0373) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-50" data-filename="c00050_NODE_9_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-50" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r50c1">
     
     
     <td class="regbutton terpene r50c1">
      <a href="#r50c1">Region&nbsp50.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">90,069</td>
     <td class="digits table-split-left">117,092</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 84%, #ffffff00 84%)">84%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00053_NODE_5_..</strong> (original name was: NODE_5_length_142893_cov_44.9034) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-53" data-filename="c00053_NODE_5_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-53" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r53c1">
     
     
     <td class="regbutton NRPS r53c1">
      <a href="#r53c1">Region&nbsp53.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">46,732</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00056_NODE_18..</strong> (original name was: NODE_183_length_11559_cov_46.6438) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-56" data-filename="c00056_NODE_18.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-56" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r56c1">
     
     
     <td class="regbutton melanin r56c1">
      <a href="#r56c1">Region&nbsp56.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1,049</td>
     <td class="digits table-split-left">11,453</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000908/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00058_NODE_85..</strong> (original name was: NODE_85_length_39760_cov_48.0069) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-58" data-filename="c00058_NODE_85.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-58" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r58c1">
     
     
     <td class="regbutton NRPS r58c1">
      <a href="#r58c1">Region&nbsp58.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,798</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00061_NODE_20..</strong> (original name was: NODE_20_length_98873_cov_47.2525) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-61" data-filename="c00061_NODE_20.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-61" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r61c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r61c1">
      <a href="#r61c1">Region&nbsp61.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">21,176</td>
     <td class="digits table-split-left">95,463</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000341/1" target="_blank">enduracidin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00070_NODE_10..</strong> (original name was: NODE_103_length_33038_cov_50.5851) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-70" data-filename="c00070_NODE_10.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-70" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r70c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r70c1">
      <a href="#r70c1">Region&nbsp70.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">33,038</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000341/1" target="_blank">enduracidin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00073_NODE_16..</strong> (original name was: NODE_16_length_104909_cov_47.5615) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-73" data-filename="c00073_NODE_16.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-73" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r73c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,T1PKS,T3PKS,betalactone hybrid r73c1">
      <a href="#r73c1">Region&nbsp73.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">82,739</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001002/1" target="_blank">jawsamycin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00077_NODE_73..</strong> (original name was: NODE_73_length_46946_cov_51.4199) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-77" data-filename="c00077_NODE_73.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-77" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r77c1">
     
     
     <td class="regbutton T3PKS r77c1">
      <a href="#r77c1">Region&nbsp77.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">23,132</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00081_NODE_20..</strong> (original name was: NODE_208_length_6409_cov_43.6684) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-81" data-filename="c00081_NODE_20.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-81" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r81c1">
     
     
     <td class="regbutton other r81c1">
      <a href="#r81c1">Region&nbsp81.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,409</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00084_NODE_8_..</strong> (original name was: NODE_8_length_122381_cov_45.8766) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-84" data-filename="c00084_NODE_8_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-84" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r84c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,ladderane hybrid r84c1">
      <a href="#r84c1">Region&nbsp84.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>
     </td>
     <td class="digits">2</td>
     <td class="digits table-split-left">65,431</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000354/1" target="_blank">friulimicin A / friulimicin B / friulimicin C / friulimicin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 21%, #ffffff00 21%)">21%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00091_NODE_14..</strong> (original name was: NODE_14_length_105475_cov_47.5929) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-91" data-filename="c00091_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-91" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r91c1">
     
     
     <td class="regbutton siderophore r91c1">
      <a href="#r91c1">Region&nbsp91.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">52,735</td>
     <td class="digits table-split-left">67,819</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00093_NODE_4_..</strong> (original name was: NODE_4_length_160966_cov_47.7722) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-93" data-filename="c00093_NODE_4_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-93" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r93c1">
     
     
     <td class="regbutton terpene r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">14,840</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000660/1" target="_blank">albaflavenone</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00094_NODE_22..</strong> (original name was: NODE_22_length_97811_cov_49.7105) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-94" data-filename="c00094_NODE_22.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-94" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r94c1">
     
     
         
     
     <td class="regbutton NRPS,PKS-like,T1PKS,transAT-PKS hybrid r94c1">
      <a href="#r94c1">Region&nbsp94.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#transat-pks" target="_blank">transAT-PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">67,707</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000296/1" target="_blank">actinomycin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00096_NODE_88..</strong> (original name was: NODE_88_length_39339_cov_88.6908) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-96" data-filename="c00096_NODE_88.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-96" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r96c1">
     
     
         
     
     <td class="regbutton T1PKS,other hybrid r96c1">
      <a href="#r96c1">Region&nbsp96.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">39,339</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001519/1" target="_blank">aurantimycin A</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00099_NODE_13..</strong> (original name was: NODE_137_length_24329_cov_48.103) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-99" data-filename="c00099_NODE_13.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-99" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r99c1">
     
     
     <td class="regbutton NRPS r99c1">
      <a href="#r99c1">Region&nbsp99.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">24,329</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001567/1" target="_blank">cysteoamide</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00103_NODE_24..</strong> (original name was: NODE_24_length_95845_cov_43.5485) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-103" data-filename="c00103_NODE_24.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-103" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r103c1">
     
     
     <td class="regbutton lanthipeptide-class-i r103c1">
      <a href="#r103c1">Region&nbsp103.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">3,211</td>
     <td class="digits table-split-left">27,757</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00104_NODE_37..</strong> (original name was: NODE_37_length_77841_cov_47.0912) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-104" data-filename="c00104_NODE_37.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-104" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r104c1">
     
     
     <td class="regbutton T1PKS r104c1">
      <a href="#r104c1">Region&nbsp104.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">77,841</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001700/1" target="_blank">niphimycins C-E</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 35%, #ffffff00 35%)">35%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00111_NODE_23..</strong> (original name was: NODE_23_length_97606_cov_46.1806) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-111" data-filename="c00111_NODE_23.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-111" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r111c1">
     
     
     <td class="regbutton RiPP-like r111c1">
      <a href="#r111c1">Region&nbsp111.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">7,335</td>
     <td class="digits table-split-left">18,918</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00118_NODE_11..</strong> (original name was: NODE_111_length_31017_cov_49.4195) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-118" data-filename="c00118_NODE_11.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-118" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r118c1">
     
     
     <td class="regbutton T1PKS r118c1">
      <a href="#r118c1">Region&nbsp118.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">31,017</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000084/1" target="_blank">laidlomycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00123_NODE_15..</strong> (original name was: NODE_15_length_105449_cov_47.4172) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-123" data-filename="c00123_NODE_15.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-123" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r123c1">
     
     
     <td class="regbutton T1PKS r123c1">
      <a href="#r123c1">Region&nbsp123.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">52,666</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000078/1" target="_blank">incednine</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00125_NODE_15..</strong> (original name was: NODE_150_length_22215_cov_46.7742) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-125" data-filename="c00125_NODE_15.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-125" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r125c1">
     
     
     <td class="regbutton terpene r125c1">
      <a href="#r125c1">Region&nbsp125.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">5,094</td>
     <td class="digits table-split-left">22,215</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00137_NODE_61..</strong> (original name was: NODE_61_length_59604_cov_47.0723) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-137" data-filename="c00137_NODE_61.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-137" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r137c1">
     
     
     <td class="regbutton NRPS r137c1">
      <a href="#r137c1">Region&nbsp137.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">23,305</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000078/1" target="_blank">incednine</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00146_NODE_40..</strong> (original name was: NODE_40_length_74423_cov_49.3125) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-146" data-filename="c00146_NODE_40.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-146" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r146c1">
     
     
         
     
     <td class="regbutton NRPS,other hybrid r146c1">
      <a href="#r146c1">Region&nbsp146.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">11,957</td>
     <td class="digits table-split-left">74,423</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001036/1" target="_blank">polyoxypeptin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 37%, #ffffff00 37%)">37%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00147_NODE_65..</strong> (original name was: NODE_65_length_54905_cov_43.6762) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-147" data-filename="c00147_NODE_65.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-147" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r147c1">
     
     
     <td class="regbutton NRPS r147c1">
      <a href="#r147c1">Region&nbsp147.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">45,354</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001537/1" target="_blank">butyrolactol A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00151_NODE_83..</strong> (original name was: NODE_83_length_40488_cov_50.0719) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-151" data-filename="c00151_NODE_83.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-151" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r151c1">
     
     
     <td class="regbutton T2PKS r151c1">
      <a href="#r151c1">Region&nbsp151.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">40,488</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002000/1" target="_blank">setomimycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 86%, #ffffff00 86%)">86%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00155_NODE_7_..</strong> (original name was: NODE_7_length_123630_cov_45.266) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-155" data-filename="c00155_NODE_7_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-155" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r155c1">
     
     
     <td class="regbutton NRPS r155c1">
      <a href="#r155c1">Region&nbsp155.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">49,163</td>
     <td class="digits table-split-left">99,464</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000392/1" target="_blank">mirubactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00157_NODE_6_..</strong> (original name was: NODE_6_length_138135_cov_45.1661) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-157" data-filename="c00157_NODE_6_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-157" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r157c1">
     
     
         
     
     <td class="regbutton NRPS,terpene hybrid r157c1">
      <a href="#r157c1">Region&nbsp157.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">46,435</td>
     <td class="digits table-split-left">90,829</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001106/1" target="_blank">oxalomycin B</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00162_NODE_35..</strong> (original name was: NODE_35_length_78516_cov_47.5625) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-162" data-filename="c00162_NODE_35.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-162" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r162c1">
     
     
     <td class="regbutton redox-cofactor r162c1">
      <a href="#r162c1">Region&nbsp162.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">13,362</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r162c2">
     
     
     <td class="regbutton terpene r162c2">
      <a href="#r162c2">Region&nbsp162.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">53,927</td>
     <td class="digits table-split-left">76,032</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00167_NODE_50..</strong> (original name was: NODE_50_length_67420_cov_47.4694) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-167" data-filename="c00167_NODE_50.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-167" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r167c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,other hybrid r167c1">
      <a href="#r167c1">Region&nbsp167.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">53,641</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 52%, #ffffff00 52%)">52%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00170_NODE_18..</strong> (original name was: NODE_18_length_101633_cov_46.1054) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-170" data-filename="c00170_NODE_18.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-170" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r170c1">
     
     
         
     
     <td class="regbutton T1PKS,terpene hybrid r170c1">
      <a href="#r170c1">Region&nbsp170.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">27,073</td>
     <td class="digits table-split-left">101,633</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001435/1" target="_blank">ikarugamycin</a></td>
      <td>NRP + Polyketide:Iterative type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00179_NODE_91..</strong> (original name was: NODE_91_length_38047_cov_53.9196) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-179" data-filename="c00179_NODE_91.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-179" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r179c1">
     
     
     <td class="regbutton NRPS r179c1">
      <a href="#r179c1">Region&nbsp179.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">38,047</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001582/1" target="_blank">ecumicin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00197_NODE_10..</strong> (original name was: NODE_10_length_118721_cov_45.3429) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-197" data-filename="c00197_NODE_10.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-197" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r197c1">
     
     
     <td class="regbutton terpene r197c1">
      <a href="#r197c1">Region&nbsp197.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">5,076</td>
     <td class="digits table-split-left">26,368</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001580/1" target="_blank">ebelactone</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00201_NODE_17..</strong> (original name was: NODE_171_length_15040_cov_48.4686) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-201" data-filename="c00201_NODE_17.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-201" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r201c1">
     
     
     <td class="regbutton NRPS r201c1">
      <a href="#r201c1">Region&nbsp201.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">15,040</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00203_NODE_1_..</strong> (original name was: NODE_1_length_202022_cov_51.4099) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-203" data-filename="c00203_NODE_1_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-203" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r203c1">
     
     
         
     
     <td class="regbutton T1PKS,lassopeptide hybrid r203c1">
      <a href="#r203c1">Region&nbsp203.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>
     </td>
     <td class="digits">14,536</td>
     <td class="digits table-split-left">62,173</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001550/1" target="_blank">citrulassin D</a></td>
      <td>RiPP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r203c2">
     
     
     <td class="regbutton phenazine r203c2">
      <a href="#r203c2">Region&nbsp203.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phenazine" target="_blank">phenazine</a>
     </td>
     <td class="digits">138,806</td>
     <td class="digits table-split-left">159,324</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00206_NODE_11..</strong> (original name was: NODE_114_length_30227_cov_48.4166) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-206" data-filename="c00206_NODE_11.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-206" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r206c1">
     
     
     <td class="regbutton NRPS r206c1">
      <a href="#r206c1">Region&nbsp206.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">30,227</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00208_NODE_26..</strong> (original name was: NODE_26_length_93594_cov_45.5888) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-208" data-filename="c00208_NODE_26.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-208" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r208c1">
     
     
     <td class="regbutton NRPS r208c1">
      <a href="#r208c1">Region&nbsp208.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">33,174</td>
     <td class="digits table-split-left">93,594</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00215_NODE_16..</strong> (original name was: NODE_161_length_17837_cov_48.5949) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-215" data-filename="c00215_NODE_16.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-215" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r215c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r215c1">
      <a href="#r215c1">Region&nbsp215.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,837</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000461/1" target="_blank">WAP-8294A2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00217_NODE_48..</strong> (original name was: NODE_48_length_68335_cov_46.6838) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-217" data-filename="c00217_NODE_48.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-217" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r217c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,RiPP-like,betalactone hybrid r217c1">
      <a href="#r217c1">Region&nbsp217.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">2</td>
     <td class="digits table-split-left">62,519</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001818/1" target="_blank">vazabitide A</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00222_NODE_22..</strong> (original name was: NODE_222_length_4228_cov_45.8783) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-222" data-filename="c00222_NODE_22.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-222" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r222c1">
     
     
     <td class="regbutton RiPP-like r222c1">
      <a href="#r222c1">Region&nbsp222.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,228</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00226_NODE_36..</strong> (original name was: NODE_36_length_78457_cov_55.9278) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-226" data-filename="c00226_NODE_36.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-226" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r226c1">
     
     
     <td class="regbutton terpene r226c1">
      <a href="#r226c1">Region&nbsp226.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">52,084</td>
     <td class="digits table-split-left">73,157</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001058/1" target="_blank">zorbamycin</a></td>
      <td>NRP:Glycopeptide + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00227_NODE_92..</strong> (original name was: NODE_92_length_37859_cov_42.9855) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-227" data-filename="c00227_NODE_92.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-227" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r227c1">
     
     
     <td class="regbutton ectoine r227c1">
      <a href="#r227c1">Region&nbsp227.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">9,879</td>
     <td class="digits table-split-left">20,283</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00228_NODE_19..</strong> (original name was: NODE_195_length_8567_cov_51.3589) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-228" data-filename="c00228_NODE_19.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-228" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r228c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r228c1">
      <a href="#r228c1">Region&nbsp228.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,567</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00238_NODE_56..</strong> (original name was: NODE_56_length_63484_cov_46.3375) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-238" data-filename="c00238_NODE_56.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-238" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r238c1">
     
     
     <td class="regbutton ectoine r238c1">
      <a href="#r238c1">Region&nbsp238.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">35,321</td>
     <td class="digits table-split-left">45,695</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001073/1" target="_blank">kosinostatin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00240_NODE_10..</strong> (original name was: NODE_106_length_32675_cov_47.7034) (Streptomyces sp. CC224B)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-240" data-filename="c00240_NODE_10.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-240" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r240c1">
     
     
     <td class="regbutton T1PKS r240c1">
      <a href="#r240c1">Region&nbsp240.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1,040</td>
     <td class="digits table-split-left">32,675</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000052/1" target="_blank">ECO-02301</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r2c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r2c1">
      <a href="#r2c1">Region&nbsp2.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">8,116</td>
     <td class="digits table-split-left">33,620</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000874/1" target="_blank">blasticidin S</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r4c1">
     
     
     <td class="regbutton NRPS-like r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">25,168</td>
     <td class="digits table-split-left">67,975</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001406/1" target="_blank">telomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r11c1">
     
     
     <td class="regbutton NRPS r11c1">
      <a href="#r11c1">Region&nbsp11.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">53,110</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001036/1" target="_blank">polyoxypeptin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r23c1">
     
     
     <td class="regbutton melanin r23c1">
      <a href="#r23c1">Region&nbsp23.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">13,088</td>
     <td class="digits table-split-left">23,498</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000700/1" target="_blank">istamycin</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r24c1">
     
     
     <td class="regbutton lassopeptide r24c1">
      <a href="#r24c1">Region&nbsp24.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,307</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000579/1" target="_blank">SSV-2083</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 54%, #ffffff00 54%)">54%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r27c1">
     
     
     <td class="regbutton RiPP-like r27c1">
      <a href="#r27c1">Region&nbsp27.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">21,346</td>
     <td class="digits table-split-left">31,801</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r30c1">
     
     
     <td class="regbutton terpene r30c1">
      <a href="#r30c1">Region&nbsp30.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">37,948</td>
     <td class="digits table-split-left">64,231</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 63%, #ffffff00 63%)">63%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r38c1">
     
     
         
     
     <td class="regbutton NRPS,T3PKS,terpene hybrid r38c1">
      <a href="#r38c1">Region&nbsp38.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">56,937</td>
     <td class="digits table-split-left">106,741</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000378/1" target="_blank">kutzneride 2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r40c1">
     
     
     <td class="regbutton T3PKS r40c1">
      <a href="#r40c1">Region&nbsp40.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">5,549</td>
     <td class="digits table-split-left">46,589</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000029/1" target="_blank">BE-14106</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r49c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS,prodigiosin hybrid r49c1">
      <a href="#r49c1">Region&nbsp49.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#prodigiosin" target="_blank">prodigiosin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">41,000</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000091/1" target="_blank">marineosin A / marineosin B</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 77%, #ffffff00 77%)">77%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r50c1">
     
     
     <td class="regbutton terpene r50c1">
      <a href="#r50c1">Region&nbsp50.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">90,069</td>
     <td class="digits table-split-left">117,092</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 84%, #ffffff00 84%)">84%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r53c1">
     
     
     <td class="regbutton NRPS r53c1">
      <a href="#r53c1">Region&nbsp53.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">46,732</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r56c1">
     
     
     <td class="regbutton melanin r56c1">
      <a href="#r56c1">Region&nbsp56.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1,049</td>
     <td class="digits table-split-left">11,453</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000908/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r58c1">
     
     
     <td class="regbutton NRPS r58c1">
      <a href="#r58c1">Region&nbsp58.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,798</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r61c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r61c1">
      <a href="#r61c1">Region&nbsp61.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">21,176</td>
     <td class="digits table-split-left">95,463</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000341/1" target="_blank">enduracidin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r70c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r70c1">
      <a href="#r70c1">Region&nbsp70.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">33,038</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000341/1" target="_blank">enduracidin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r73c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,T1PKS,T3PKS,betalactone hybrid r73c1">
      <a href="#r73c1">Region&nbsp73.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">82,739</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001002/1" target="_blank">jawsamycin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r77c1">
     
     
     <td class="regbutton T3PKS r77c1">
      <a href="#r77c1">Region&nbsp77.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">23,132</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r81c1">
     
     
     <td class="regbutton other r81c1">
      <a href="#r81c1">Region&nbsp81.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,409</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r84c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,ladderane hybrid r84c1">
      <a href="#r84c1">Region&nbsp84.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>
     </td>
     <td class="digits">2</td>
     <td class="digits table-split-left">65,431</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000354/1" target="_blank">friulimicin A / friulimicin B / friulimicin C / friulimicin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 21%, #ffffff00 21%)">21%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r91c1">
     
     
     <td class="regbutton siderophore r91c1">
      <a href="#r91c1">Region&nbsp91.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">52,735</td>
     <td class="digits table-split-left">67,819</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r93c1">
     
     
     <td class="regbutton terpene r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">14,840</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000660/1" target="_blank">albaflavenone</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r94c1">
     
     
         
     
     <td class="regbutton NRPS,PKS-like,T1PKS,transAT-PKS hybrid r94c1">
      <a href="#r94c1">Region&nbsp94.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#transat-pks" target="_blank">transAT-PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">67,707</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000296/1" target="_blank">actinomycin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r96c1">
     
     
         
     
     <td class="regbutton T1PKS,other hybrid r96c1">
      <a href="#r96c1">Region&nbsp96.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">39,339</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001519/1" target="_blank">aurantimycin A</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r99c1">
     
     
     <td class="regbutton NRPS r99c1">
      <a href="#r99c1">Region&nbsp99.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">24,329</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001567/1" target="_blank">cysteoamide</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r103c1">
     
     
     <td class="regbutton lanthipeptide-class-i r103c1">
      <a href="#r103c1">Region&nbsp103.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">3,211</td>
     <td class="digits table-split-left">27,757</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r104c1">
     
     
     <td class="regbutton T1PKS r104c1">
      <a href="#r104c1">Region&nbsp104.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">77,841</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001700/1" target="_blank">niphimycins C-E</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 35%, #ffffff00 35%)">35%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r111c1">
     
     
     <td class="regbutton RiPP-like r111c1">
      <a href="#r111c1">Region&nbsp111.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">7,335</td>
     <td class="digits table-split-left">18,918</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r118c1">
     
     
     <td class="regbutton T1PKS r118c1">
      <a href="#r118c1">Region&nbsp118.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">31,017</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000084/1" target="_blank">laidlomycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r123c1">
     
     
     <td class="regbutton T1PKS r123c1">
      <a href="#r123c1">Region&nbsp123.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">52,666</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000078/1" target="_blank">incednine</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r125c1">
     
     
     <td class="regbutton terpene r125c1">
      <a href="#r125c1">Region&nbsp125.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">5,094</td>
     <td class="digits table-split-left">22,215</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r137c1">
     
     
     <td class="regbutton NRPS r137c1">
      <a href="#r137c1">Region&nbsp137.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">23,305</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000078/1" target="_blank">incednine</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r146c1">
     
     
         
     
     <td class="regbutton NRPS,other hybrid r146c1">
      <a href="#r146c1">Region&nbsp146.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">11,957</td>
     <td class="digits table-split-left">74,423</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001036/1" target="_blank">polyoxypeptin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 37%, #ffffff00 37%)">37%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r147c1">
     
     
     <td class="regbutton NRPS r147c1">
      <a href="#r147c1">Region&nbsp147.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">45,354</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001537/1" target="_blank">butyrolactol A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r151c1">
     
     
     <td class="regbutton T2PKS r151c1">
      <a href="#r151c1">Region&nbsp151.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">40,488</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002000/1" target="_blank">setomimycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 86%, #ffffff00 86%)">86%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r155c1">
     
     
     <td class="regbutton NRPS r155c1">
      <a href="#r155c1">Region&nbsp155.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">49,163</td>
     <td class="digits table-split-left">99,464</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000392/1" target="_blank">mirubactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r157c1">
     
     
         
     
     <td class="regbutton NRPS,terpene hybrid r157c1">
      <a href="#r157c1">Region&nbsp157.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">46,435</td>
     <td class="digits table-split-left">90,829</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001106/1" target="_blank">oxalomycin B</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r162c1">
     
     
     <td class="regbutton redox-cofactor r162c1">
      <a href="#r162c1">Region&nbsp162.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">13,362</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r162c2">
     
     
     <td class="regbutton terpene r162c2">
      <a href="#r162c2">Region&nbsp162.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">53,927</td>
     <td class="digits table-split-left">76,032</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r167c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,other hybrid r167c1">
      <a href="#r167c1">Region&nbsp167.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">53,641</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 52%, #ffffff00 52%)">52%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r170c1">
     
     
         
     
     <td class="regbutton T1PKS,terpene hybrid r170c1">
      <a href="#r170c1">Region&nbsp170.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">27,073</td>
     <td class="digits table-split-left">101,633</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001435/1" target="_blank">ikarugamycin</a></td>
      <td>NRP + Polyketide:Iterative type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r179c1">
     
     
     <td class="regbutton NRPS r179c1">
      <a href="#r179c1">Region&nbsp179.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">38,047</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001582/1" target="_blank">ecumicin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r197c1">
     
     
     <td class="regbutton terpene r197c1">
      <a href="#r197c1">Region&nbsp197.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">5,076</td>
     <td class="digits table-split-left">26,368</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001580/1" target="_blank">ebelactone</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r201c1">
     
     
     <td class="regbutton NRPS r201c1">
      <a href="#r201c1">Region&nbsp201.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">15,040</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r203c1">
     
     
         
     
     <td class="regbutton T1PKS,lassopeptide hybrid r203c1">
      <a href="#r203c1">Region&nbsp203.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>
     </td>
     <td class="digits">14,536</td>
     <td class="digits table-split-left">62,173</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001550/1" target="_blank">citrulassin D</a></td>
      <td>RiPP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r203c2">
     
     
     <td class="regbutton phenazine r203c2">
      <a href="#r203c2">Region&nbsp203.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phenazine" target="_blank">phenazine</a>
     </td>
     <td class="digits">138,806</td>
     <td class="digits table-split-left">159,324</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r206c1">
     
     
     <td class="regbutton NRPS r206c1">
      <a href="#r206c1">Region&nbsp206.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">30,227</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r208c1">
     
     
     <td class="regbutton NRPS r208c1">
      <a href="#r208c1">Region&nbsp208.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">33,174</td>
     <td class="digits table-split-left">93,594</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r215c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r215c1">
      <a href="#r215c1">Region&nbsp215.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,837</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000461/1" target="_blank">WAP-8294A2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r217c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,RiPP-like,betalactone hybrid r217c1">
      <a href="#r217c1">Region&nbsp217.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">2</td>
     <td class="digits table-split-left">62,519</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001818/1" target="_blank">vazabitide A</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r222c1">
     
     
     <td class="regbutton RiPP-like r222c1">
      <a href="#r222c1">Region&nbsp222.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,228</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r226c1">
     
     
     <td class="regbutton terpene r226c1">
      <a href="#r226c1">Region&nbsp226.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">52,084</td>
     <td class="digits table-split-left">73,157</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001058/1" target="_blank">zorbamycin</a></td>
      <td>NRP:Glycopeptide + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r227c1">
     
     
     <td class="regbutton ectoine r227c1">
      <a href="#r227c1">Region&nbsp227.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">9,879</td>
     <td class="digits table-split-left">20,283</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r228c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r228c1">
      <a href="#r228c1">Region&nbsp228.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,567</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001117/1" target="_blank">himastatin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r238c1">
     
     
     <td class="regbutton ectoine r238c1">
      <a href="#r238c1">Region&nbsp238.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">35,321</td>
     <td class="digits table-split-left">45,695</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001073/1" target="_blank">kosinostatin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r240c1">
     
     
     <td class="regbutton T1PKS r240c1">
      <a href="#r240c1">Region&nbsp240.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1,040</td>
     <td class="digits table-split-left">32,675</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000052/1" target="_blank">ECO-02301</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00001_NODE_87..</strong> (original name was: NODE_87_length_39506_cov_47.8394) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00003_NODE_11..</strong> (original name was: NODE_113_length_30384_cov_46.4446) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00005_NODE_20..</strong> (original name was: NODE_200_length_7515_cov_45.1385) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00006_NODE_17..</strong> (original name was: NODE_176_length_13526_cov_47.4142) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00007_NODE_97..</strong> (original name was: NODE_97_length_35660_cov_47.5072) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00008_NODE_79..</strong> (original name was: NODE_79_length_43683_cov_46.6404) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00009_NODE_13..</strong> (original name was: NODE_130_length_27053_cov_46.2188) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00010_NODE_24..</strong> (original name was: NODE_245_length_1022_cov_96.6656) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00012_NODE_22..</strong> (original name was: NODE_221_length_4300_cov_49.5413) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00013_NODE_3_..</strong> (original name was: NODE_3_length_165632_cov_61.8433) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00014_NODE_10..</strong> (original name was: NODE_105_length_32698_cov_43.961) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00015_NODE_17..</strong> (original name was: NODE_173_length_14380_cov_45.6654) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00016_NODE_17..</strong> (original name was: NODE_177_length_13412_cov_51.0125) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00017_NODE_19..</strong> (original name was: NODE_199_length_7701_cov_43.6386) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00018_NODE_14..</strong> (original name was: NODE_144_length_23063_cov_41.6181) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00019_NODE_14..</strong> (original name was: NODE_140_length_23354_cov_43.8025) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00020_NODE_28..</strong> (original name was: NODE_28_length_91058_cov_46.2292) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00021_NODE_22..</strong> (original name was: NODE_225_length_3893_cov_47.0147) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00022_NODE_81..</strong> (original name was: NODE_81_length_41899_cov_48.31) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00025_NODE_19..</strong> (original name was: NODE_196_length_8476_cov_45.8661) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00026_NODE_17..</strong> (original name was: NODE_174_length_14359_cov_47.2491) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00028_NODE_13..</strong> (original name was: NODE_138_length_24210_cov_45.5373) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00029_NODE_13..</strong> (original name was: NODE_133_length_25732_cov_51.4447) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00031_NODE_13..</strong> (original name was: NODE_135_length_25052_cov_46.2084) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00032_NODE_22..</strong> (original name was: NODE_220_length_4301_cov_45.1806) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00033_NODE_15..</strong> (original name was: NODE_151_length_21455_cov_46.8247) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00034_NODE_72..</strong> (original name was: NODE_72_length_48176_cov_46.609) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00035_NODE_2_..</strong> (original name was: NODE_2_length_173093_cov_52.5256) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00036_NODE_18..</strong> (original name was: NODE_184_length_10858_cov_46.6388) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00037_NODE_25..</strong> (original name was: NODE_25_length_94909_cov_47.4905) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00039_NODE_17..</strong> (original name was: NODE_178_length_13037_cov_47.6751) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00041_NODE_62..</strong> (original name was: NODE_62_length_59201_cov_42.1046) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00042_NODE_17..</strong> (original name was: NODE_17_length_102440_cov_46.3614) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00043_NODE_19..</strong> (original name was: NODE_197_length_8373_cov_45.1167) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00044_NODE_21..</strong> (original name was: NODE_214_length_5009_cov_45.5845) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00045_NODE_12..</strong> (original name was: NODE_126_length_27486_cov_43.8823) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00046_NODE_44..</strong> (original name was: NODE_44_length_69715_cov_44.9271) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00047_NODE_27..</strong> (original name was: NODE_27_length_92452_cov_44.316) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00048_NODE_10..</strong> (original name was: NODE_104_length_32836_cov_46.353) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00051_NODE_10..</strong> (original name was: NODE_108_length_31256_cov_48.8216) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00052_NODE_24..</strong> (original name was: NODE_240_length_1993_cov_47.6331) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00054_NODE_59..</strong> (original name was: NODE_59_length_61520_cov_46.7548) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00055_NODE_11..</strong> (original name was: NODE_116_length_29692_cov_45.5318) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00057_NODE_22..</strong> (original name was: NODE_224_length_4102_cov_48.2007) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00059_NODE_20..</strong> (original name was: NODE_209_length_5992_cov_47.4064) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00060_NODE_22..</strong> (original name was: NODE_223_length_4115_cov_41.7464) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00062_NODE_43..</strong> (original name was: NODE_43_length_72285_cov_45.8121) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00063_NODE_20..</strong> (original name was: NODE_204_length_6765_cov_42.6048) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00064_NODE_11..</strong> (original name was: NODE_115_length_30042_cov_49.1326) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00065_NODE_22..</strong> (original name was: NODE_226_length_3789_cov_43.3332) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00066_NODE_99..</strong> (original name was: NODE_99_length_34667_cov_45.2104) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00067_NODE_12..</strong> (original name was: NODE_12_length_107197_cov_49.4954) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00068_NODE_24..</strong> (original name was: NODE_241_length_1548_cov_80.9844) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00069_NODE_23..</strong> (original name was: NODE_236_length_2657_cov_54.2702) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00071_NODE_77..</strong> (original name was: NODE_77_length_44500_cov_44.6937) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00072_NODE_16..</strong> (original name was: NODE_160_length_18578_cov_65.1439) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00074_NODE_24..</strong> (original name was: NODE_242_length_1426_cov_95.7324) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00075_NODE_21..</strong> (original name was: NODE_213_length_5234_cov_256.919) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00076_NODE_13..</strong> (original name was: NODE_136_length_24548_cov_47.0257) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00078_NODE_86..</strong> (original name was: NODE_86_length_39532_cov_46.6254) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00079_NODE_13..</strong> (original name was: NODE_139_length_23560_cov_47.9457) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00080_NODE_19..</strong> (original name was: NODE_193_length_9162_cov_49.262) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00082_NODE_95..</strong> (original name was: NODE_95_length_36075_cov_45.8937) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00083_NODE_24..</strong> (original name was: NODE_243_length_1286_cov_83.2614) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00085_NODE_14..</strong> (original name was: NODE_141_length_23250_cov_45.2452) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00086_NODE_64..</strong> (original name was: NODE_64_length_56294_cov_44.4094) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00087_NODE_12..</strong> (original name was: NODE_123_length_28047_cov_47.9684) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00088_NODE_11..</strong> (original name was: NODE_118_length_29438_cov_45.4109) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00089_NODE_30..</strong> (original name was: NODE_30_length_85994_cov_45.309) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00090_NODE_21..</strong> (original name was: NODE_218_length_4658_cov_39.691) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00092_NODE_19..</strong> (original name was: NODE_191_length_9820_cov_43.0003) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00095_NODE_23..</strong> (original name was: NODE_238_length_2140_cov_92.8337) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00097_NODE_89..</strong> (original name was: NODE_89_length_39303_cov_45.2868) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00098_NODE_12..</strong> (original name was: NODE_120_length_29237_cov_53.9203) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00100_NODE_15..</strong> (original name was: NODE_153_length_20516_cov_43.65) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00101_NODE_15..</strong> (original name was: NODE_159_length_18661_cov_45.5554) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00102_NODE_23..</strong> (original name was: NODE_234_length_2758_cov_40.0026) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00105_NODE_11..</strong> (original name was: NODE_119_length_29354_cov_47.5207) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00106_NODE_21..</strong> (original name was: NODE_21_length_98037_cov_48.4233) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00107_NODE_14..</strong> (original name was: NODE_147_length_22470_cov_44.9496) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00108_NODE_32..</strong> (original name was: NODE_32_length_83422_cov_46.2705) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00109_NODE_23..</strong> (original name was: NODE_232_length_2859_cov_65.6657) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00110_NODE_15..</strong> (original name was: NODE_152_length_20796_cov_48.1791) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00112_NODE_22..</strong> (original name was: NODE_229_length_3252_cov_39.8976) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00113_NODE_16..</strong> (original name was: NODE_163_length_17368_cov_45.2225) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00114_NODE_14..</strong> (original name was: NODE_149_length_22354_cov_47.9478) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00115_NODE_21..</strong> (original name was: NODE_216_length_4915_cov_44.4384) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00116_NODE_41..</strong> (original name was: NODE_41_length_73983_cov_49.3083) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00117_NODE_63..</strong> (original name was: NODE_63_length_56310_cov_51.4794) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00119_NODE_12..</strong> (original name was: NODE_122_length_28975_cov_45.6072) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00120_NODE_11..</strong> (original name was: NODE_117_length_29588_cov_63.01) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00121_NODE_19..</strong> (original name was: NODE_190_length_10141_cov_46.3763) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00122_NODE_12..</strong> (original name was: NODE_128_length_27324_cov_45.1649) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00124_NODE_18..</strong> (original name was: NODE_187_length_10394_cov_46.758) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00126_NODE_21..</strong> (original name was: NODE_210_length_5839_cov_48.8804) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00127_NODE_20..</strong> (original name was: NODE_207_length_6481_cov_41.443) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00128_NODE_70..</strong> (original name was: NODE_70_length_50574_cov_45.1286) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00129_NODE_10..</strong> (original name was: NODE_109_length_31159_cov_47.8578) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00130_NODE_11..</strong> (original name was: NODE_112_length_30973_cov_45.7956) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00131_NODE_17..</strong> (original name was: NODE_179_length_12943_cov_51.674) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00132_NODE_15..</strong> (original name was: NODE_157_length_19857_cov_48.7645) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00133_NODE_12..</strong> (original name was: NODE_125_length_27991_cov_47.3302) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00134_NODE_24..</strong> (original name was: NODE_247_length_1008_cov_50.246) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00135_NODE_60..</strong> (original name was: NODE_60_length_60707_cov_74.2763) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00136_NODE_71..</strong> (original name was: NODE_71_length_48176_cov_47.3091) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00138_NODE_45..</strong> (original name was: NODE_45_length_69440_cov_47.2279) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00139_NODE_10..</strong> (original name was: NODE_101_length_34163_cov_46.8448) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00140_NODE_15..</strong> (original name was: NODE_155_length_20324_cov_48.2431) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00141_NODE_21..</strong> (original name was: NODE_211_length_5809_cov_42.2722) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00142_NODE_34..</strong> (original name was: NODE_34_length_80345_cov_44.7291) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00143_NODE_18..</strong> (original name was: NODE_189_length_10186_cov_50.719) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00144_NODE_80..</strong> (original name was: NODE_80_length_42606_cov_46.3713) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00145_NODE_94..</strong> (original name was: NODE_94_length_36224_cov_48.9816) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00148_NODE_38..</strong> (original name was: NODE_38_length_76423_cov_47.0041) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00149_NODE_12..</strong> (original name was: NODE_129_length_27214_cov_49.3326) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00150_NODE_14..</strong> (original name was: NODE_143_length_23108_cov_46.2126) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00152_NODE_16..</strong> (original name was: NODE_162_length_17423_cov_45.7549) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00153_NODE_17..</strong> (original name was: NODE_170_length_15407_cov_44.8595) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00154_NODE_57..</strong> (original name was: NODE_57_length_62875_cov_46.7687) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00156_NODE_23..</strong> (original name was: NODE_230_length_2941_cov_39.1536) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00158_NODE_14..</strong> (original name was: NODE_142_length_23110_cov_47.1483) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00159_NODE_15..</strong> (original name was: NODE_154_length_20473_cov_51.2487) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00160_NODE_16..</strong> (original name was: NODE_169_length_15811_cov_48.5055) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00161_NODE_18..</strong> (original name was: NODE_186_length_10445_cov_49.7363) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00163_NODE_21..</strong> (original name was: NODE_217_length_4667_cov_37.3821) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00164_NODE_96..</strong> (original name was: NODE_96_length_36051_cov_50.5617) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00165_NODE_68..</strong> (original name was: NODE_68_length_53676_cov_45.2419) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00166_NODE_90..</strong> (original name was: NODE_90_length_38596_cov_44.905) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00168_NODE_23..</strong> (original name was: NODE_235_length_2722_cov_37.5705) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00169_NODE_18..</strong> (original name was: NODE_180_length_12561_cov_47.6922) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00171_NODE_66..</strong> (original name was: NODE_66_length_54666_cov_45.8545) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00172_NODE_42..</strong> (original name was: NODE_42_length_72721_cov_47.2088) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00173_NODE_39..</strong> (original name was: NODE_39_length_75716_cov_49.1118) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00174_NODE_55..</strong> (original name was: NODE_55_length_63686_cov_46.2597) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00175_NODE_75..</strong> (original name was: NODE_75_length_45423_cov_45.1372) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00176_NODE_10..</strong> (original name was: NODE_100_length_34567_cov_80.6873) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00177_NODE_13..</strong> (original name was: NODE_131_length_27011_cov_44.643) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00178_NODE_14..</strong> (original name was: NODE_145_length_22961_cov_43.6647) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00180_NODE_33..</strong> (original name was: NODE_33_length_81685_cov_55.1586) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00181_NODE_74..</strong> (original name was: NODE_74_length_45591_cov_44.1537) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00182_NODE_78..</strong> (original name was: NODE_78_length_44043_cov_51.9714) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00183_NODE_84..</strong> (original name was: NODE_84_length_40046_cov_46.256) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00184_NODE_20..</strong> (original name was: NODE_206_length_6516_cov_45.597) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00185_NODE_15..</strong> (original name was: NODE_158_length_19730_cov_45.8719) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00186_NODE_16..</strong> (original name was: NODE_167_length_16784_cov_41.755) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00187_NODE_16..</strong> (original name was: NODE_168_length_16106_cov_48.0921) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00188_NODE_16..</strong> (original name was: NODE_165_length_17159_cov_45.8779) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00189_NODE_21..</strong> (original name was: NODE_219_length_4655_cov_40.457) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00190_NODE_10..</strong> (original name was: NODE_102_length_33926_cov_44.9854) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00191_NODE_69..</strong> (original name was: NODE_69_length_51268_cov_48.0754) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00192_NODE_14..</strong> (original name was: NODE_148_length_22452_cov_45.7197) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00193_NODE_17..</strong> (original name was: NODE_172_length_14422_cov_47.2867) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00194_NODE_24..</strong> (original name was: NODE_244_length_1148_cov_30.1699) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00195_NODE_12..</strong> (original name was: NODE_127_length_27461_cov_45.1928) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00196_NODE_15..</strong> (original name was: NODE_156_length_19911_cov_41.7086) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00198_NODE_24..</strong> (original name was: NODE_246_length_1013_cov_10.484) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00199_NODE_23..</strong> (original name was: NODE_233_length_2770_cov_91.059) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00200_NODE_18..</strong> (original name was: NODE_182_length_11957_cov_42.8909) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00202_NODE_16..</strong> (original name was: NODE_164_length_17209_cov_45.6699) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00204_NODE_19..</strong> (original name was: NODE_19_length_100854_cov_47.0425) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00205_NODE_13..</strong> (original name was: NODE_134_length_25517_cov_42.6809) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00207_NODE_23..</strong> (original name was: NODE_239_length_2015_cov_96.2802) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00209_NODE_11..</strong> (original name was: NODE_110_length_31030_cov_45.552) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00210_NODE_10..</strong> (original name was: NODE_107_length_31716_cov_46.2273) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00211_NODE_21..</strong> (original name was: NODE_212_length_5239_cov_44.1635) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00212_NODE_20..</strong> (original name was: NODE_203_length_6899_cov_45.7231) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00213_NODE_17..</strong> (original name was: NODE_175_length_14319_cov_42.1189) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00214_NODE_20..</strong> (original name was: NODE_205_length_6600_cov_46.0806) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00216_NODE_98..</strong> (original name was: NODE_98_length_35295_cov_46.6181) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00218_NODE_76..</strong> (original name was: NODE_76_length_44629_cov_48.2078) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00219_NODE_12..</strong> (original name was: NODE_124_length_28009_cov_44.989) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00220_NODE_20..</strong> (original name was: NODE_202_length_6937_cov_44.9064) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00221_NODE_11..</strong> (original name was: NODE_11_length_113279_cov_48.0857) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00223_NODE_21..</strong> (original name was: NODE_215_length_4976_cov_46.3991) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00224_NODE_51..</strong> (original name was: NODE_51_length_67119_cov_47.3377) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00225_NODE_67..</strong> (original name was: NODE_67_length_54248_cov_46.3713) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00229_NODE_23..</strong> (original name was: NODE_231_length_2929_cov_39.8716) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00230_NODE_18..</strong> (original name was: NODE_181_length_12438_cov_47.4082) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00231_NODE_52..</strong> (original name was: NODE_52_length_66350_cov_45.2388) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00232_NODE_18..</strong> (original name was: NODE_185_length_10755_cov_50.9666) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00233_NODE_12..</strong> (original name was: NODE_121_length_29177_cov_46.8686) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00234_NODE_93..</strong> (original name was: NODE_93_length_36599_cov_43.9011) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00235_NODE_29..</strong> (original name was: NODE_29_length_87133_cov_44.0347) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00236_NODE_23..</strong> (original name was: NODE_237_length_2146_cov_41.8434) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00237_NODE_19..</strong> (original name was: NODE_198_length_8159_cov_45.4866) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00239_NODE_18..</strong> (original name was: NODE_188_length_10376_cov_45.3723) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00241_NODE_22..</strong> (original name was: NODE_227_length_3524_cov_43.3916) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00242_NODE_13..</strong> (original name was: NODE_132_length_26565_cov_45.3075) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00243_NODE_19..</strong> (original name was: NODE_194_length_8979_cov_46.5298) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00244_NODE_22..</strong> (original name was: NODE_228_length_3403_cov_46.0983) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00245_NODE_16..</strong> (original name was: NODE_166_length_17063_cov_44.021) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00246_NODE_20..</strong> (original name was: NODE_201_length_7180_cov_42.748) (Streptomyces sp. CC224B)</dd>
       
         <dd><strong>c00247_NODE_14..</strong> (original name was: NODE_146_length_22856_cov_44.9213) (Streptomyces sp. CC224B)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r2c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00002_NODE_54.. - Region 1 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-388"></div> <span class="help-tooltip" id="region_svg-help-388">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00002_NODE_54...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r2c1-svg" data-filename="c00002_NODE_54.._r2c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 8,116 - 33,620 nt. (total: 25,505 nt)
        
        <a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r2c1-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r2c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-389"></div> <span class="help-tooltip" id="focus-panel-help-389">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r2c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r2c1-lanthipeptides" data-name="r2c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r2c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-1"></div> <span class="help-tooltip" id="lanthi-body-help-1">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds18115_20752 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds20849_20948</span> leader / core peptide
          </dt>
          <dd>
            MTDILELQELEEDTLSL
            -
            WP<span class="cys">C</span>N<span class="dha">Dha</span>G<span class="dha">Dha</span><span class="dha">Dha</span>A<span class="dha">Dha</span>GFAFL
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r2c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r2c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r2c1_1.svg">
        NZ_CP030771 (7308085-7382556): Streptomyces sp. YIM 121038 chromosome, comple... (17% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r2c1_2.svg">
        NZ_RHGC01000002 (926972-1089427): Streptomyces alboflavus strain 313 contig00... (3% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane,lanthipeptide,nucleoside,terpene
      </option>
      
      <option value = "svg/clusterblast_r2c1_3.svg">
        NZ_CP040752 (6733148-6830717): Streptomyces rectiverticillatus strain B15-08 ... (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r2c1_4.svg">
        NZ_CP048641 (146046-223681): Streptomyces aureoverticillatus strain HN6 chrom... (9% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r2c1_5.svg">
        NZ_CP016824 (54347-384189): Streptomyces sampsonii strain KJ40 chromosome, co... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,ectoine,lanthipeptide,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r2c1_6.svg">
        NC_020990 (6567851-6838639): Streptomyces albidoflavus, complete sequence (2% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r2c1_7.svg">
        NZ_CP056773 (6686330-6958067): Streptomyces violascens strain YIM 100212 chro... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r2c1_8.svg">
        NZ_CM002273 (6555444-6843909): Streptomyces sp. PVA 94-07 strain PVA 94-07 ch... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r2c1_9.svg">
        NZ_CM002271 (6545259-6848841): Streptomyces sp. GBA 94-10 chromosome, whole g... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r2c1_10.svg">
        NZ_CP026490 (191791-441920): Streptomyces sp. 604F chromosome, complete genome (2% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r2c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r2c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r2c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r2c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r2c1_1.svg">
        blasticidin S
      </option>
      
    </select>
    <button id="knownclusterblast-r2c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r2c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r2c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r2c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-6"></div> <span class="help-tooltip" id="MIBiG-help-6">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r2c1"></div>
    <div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277.1: 621-57840">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
        
        
        <td>Fusarium graminearum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000877.1: 436-46016">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000877/index.html#r1c1" target="_new">BGC0000877.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>polyoxin A, polyoxin H</td>
        
        
        <td>Streptomyces cacaoi subsp. asoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.47, order:0.35, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277.1: 621-57840">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
      <td>Fusarium graminearum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000877.1: 436-46016">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000877/index.html#r1c1" target="_new">BGC0000877.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>polyoxin A, polyoxin H</td>
      <td>Streptomyces cacaoi subsp. asoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.47, order:0.35, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r2c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-7"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-7">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="c00002_NODE_54.._r2c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r2c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r2c1-lanthipeptides" data-name="r2c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r2c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-2"></div> <span class="help-tooltip" id="lanthi-side-help-2">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds20849_20948</span> - Class III</dt>
        <dd>Cleavage pHMM score: -3.10</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1473.7 Da</dd>
        <dd>Molecular weight: 1474.6 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>Additional modifications: OH</dd>
        
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1492.6 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1510.6 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1528.6 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r4c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_NODE_46.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-390"></div> <span class="help-tooltip" id="region_svg-help-390">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_NODE_46...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c1-svg" data-filename="c00004_NODE_46.._r4c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 25,168 - 67,975 nt. (total: 42,808 nt)
        
        <a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r4c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-391"></div> <span class="help-tooltip" id="focus-panel-help-391">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-8"></div> <span class="help-tooltip" id="nrps-domain-help-8">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="c00004_NODE_46.._r4c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c1_1.svg">
        NZ_CP040916 (1044929-1137636): Streptomyces spectabilis strain NRRL 2792 chro... (41% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r4c1_2.svg">
        NZ_CP030771 (1151977-1233127): Streptomyces sp. YIM 121038 chromosome, comple... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c1_3.svg">
        NZ_CP023690 (1228599-1289798): Streptomyces spectabilis strain ATCC 27465 chr... (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c1_4.svg">
        NZ_VJOK01000001 (401877-491089): Streptomyces albus subsp. chlorinus strain N... (8% of genes show similarity), NRPS,NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r4c1_5.svg">
        NZ_CP026730 (6096827-6139302): Streptomyces sp. CB09001 chromosome, complete ... (14% of genes show similarity), NRPS-like,siderophore
      </option>
      
      <option value = "svg/clusterblast_r4c1_6.svg">
        NZ_BEWD01000003 (12186-53751): Streptomyces rubrogriseus strain NBRC 15455, w... (15% of genes show similarity), NRPS-like,siderophore
      </option>
      
      <option value = "svg/clusterblast_r4c1_7.svg">
        NZ_RHGC01000002 (926972-1089427): Streptomyces alboflavus strain 313 contig00... (6% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane,lanthipeptide,nucleoside,terpene
      </option>
      
      <option value = "svg/clusterblast_r4c1_8.svg">
        NC_016582 (10715816-10757387): Streptomyces bingchenggensis BCW-1, complete s... (20% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c1_9.svg">
        NZ_CP027306 (8641282-8690049): Streptomyces atratus strain SCSIO ZH16 chromos... (15% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r4c1_10.svg">
        NZ_CP022685 (547821-626611): Streptomyces formicae strain KY5 chromosome, com... (9% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_1.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_2.svg">
        echoside A / echoside B / echoside C / echoside D / echoside E
      </option>
      
    </select>
    <button id="knownclusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-14"></div> <span class="help-tooltip" id="MIBiG-help-14">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c1"></div>
    <div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000340.1: 200-18539">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000340/index.html#r1c1" target="_new">BGC0000340.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.37, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>echoside A, echoside B, echoside C, echoside D, echoside E</td>
        
        
        <td>Streptomyces sp. LZ35</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000957.1: 361-40389">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000957/index.html#r1c1" target="_new">BGC0000957.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>NRP, Polyketide</td>
        
        <td>mycotrienin I</td>
        
        
        <td>Streptomyces sp. XZQH13</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001668.1: 790460-797948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001668/index.html#r1c1" target="_new">BGC0001668.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>EQ-4</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.54, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001002.1: 0-25111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001002/index.html#r1c1" target="_new">BGC0001002.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.60, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>NRP, Polyketide</td>
        
        <td>jawsamycin</td>
        
        
        <td>Streptomyces roseoverticillatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000340.1: 200-18539">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000340/index.html#r1c1" target="_new">BGC0000340.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.37, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>echoside A, echoside B, echoside C, echoside D, echoside E</td>
      <td>Streptomyces sp. LZ35</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000957.1: 361-40389">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000957/index.html#r1c1" target="_new">BGC0000957.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP, Polyketide</td>
      <td>mycotrienin I</td>
      <td>Streptomyces sp. XZQH13</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001668.1: 790460-797948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001668/index.html#r1c1" target="_new">BGC0001668.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>EQ-4</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.54, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001002.1: 0-25111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001002/index.html#r1c1" target="_new">BGC0001002.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.60, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP, Polyketide</td>
      <td>jawsamycin</td>
      <td>Streptomyces roseoverticillatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-15"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-15">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="c00004_NODE_46.._r4c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-9"></div> <span class="help-tooltip" id="nrps-structures-help-9">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 25167 - 67975:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-10"></div> <span class="help-tooltip" id="nrps-monomers-help-10">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds45167_47975</span></strong>: X
      <div class="collapser collapser-target-cds45167_47975 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (29..448)</strong>: X
          <div class="collapser collapser-target-cds45167_47975 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds45167_47975-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>glu, gln</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">iAEfIGaagK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r11c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00011_NODE_49.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-392"></div> <span class="help-tooltip" id="region_svg-help-392">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00011_NODE_49...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r11c1-svg" data-filename="c00011_NODE_49.._r11c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 53,110 nt. (total: 53,110 nt)
        
        <a class="cluster-rules-header" id="r11c1-rules-header" href="#r11c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r11c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r11c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-393"></div> <span class="help-tooltip" id="focus-panel-help-393">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r11c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r11c1-nrps_pks" data-name="r11c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r11c1-clusterblast" data-name="r11c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r11c1-knownclusterblast" data-name="r11c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r11c1-subclusterblast" data-name="r11c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r11c1-MIBiG-cluster-compare" data-name="r11c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r11c1-pfam-details" data-name="r11c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r11c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-16"></div> <span class="help-tooltip" id="nrps-domain-help-16">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r11c1-details-svg" data-filename="c00011_NODE_49.._r11c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r11c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r11c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-19"></div> <span class="help-tooltip" id="cb-general-help-19">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r11c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r11c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r11c1_1.svg">
        NZ_CP029823 (2507268-2632312): Streptomyces malaysiensis strain DSM 4137 chro... (10% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r11c1_2.svg">
        NZ_CP020569 (1998892-2098465): Streptomyces gilvosporeus strain F607 chromoso... (12% of genes show similarity), NRPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r11c1_3.svg">
        NC_013441 (3915879-4049788): Gordonia bronchialis DSM 43247, complete sequence (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r11c1_4.svg">
        NC_016906 (3993609-4119698): Gordonia polyisoprenivorans VH2, complete sequence (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r11c1_5.svg">
        NZ_CP047235 (3888053-4016180): Gordonia sp. JH63 chromosome, complete genome (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r11c1_6.svg">
        NZ_CP023699 (2375586-2502315): Streptomyces kanamyceticus strain ATCC 12853 c... (15% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r11c1_7.svg">
        NZ_GL877172 (3937216-4060984): Streptomyces sp. ACT-1 strain XylebKG-1 scaffo... (13% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r11c1_8.svg">
        NZ_FLTQ01000003 (173125-298252): Streptomyces sp. MnatMP-M77, whole genome sh... (12% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r11c1_9.svg">
        NC_010572 (3747946-3871878): Streptomyces griseus subsp. griseus NBRC 13350, ... (13% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r11c1_10.svg">
        NZ_CP012949 (5818422-5994901): Streptomyces ambofaciens strain DSM 40697 chro... (9% of genes show similarity), NRPS,T1PKS,oligosaccharide
      </option>
      
    </select>
    <button id= "clusterblast-r11c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r11c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r11c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-20"></div> <span class="help-tooltip" id="cb-known-help-20">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r11c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r11c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_1.svg">
        polyoxypeptin
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_2.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_3.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_4.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_5.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_6.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_7.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_8.svg">
        A-500359 A / A-500359 B
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_9.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r11c1_10.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
    </select>
    <button id="knownclusterblast-r11c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r11c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r11c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-21"></div> <span class="help-tooltip" id="cb-sub-help-21">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r11c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-22"></div> <span class="help-tooltip" id="MIBiG-help-22">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r11c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r11c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r11c1"></div>
    <div id="comparison-MIBiG-r11c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.43, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>erythrochelin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>Jessenipeptin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:0.57, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001502.1: 2741865-2757479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001502/index.html#r1c1" target="_new">BGC0001502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>amonabactin P 750</td>
        
        
        <td>Aeromonas hydrophila subsp. hydrophila ATCC 7966</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>nematophin</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r11c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.43, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>erythrochelin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>mirubactin</td>
      <td>Actinosynnema mirum DSM 43827</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>Jessenipeptin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:0.57, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001502.1: 2741865-2757479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001502/index.html#r1c1" target="_new">BGC0001502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>amonabactin P 750</td>
      <td>Aeromonas hydrophila subsp. hydrophila ATCC 7966</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>nematophin</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r11c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-23"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-23">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r11c1-pfam-details-svg" data-filename="c00011_NODE_49.._r11c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r11c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r11c1-nrps_pks_products" data-name="r11c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r11c1-nrps_pks" data-name="r11c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r11c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-17"></div> <span class="help-tooltip" id="nrps-structures-help-17">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 53110:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r11c1-1" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)CC(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-val - ala) + (D-X) + (mal - X - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val,ala&amp;nrps2=d-x&amp;nrps3=mal,x,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val*,ala*&amp;nrps2=d-x*&amp;nrps3=mal*,x,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r11c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)CC(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r11c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-18"></div> <span class="help-tooltip" id="nrps-monomers-help-18">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds8800_15349</span></strong>: val - ala
      <div class="collapser collapser-target-cds8800_15349 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val,ala" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*,ala*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (11..434)</strong>: val
          <div class="collapser collapser-target-cds8800_15349 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds8800_15349-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYFWGVvFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1599..1991)</strong>: ala
          <div class="collapser collapser-target-cds8800_15349 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds8800_15349-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSvAgfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds15345_21654</span></strong>: X
      <div class="collapser collapser-target-cds15345_21654 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (462..874)</strong>: X
          <div class="collapser collapser-target-cds15345_21654 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds15345_21654-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVFSVGcasK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>d-lyserg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21650_33110</span></strong>: X - X
      <div class="collapser collapser-target-cds21650_33110 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1297..1706)</strong>: X
          <div class="collapser collapser-target-cds21650_33110 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds21650_33110-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAyLwGAVvK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2353..2730)</strong>: X
          <div class="collapser collapser-target-cds21650_33110 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala
              <div class="collapser collapser-target-cds21650_33110-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVfCvAAfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r23c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00023_NODE_53.. - Region 1 - melanin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-394"></div> <span class="help-tooltip" id="region_svg-help-394">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00023_NODE_53...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r23c1-svg" data-filename="c00023_NODE_53.._r23c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 13,088 - 23,498 nt. (total: 10,411 nt)
        
        <a class="cluster-rules-header" id="r23c1-rules-header" href="#r23c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r23c1-rules">melanin: melC</div>
      
      <div class="region-svg-container">
       <div id='r23c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-395"></div> <span class="help-tooltip" id="focus-panel-help-395">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r23c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r23c1-clusterblast" data-name="r23c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r23c1-knownclusterblast" data-name="r23c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r23c1-subclusterblast" data-name="r23c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r23c1-MIBiG-cluster-compare" data-name="r23c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r23c1-pfam-details" data-name="r23c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r23c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r23c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r23c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r23c1_1.svg">
        NZ_CP029193 (4362597-4372975): Streptomyces venezuelae strain ATCC 14583 chro... (60% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_2.svg">
        NZ_CP048641 (4566767-4574094): Streptomyces aureoverticillatus strain HN6 chr... (83% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_3.svg">
        NZ_CP021748 (4786298-4796700): Streptomyces alboflavus strain MDJK44 chromoso... (69% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_4.svg">
        NZ_RHGC01000005 (375578-385980): Streptomyces alboflavus strain 313 contig005... (69% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_5.svg">
        NZ_WPNZ01000020 (75605-85977): Streptomyces sp. p1417 Scaffold20, whole genom... (75% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_6.svg">
        NZ_CP029192 (4210906-4221284): Streptomyces venezuelae strain ATCC 14584 chro... (53% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_7.svg">
        NZ_CP029191 (4299223-4309601): Streptomyces venezuelae strain ATCC 14585 chro... (50% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_8.svg">
        NZ_CP045096 (3407485-3417926): Streptomyces sp. GY16 chromosome, complete genome (57% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_9.svg">
        NZ_KB911581 (7515748-7526189): Streptomyces bottropensis ATCC 25435 StrboDRAF... (66% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r23c1_10.svg">
        NC_013929 (6584190-6592550): Streptomyces scabiei 87.22, complete genome (80% of genes show similarity), melanin
      </option>
      
    </select>
    <button id= "clusterblast-r23c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r23c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r23c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r23c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r23c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r23c1_1.svg">
        istamycin
      </option>
      
      <option value = "svg/knownclusterblast_r23c1_2.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r23c1_3.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r23c1_4.svg">
        lactonamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r23c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r23c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r23c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r23c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-27"></div> <span class="help-tooltip" id="MIBiG-help-27">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r23c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r23c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r23c1"></div>
    <div id="comparison-MIBiG-r23c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">melanin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.82 (id:0.55, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
         
        
        <td>0.56</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
         
        
        <td>0.52</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.54, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.48, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.47, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.47, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.47, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.47, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r23c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.82 (id:0.55, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
      <td>0.82</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.54, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.48, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.47, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.47, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.47, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.47, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r23c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-28"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-28">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r23c1-pfam-details-svg" data-filename="c00023_NODE_53.._r23c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r23c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r24c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00024_NODE_19.. - Region 1 - lassopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-396"></div> <span class="help-tooltip" id="region_svg-help-396">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00024_NODE_19...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r24c1-svg" data-filename="c00024_NODE_19.._r24c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 9,307 nt. (total: 9,307 nt)
        
        <a class="cluster-rules-header" id="r24c1-rules-header" href="#r24c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r24c1-rules">lassopeptide: (PF13471 and Asn_synthase or micJ25 or mcjC)</div>
      
      <div class="region-svg-container">
       <div id='r24c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-397"></div> <span class="help-tooltip" id="focus-panel-help-397">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r24c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r24c1-clusterblast" data-name="r24c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r24c1-knownclusterblast" data-name="r24c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r24c1-subclusterblast" data-name="r24c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r24c1-lassopeptides" data-name="r24c1-lassopeptides">
        Lasso peptides
      </div>
     
    
     
    
     
      <div class="body-details-header r24c1-MIBiG-cluster-compare" data-name="r24c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r24c1-pfam-details" data-name="r24c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r24c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-29"></div> <span class="help-tooltip" id="cb-general-help-29">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r24c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r24c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r24c1_1.svg">
        NZ_CP023703 (6065116-6087641): Streptomyces galilaeus strain ATCC 14969 chrom... (27% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_2.svg">
        NZ_LFBV01000003 (428239-508216): Streptomyces uncialis strain DCA2648 scaffol... (15% of genes show similarity), NRPS,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_3.svg">
        NZ_AQRJ01000070 (440148-528806): Streptomyces sp. AmelKG-E11A YW7DRAFT scaffo... (15% of genes show similarity), NRPS,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_4.svg">
        NZ_CDME01000002 (58394-246440): Kibdelosporangium sp. MJ126-NF4, whole genome... (6% of genes show similarity), NRPS,NRPS-like,T1PKS,betalactone,lassopeptide,other
      </option>
      
      <option value = "svg/clusterblast_r24c1_5.svg">
        NZ_JACJIE010000001 (1087838-1110359): Streptomyces calvus strain CECT 3271 Ga... (33% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_6.svg">
        NZ_CP022310 (1616274-1637963): Streptomyces asterosporus strain DSM 41452 chr... (33% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_7.svg">
        NZ_PVTF01000005 (248379-270903): Umezawaea tangerina strain DSM 44720 Ga01809... (36% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_8.svg">
        NZ_SGWQ01000010 (116960-139463): Herbihabitans rhizosphaerae strain DSM 10172... (29% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_9.svg">
        NZ_LIPD01000011 (84614-159054): Streptomyces sp. CB02056 scaffold11, whole ge... (20% of genes show similarity), NRPS,T1PKS,bacteriocin,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_10.svg">
        NZ_CP048875 (5466281-5488276): Streptomyces albus strain CAS922 chromosome, c... (50% of genes show similarity), lassopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r24c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r24c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r24c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-30"></div> <span class="help-tooltip" id="cb-known-help-30">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r24c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r24c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_1.svg">
        SSV-2083
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_2.svg">
        siamycin I
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_3.svg">
        MS-271
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_4.svg">
        MS-271
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_5.svg">
        siamycin
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_6.svg">
        aborycin
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_7.svg">
        moomysin
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_8.svg">
        anantin C
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_9.svg">
        anantin B1 / anantin B2
      </option>
      
      <option value = "svg/knownclusterblast_r24c1_10.svg">
        cattlecin
      </option>
      
    </select>
    <button id="knownclusterblast-r24c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r24c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r24c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-31"></div> <span class="help-tooltip" id="cb-sub-help-31">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r24c1-lassopeptides">
      <div class="details">
    <div class="heading">
      <span>Lasso peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-body-help-32"></div> <span class="help-tooltip" id="lasso-body-help-32">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lassopeptide(s) for cds4331_4457 - Putative Class III</h4>
    <hr>
      <dl class="details-text">
        
          <dt><span class="serif">cds4331_4457</span> leader / core peptide / cleaved tail</dt>
          <dd>
            METNDVYEPPALEEIGGFAELTM
            -
            <span class ="macrolactam">A</span><span class ="macrolactam">F</span><span class ="macrolactam">G</span><span class ="macrolactam">P</span><span class="cys">C</span><span class ="macrolactam">V</span><span class ="macrolactam">E</span><span class ="macrolactam">N</span><span class ="macrolactam">D</span>WLWGTAWI<span class="cys">C</span>
          </dd>
        
      </dl>
    
  <div class="legend">
    <h4> Legend: </h4>
    <div>
      <span class="macrolactam">Red</span>: Putative macrolactam<br>
      <span class="cut">Purple</span>: Putative cleaved off residues<br>
    </div>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r24c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-35"></div> <span class="help-tooltip" id="MIBiG-help-35">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r24c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r24c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r24c1"></div>
    <div id="comparison-MIBiG-r24c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lassopeptide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.47, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.49</td>
        <td>RiPP</td>
        
        <td>citrulassin E</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001539.1: 77018-79728">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001539/index.html#r1c1" target="_new">BGC0001539.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>RiPP</td>
        
        <td>cattlecin</td>
        
        
        <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001673.1: 77019-79729">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001673/index.html#r1c1" target="_new">BGC0001673.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>RiPP</td>
        
        <td>moomysin</td>
        
        
        <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>RiPP</td>
        
        <td>ulleungdin</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002074.1: 0-2630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002074/index.html#r1c1" target="_new">BGC0002074.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>RiPP</td>
        
        <td>huascopeptin</td>
        
        
        <td>Streptomyces huasconensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001507.1: 43037-46158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001507/index.html#r1c1" target="_new">BGC0001507.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>RiPP</td>
        
        <td>anantin C</td>
        
        
        <td>Streptomyces olindensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001549.1: 98590-103146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001549/index.html#r1c1" target="_new">BGC0001549.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.47, order:0.80, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>RiPP</td>
        
        <td>citrulassin B</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>RiPP</td>
        
        <td>citrulassin A</td>
        
        
        <td>Streptomyces albulus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001889.1: 1333039-1337282">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001889/index.html#r1c1" target="_new">BGC0001889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.47, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>RiPP</td>
        
        <td>fusilassin</td>
        
        
        <td>Thermobifida fusca YX</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001655.1: 360744-365660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001655/index.html#r1c1" target="_new">BGC0001655.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>RiPP</td>
        
        <td>LP2006</td>
        
        
        <td>Nocardiopsis alba DSM 43377</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r24c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.47, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>RiPP</td>
      <td>citrulassin E</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001539.1: 77018-79728">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001539/index.html#r1c1" target="_new">BGC0001539.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>RiPP</td>
      <td>cattlecin</td>
      <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001673.1: 77019-79729">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001673/index.html#r1c1" target="_new">BGC0001673.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>RiPP</td>
      <td>moomysin</td>
      <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>RiPP</td>
      <td>ulleungdin</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002074.1: 0-2630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002074/index.html#r1c1" target="_new">BGC0002074.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>RiPP</td>
      <td>huascopeptin</td>
      <td>Streptomyces huasconensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001507.1: 43037-46158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001507/index.html#r1c1" target="_new">BGC0001507.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>RiPP</td>
      <td>anantin C</td>
      <td>Streptomyces olindensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001549.1: 98590-103146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001549/index.html#r1c1" target="_new">BGC0001549.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.47, order:0.80, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>RiPP</td>
      <td>citrulassin B</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>RiPP</td>
      <td>citrulassin A</td>
      <td>Streptomyces albulus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001889.1: 1333039-1337282">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001889/index.html#r1c1" target="_new">BGC0001889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.47, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>RiPP</td>
      <td>fusilassin</td>
      <td>Thermobifida fusca YX</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001655.1: 360744-365660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001655/index.html#r1c1" target="_new">BGC0001655.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>RiPP</td>
      <td>LP2006</td>
      <td>Nocardiopsis alba DSM 43377</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r24c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-36"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-36">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r24c1-pfam-details-svg" data-filename="c00024_NODE_19.._r24c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r24c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r24c1-lassopeptides" data-name="r24c1-lassopeptides">
        Lasso peptides
      </div>
     
    
     
      <div class="sidepanel-details-header r24c1-RREfinder" data-name="r24c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r24c1-lassopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-side-help-33"></div> <span class="help-tooltip" id="lasso-side-help-33">Lists the possible core peptides in the region. Each core peptide lists the number of disulfide bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
    </div>

  <dl class ="prediction-text">
    <dt><span class="serif">cds4331_4457</span> - Class III</dt>
      <dd>Cleavage pHMM score: -0.50</dd>
      <dd>RODEO score: 21</dd>
      <dd>Monoisotopic mass: 2050.9 Da</dd>
      <dd>Molecular weight: 2052.3 Da</dd>
      <dd>Number of bridges: 1</dd>
    
  </dl>

</div>

    </div>
     
    
     
    <div class="sidepanel-details-section r24c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-34"></div> <span class="help-tooltip" id="rre-side-help-34">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lassopeptide protocluster</strong> (0...9307)</span>
      <dt></dt>
      
      
      
        
        <strong>cds6541_6802</strong> <div class="collapser collapser-target-cds6541_6802 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Stand_Alone_Lasso_RRE</dd>
          <dd>Location: 0 - 84</dd>
          <dd>Score: 81.4</dd>
        
          <dd>RRE type: Lasso_Fused_RRE</dd>
          <dd>Location: 0 - 54</dd>
          <dd>Score: 37.5</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r27c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00027_NODE_58.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-398"></div> <span class="help-tooltip" id="region_svg-help-398">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00027_NODE_58...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r27c1-svg" data-filename="c00027_NODE_58.._r27c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 21,346 - 31,801 nt. (total: 10,456 nt)
        
        <a class="cluster-rules-header" id="r27c1-rules-header" href="#r27c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r27c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r27c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-399"></div> <span class="help-tooltip" id="focus-panel-help-399">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r27c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r27c1-clusterblast" data-name="r27c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r27c1-knownclusterblast" data-name="r27c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r27c1-subclusterblast" data-name="r27c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r27c1-MIBiG-cluster-compare" data-name="r27c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r27c1-pfam-details" data-name="r27c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r27c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r27c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r27c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r27c1_1.svg">
        NZ_LMDL01000014 (94150-116474): Streptomyces sp. Root369 contig 21, whole gen... (11% of genes show similarity), TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r27c1_2.svg">
        NZ_ML123049 (460029-470494): Streptomyces sp. ADI95-17 scaffold006, whole gen... (25% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r27c1_3.svg">
        NC_015957 (8814683-8837881): Streptomyces violaceusniger Tu 4113, complete se... (11% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r27c1_4.svg">
        NZ_CP029191 (5397372-5464392): Streptomyces venezuelae strain ATCC 14585 chro... (4% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r27c1_5.svg">
        NZ_AUKN01000001 (514512-555697): Streptomyces sp. CNT318 H296DRAFT scaffold00... (4% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r27c1_6.svg">
        NZ_FOGO01000009 (6866-48048): Streptomyces qinglanensis strain CGMCC 4.6825, ... (5% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r27c1_7.svg">
        NZ_CP015596 (4976859-5019421): Mycobacterium sp. YC-RL4 chromosome, complete ... (4% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r27c1_8.svg">
        NZ_CP023693 (3291783-3347696): Streptomyces cinereoruber strain ATCC 19740 ch... (4% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r27c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r27c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r27c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r27c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r27c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-40"></div> <span class="help-tooltip" id="MIBiG-help-40">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r27c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r27c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r27c1"></div>
    <div id="comparison-MIBiG-r27c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548.1: 588-10610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>salivaricin A</td>
        
        
        <td>Streptococcus salivarius</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001765.1: 1227026-1251375">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001765/index.html#r1c1" target="_new">BGC0001765.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>salecan</td>
        
        
        <td>Agrobacterium tumefaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>gellan polysaccharide</td>
        
        
        <td>Sphingomonas elodea ATCC 31461</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>sphingan polysaccharide</td>
        
        
        <td>Sphingomonas sp. S88</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000759.1: 1-29377">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000759/index.html#r1c1" target="_new">BGC0000759.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>diutan polysaccharide</td>
        
        
        <td>Sphingomonas sp. ATCC 53159</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
        
        
        <td>Actinomadura sp. ATCC 39365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>NRP, Polyketide</td>
        
        <td>burkholderic acid</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r27c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548.1: 588-10610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>salivaricin A</td>
      <td>Streptococcus salivarius</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001765.1: 1227026-1251375">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001765/index.html#r1c1" target="_new">BGC0001765.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>salecan</td>
      <td>Agrobacterium tumefaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>gellan polysaccharide</td>
      <td>Sphingomonas elodea ATCC 31461</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>sphingan polysaccharide</td>
      <td>Sphingomonas sp. S88</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000759.1: 1-29377">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000759/index.html#r1c1" target="_new">BGC0000759.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>diutan polysaccharide</td>
      <td>Sphingomonas sp. ATCC 53159</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
      <td>Actinomadura sp. ATCC 39365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>NRP, Polyketide</td>
      <td>burkholderic acid</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r27c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-41"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-41">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r27c1-pfam-details-svg" data-filename="c00027_NODE_58.._r27c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r27c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r30c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00030_NODE_31.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-400"></div> <span class="help-tooltip" id="region_svg-help-400">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00030_NODE_31...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r30c1-svg" data-filename="c00030_NODE_31.._r30c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 37,948 - 64,231 nt. (total: 26,284 nt)
        
        <a class="cluster-rules-header" id="r30c1-rules-header" href="#r30c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r30c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r30c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-401"></div> <span class="help-tooltip" id="focus-panel-help-401">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r30c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r30c1-clusterblast" data-name="r30c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r30c1-knownclusterblast" data-name="r30c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r30c1-subclusterblast" data-name="r30c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r30c1-MIBiG-cluster-compare" data-name="r30c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r30c1-pfam-details" data-name="r30c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r30c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-42"></div> <span class="help-tooltip" id="cb-general-help-42">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r30c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r30c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r30c1_1.svg">
        NZ_KQ948549 (718558-769573): Streptomyces longwoodensis strain DSM 41677 PRJN... (22% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_2.svg">
        NZ_KQ948852 (204386-265819): Streptomyces bungoensis strain DSM 41781 PRJNA29... (19% of genes show similarity), other,terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_3.svg">
        NZ_BJTV01000002 (379618-441555): Streptomyces sp. 1-11 strain 1 11 sequence02... (15% of genes show similarity), T3PKS,lassopeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_4.svg">
        NZ_SZNQ01000001 (582308-606657): Streptomyces lasalocidi strain X-537 Scaffol... (34% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_5.svg">
        NZ_SLZB01000001 (770772-796664): Streptomyces sp. BK308 Ga0307715 101, whole ... (45% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_6.svg">
        NZ_CP047144 (7700759-7747106): Streptomyces sp. HF10 chromosome, complete genome (23% of genes show similarity), lanthipeptide,other,terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_7.svg">
        NZ_CP045702 (207549-231831): Streptomyces finlayi strain NBSH44 chromosome, c... (42% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_8.svg">
        NZ_CP047140 (2102126-2126425): Streptomyces sp. Tu 2975 chromosome, complete ... (45% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_9.svg">
        NZ_QFRX01000001 (596061-620996): Streptomyces sp. Act143 Scaffold 1, whole ge... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r30c1_10.svg">
        NZ_CP015866 (133416-190400): Streptomyces parvulus strain 2297 chromosome, co... (20% of genes show similarity), NRPS-like,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r30c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r30c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r30c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-43"></div> <span class="help-tooltip" id="cb-known-help-43">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r30c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r30c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_1.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_4.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_5.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_6.svg">
        rustmicin
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_7.svg">
        acarviostatin I03 / acarviostatin II03 / acarviostatin III03 / acarviostatin IV03
      </option>
      
      <option value = "svg/knownclusterblast_r30c1_8.svg">
        nocamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r30c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r30c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r30c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-44"></div> <span class="help-tooltip" id="cb-sub-help-44">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r30c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-45"></div> <span class="help-tooltip" id="MIBiG-help-45">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r30c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r30c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r30c1"></div>
    <div id="comparison-MIBiG-r30c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.56, order:0.93, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.53</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.58, order:0.82, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.49</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.59, order:0.67, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.67, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.41</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.55, order:0.82, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC260</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r30c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.56, order:0.93, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.58, order:0.82, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.59, order:0.67, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.67, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.55, order:0.82, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC260</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r30c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-46"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-46">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r30c1-pfam-details-svg" data-filename="c00030_NODE_31.._r30c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r30c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r38c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00038_NODE_13.. - Region 1 - NRPS,T3PKS,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-402"></div> <span class="help-tooltip" id="region_svg-help-402">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00038_NODE_13...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r38c1-svg" data-filename="c00038_NODE_13.._r38c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 56,937 - 106,741 nt. (total: 49,805 nt)
        
        <a class="cluster-rules-header" id="r38c1-rules-header" href="#r38c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r38c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)<br>terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r38c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-403"></div> <span class="help-tooltip" id="focus-panel-help-403">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r38c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r38c1-nrps_pks" data-name="r38c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r38c1-clusterblast" data-name="r38c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r38c1-knownclusterblast" data-name="r38c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r38c1-subclusterblast" data-name="r38c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r38c1-MIBiG-cluster-compare" data-name="r38c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r38c1-pfam-details" data-name="r38c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r38c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-47"></div> <span class="help-tooltip" id="nrps-domain-help-47">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r38c1-details-svg" data-filename="c00038_NODE_13.._r38c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r38c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r38c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-50"></div> <span class="help-tooltip" id="cb-general-help-50">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r38c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r38c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r38c1_1.svg">
        NZ_CP023690 (7364786-7440020): Streptomyces spectabilis strain ATCC 27465 chr... (32% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r38c1_2.svg">
        NZ_CP030771 (7308085-7382556): Streptomyces sp. YIM 121038 chromosome, comple... (37% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r38c1_3.svg">
        NZ_CP040916 (7029502-7103198): Streptomyces spectabilis strain NRRL 2792 chro... (35% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r38c1_4.svg">
        NZ_KQ947989 (383161-405153): Streptomyces curacoi strain DSM 40107 PRJNA29921... (60% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r38c1_5.svg">
        NZ_CP029193 (7590311-7706501): Streptomyces venezuelae strain ATCC 14583 chro... (14% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r38c1_6.svg">
        NZ_CP040752 (41803-63024): Streptomyces rectiverticillatus strain B15-08 chro... (75% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r38c1_7.svg">
        NZ_BBUY01000010 (181821-209759): Streptomyces sp. NBRC 110611, whole genome s... (39% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r38c1_8.svg">
        NZ_CP029191 (7530029-7647507): Streptomyces venezuelae strain ATCC 14585 chro... (12% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r38c1_9.svg">
        NZ_JACHJF010000005 (229535-348707): Streptomyces eurocidicus strain CECT 3259... (10% of genes show similarity), NRPS,T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r38c1_10.svg">
        NZ_CP011868 (5694644-5778127): Pseudonocardia sp. HH130629-09 chromosome, com... (12% of genes show similarity), NRPS,other
      </option>
      
    </select>
    <button id= "clusterblast-r38c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r38c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r38c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-51"></div> <span class="help-tooltip" id="cb-known-help-51">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r38c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r38c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_1.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_2.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_3.svg">
        malacidin A / malacidin B
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_4.svg">
        griselimycin
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_5.svg">
        acyldepsipeptide 1
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_6.svg">
        actinospectacin
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_7.svg">
        spectinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_8.svg">
        C-1027
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_9.svg">
        lidamycin
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_10.svg">
        C-1027
      </option>
      
    </select>
    <button id="knownclusterblast-r38c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r38c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r38c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-52"></div> <span class="help-tooltip" id="cb-sub-help-52">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r38c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-53"></div> <span class="help-tooltip" id="MIBiG-help-53">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r38c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r38c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r38c1"></div>
    <div id="comparison-MIBiG-r38c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
         <th><span class="cc-heat-product">terpene</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>1.15</td>
        <td>Polyketide</td>
        
        <td>alkylresorcinol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.49, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>1.06</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.46, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.72</td>
        <td>Polyketide</td>
        
        <td>cepacin A</td>
        
        
        <td>Burkholderia ambifaria IOP40-10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.68</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.66</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669.1: 17023645-17057975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.64</td>
        <td>Terpene</td>
        
        <td>marneral</td>
        
        
        <td>Arabidopsis thaliana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.64</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.64</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.63</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.46, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.61</td>
        <td>Polyketide</td>
        
        <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
        
        
        <td>Streptomyces sp. CB09001</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r38c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Polyketide</td>
      <td>alkylresorcinol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>alkylpyrone-407, alkylpyrone-393</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.54, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Alkaloid</td>
      <td>K-252a</td>
      <td>Nonomuraea longicatena</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669.1: 17023645-17057975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>marneral</td>
      <td>Arabidopsis thaliana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000827.1: 2672472-2696772">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000827/index.html#r1c1" target="_new">BGC0000827.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Alkaloid</td>
      <td>staurosporine</td>
      <td>Salinispora arenicola CNS-205</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000724.1: 6919727-6951001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000724/index.html#r1c1" target="_new">BGC0000724.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.54, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Saccharide</td>
      <td>streptomycin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>cepacin A</td>
      <td>Burkholderia ambifaria IOP40-10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000653.1: 0-21693">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000653/index.html#r1c1" target="_new">BGC0000653.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r38c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-54"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-54">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r38c1-pfam-details-svg" data-filename="c00038_NODE_13.._r38c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r38c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r38c1-nrps_pks_products" data-name="r38c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r38c1-nrps_pks" data-name="r38c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r38c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-48"></div> <span class="help-tooltip" id="nrps-structures-help-48">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 56936 - 106741:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r38c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-val)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-val" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-val*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r38c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 56936 - 98037:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r38c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r38c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 72262 - 106741:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r38c1-4" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-val)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-val" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-val*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r38c1-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r38c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-49"></div> <span class="help-tooltip" id="nrps-monomers-help-49">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds92262_95400</span></strong>: X
      <div class="collapser collapser-target-cds92262_95400 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (468..858)</strong>: X
          <div class="collapser collapser-target-cds92262_95400 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds92262_95400-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVfhVAAfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>alaninol</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds99570_105705</span></strong>: val
      <div class="collapser collapser-target-cds99570_105705 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (31..422)</strong>: val
          <div class="collapser collapser-target-cds99570_105705 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds99570_105705-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlWWGGvFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r40c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00040_NODE_47.. - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-404"></div> <span class="help-tooltip" id="region_svg-help-404">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00040_NODE_47...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r40c1-svg" data-filename="c00040_NODE_47.._r40c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 5,549 - 46,589 nt. (total: 41,041 nt)
        
        <a class="cluster-rules-header" id="r40c1-rules-header" href="#r40c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r40c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r40c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-405"></div> <span class="help-tooltip" id="focus-panel-help-405">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r40c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r40c1-clusterblast" data-name="r40c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r40c1-knownclusterblast" data-name="r40c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r40c1-subclusterblast" data-name="r40c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r40c1-MIBiG-cluster-compare" data-name="r40c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r40c1-pfam-details" data-name="r40c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r40c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-55"></div> <span class="help-tooltip" id="cb-general-help-55">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r40c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r40c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r40c1_1.svg">
        NZ_CP021748 (1213222-1254263): Streptomyces alboflavus strain MDJK44 chromoso... (68% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_2.svg">
        NZ_RHGC01000003 (662866-703907): Streptomyces alboflavus strain 313 contig003... (73% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_3.svg">
        NZ_WPNZ01000016 (160164-201205): Streptomyces sp. p1417 Scaffold16, whole gen... (78% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_4.svg">
        NZ_CP022685 (314533-440236): Streptomyces formicae strain KY5 chromosome, com... (26% of genes show similarity), NRPS,T3PKS,bacteriocin,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r40c1_5.svg">
        NZ_CP023699 (9456716-9502870): Streptomyces kanamyceticus strain ATCC 12853 c... (68% of genes show similarity), T3PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_6.svg">
        NZ_CP029193 (299191-338016): Streptomyces venezuelae strain ATCC 14583 chromo... (71% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_7.svg">
        NZ_CP015588 (7524603-7563312): Streptomyces alfalfae strain ACCC40021 chromos... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_8.svg">
        NZ_CP040916 (1456557-1496803): Streptomyces spectabilis strain NRRL 2792 chro... (69% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_9.svg">
        NZ_CP029617 (292430-331932): Streptomyces sp. WAC 01529 chromosome, complete ... (65% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r40c1_10.svg">
        NZ_CP023690 (1601643-1642333): Streptomyces spectabilis strain ATCC 27465 chr... (65% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r40c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r40c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r40c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-56"></div> <span class="help-tooltip" id="cb-known-help-56">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r40c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r40c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_1.svg">
        BE-14106
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_2.svg">
        violapyrone B
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_3.svg">
        akaeolide
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_4.svg">
        amycolamycin A / amycolamycin B
      </option>
      
    </select>
    <button id="knownclusterblast-r40c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r40c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r40c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-57"></div> <span class="help-tooltip" id="cb-sub-help-57">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r40c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-58"></div> <span class="help-tooltip" id="MIBiG-help-58">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r40c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r40c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r40c1"></div>
    <div id="comparison-MIBiG-r40c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.52, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.62, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001905.1: 0-3803">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001905/index.html#r1c1" target="_new">BGC0001905.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.55, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>violapyrone B</td>
        
        
        <td>Streptomyces somaliensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846.1: 192-22933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>tabtoxin</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937.1: 9913-33560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>fosfazinomycin A</td>
        
        
        <td>Streptomyces sp. WM6372</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001739.1: 94227-109042">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001739/index.html#r1c1" target="_new">BGC0001739.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>phosphonoacetic Acid</td>
        
        
        <td>Streptomyces sp. NRRL F-525</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.53, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>A-94964</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374.1: 52-17210">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>thienodolin</td>
        
        
        <td>Streptomyces albogriseolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000874.1: 0-36223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000874/index.html#r1c1" target="_new">BGC0000874.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>blasticidin S</td>
        
        
        <td>Streptomyces griseochromogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076.1: 0-29709">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.53, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
        
        
        <td>Streptomyces sp. SN-1061M</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r40c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.52, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.62, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001905.1: 0-3803">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001905/index.html#r1c1" target="_new">BGC0001905.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.55, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>violapyrone B</td>
      <td>Streptomyces somaliensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846.1: 192-22933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>tabtoxin</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937.1: 9913-33560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>fosfazinomycin A</td>
      <td>Streptomyces sp. WM6372</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001739.1: 94227-109042">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001739/index.html#r1c1" target="_new">BGC0001739.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>phosphonoacetic Acid</td>
      <td>Streptomyces sp. NRRL F-525</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.53, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>A-94964</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374.1: 52-17210">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>thienodolin</td>
      <td>Streptomyces albogriseolus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000874.1: 0-36223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000874/index.html#r1c1" target="_new">BGC0000874.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>blasticidin S</td>
      <td>Streptomyces griseochromogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076.1: 0-29709">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.53, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
      <td>Streptomyces sp. SN-1061M</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r40c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-59"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-59">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r40c1-pfam-details-svg" data-filename="c00040_NODE_47.._r40c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r40c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r49c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00049_NODE_82.. - Region 1 - NRPS-like,T1PKS,prodigiosin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-406"></div> <span class="help-tooltip" id="region_svg-help-406">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00049_NODE_82...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r49c1-svg" data-filename="c00049_NODE_82.._r49c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 41,000 nt. (total: 41,000 nt)
        
        <a class="cluster-rules-header" id="r49c1-rules-header" href="#r49c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r49c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>prodigiosin: (PPDK_N and AMP-binding and PP-binding and PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r49c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-407"></div> <span class="help-tooltip" id="focus-panel-help-407">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r49c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r49c1-nrps_pks" data-name="r49c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r49c1-clusterblast" data-name="r49c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r49c1-knownclusterblast" data-name="r49c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r49c1-subclusterblast" data-name="r49c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r49c1-MIBiG-cluster-compare" data-name="r49c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r49c1-pfam-details" data-name="r49c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r49c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-60"></div> <span class="help-tooltip" id="nrps-domain-help-60">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r49c1-details-svg" data-filename="c00049_NODE_82.._r49c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r49c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r49c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-63"></div> <span class="help-tooltip" id="cb-general-help-63">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r49c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r49c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r49c1_1.svg">
        NZ_CP048641 (1255717-1355003): Streptomyces aureoverticillatus strain HN6 chr... (37% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_2.svg">
        NZ_CP040916 (7561982-7630130): Streptomyces spectabilis strain NRRL 2792 chro... (50% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_3.svg">
        NZ_CP023690 (7874786-7942911): Streptomyces spectabilis strain ATCC 27465 chr... (50% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_4.svg">
        NZ_CP030771 (7834424-7901028): Streptomyces sp. YIM 121038 chromosome, comple... (49% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_5.svg">
        NZ_WPNZ01000004 (400429-487232): Streptomyces sp. p1417 Scaffold4, whole geno... (38% of genes show similarity), NRPS-like,T1PKS,hglE-KS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_6.svg">
        NZ_RHGC01000006 (59286-149148): Streptomyces alboflavus strain 313 contig006,... (41% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_7.svg">
        NZ_CP021748 (8069033-&gt;8156625): Streptomyces alboflavus strain MDJK44 chromos... (41% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r49c1_8.svg">
        NZ_KZ679042 (391086-439159): Streptomyces dioscori strain A217 scaffold5, who... (54% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r49c1_9.svg">
        NZ_CP031194 (6977961-7044382): Streptomyces paludis strain GSSD-12 chromosome... (38% of genes show similarity), NRPS-like,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r49c1_10.svg">
        NZ_AUKN01000013 (7973-54759): Streptomyces sp. CNT318 H296DRAFT scaffold00012... (51% of genes show similarity), NRPS-like,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r49c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r49c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r49c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-64"></div> <span class="help-tooltip" id="cb-known-help-64">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r49c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r49c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_1.svg">
        marineosin A / marineosin B
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_2.svg">
        undecylprodigiosin
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_3.svg">
        prodigiosin
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_4.svg">
        prodigiosin
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_5.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_6.svg">
        stenothricin
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_7.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_8.svg">
        prodigiosin
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_9.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_10.svg">
        sanglifehrin A
      </option>
      
    </select>
    <button id="knownclusterblast-r49c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r49c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r49c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-65"></div> <span class="help-tooltip" id="cb-sub-help-65">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r49c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r49c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r49c1_1.svg">
        AL939125_c1: prodigiosin 2-undecylpyrrole (87% of genes show similarity), 2-undecylpyrrole
      </option>
      
    </select>
    <button id="subclusterblast-r49c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r49c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r49c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-66"></div> <span class="help-tooltip" id="MIBiG-help-66">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r49c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r49c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r49c1"></div>
    <div id="comparison-MIBiG-r49c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">prodigiosin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002090.1: 255171-291481">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002090/index.html#r1c1" target="_new">BGC0002090.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.56, order:0.56, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.56, order:0.56, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.57, order:0.54, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.97</td>
        <td>Polyketide</td>
        
        <td>undecylprodigiosin, metacycloprodigiosin</td>
        
        
        <td>Streptomyces griseoaurantiacus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.92</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.88</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465.1: 4521274-4533494">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.82</td>
        <td>Other</td>
        
        <td>bromopyrroles/bromophenols</td>
        
        
        <td>Marinomonas mediterranea MMB-1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.79</td>
        <td>Polyketide</td>
        
        <td>cepacin A</td>
        
        
        <td>Burkholderia ambifaria IOP40-10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001440.1: 0-13771">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001440/index.html#r1c1" target="_new">BGC0001440.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.52, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.52, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.52, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.78</td>
        <td>Other</td>
        
        <td>cystargolide A, cystargolide B</td>
        
        
        <td>Kitasatospora cystarginea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.73</td>
        <td>Polyketide</td>
        
        <td>AF-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.73</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000065.1: 315-41590">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000065/index.html#r1c1" target="_new">BGC0000065.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.72</td>
        <td>Polyketide</td>
        
        <td>rustmicin</td>
        
        
        <td>Streptomyces galbus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002008.1: 3655965-3671216">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002008/index.html#r1c1" target="_new">BGC0002008.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.68</td>
        <td>Other</td>
        
        <td>aryl polyenes</td>
        
        
        <td>Xenorhabdus doucetiae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r49c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002090.1: 255171-291481">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002090/index.html#r1c1" target="_new">BGC0002090.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.56, order:0.56, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>undecylprodigiosin, metacycloprodigiosin</td>
      <td>Streptomyces griseoaurantiacus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.55, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465.1: 4521274-4533494">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>bromopyrroles/bromophenols</td>
      <td>Marinomonas mediterranea MMB-1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>cepacin A</td>
      <td>Burkholderia ambifaria IOP40-10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000065.1: 315-41590">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000065/index.html#r1c1" target="_new">BGC0000065.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>rustmicin</td>
      <td>Streptomyces galbus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001440.1: 0-13771">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001440/index.html#r1c1" target="_new">BGC0001440.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.52, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>cystargolide A, cystargolide B</td>
      <td>Kitasatospora cystarginea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000261.1: 375-17109">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000261/index.html#r1c1" target="_new">BGC0000261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>R1128</td>
      <td>Streptomyces sp. R1128</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>AF-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001140.1: 4627-45606">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001140/index.html#r1c1" target="_new">BGC0001140.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>platensimycin, platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r49c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-67"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-67">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r49c1-pfam-details-svg" data-filename="c00049_NODE_82.._r49c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r49c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r49c1-nrps_pks_products" data-name="r49c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r49c1-nrps_pks" data-name="r49c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r49c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-61"></div> <span class="help-tooltip" id="nrps-structures-help-61">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 41000:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r49c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r49c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r49c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-62"></div> <span class="help-tooltip" id="nrps-monomers-help-62">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds13876_15586</span></strong>: X
      <div class="collapser collapser-target-cds13876_15586 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (43..460)</strong>: X
          <div class="collapser collapser-target-cds13876_15586 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds13876_15586-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLfYLAlVcK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>iva</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds15608_23894</span></strong>: pk - mal
      <div class="collapser collapser-target-cds15608_23894 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (216..558)</strong>: pk
          <div class="collapser collapser-target-cds15608_23894 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  NH2
              <div class="collapser collapser-target-cds15608_23894-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>NH2: 124.0</dt>

  <dd></dd><dt>fatty_acid: 59.7</dt>

  <dd></dd><dt>Acetyl-CoA: 30.0</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>AHBA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1378..1678)</strong>: mal
          <div class="collapser collapser-target-cds15608_23894 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds15608_23894-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>inactive: 75.0%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds15608_23894-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 147.5</dt>

  <dd></dd><dt>inactive: 81.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 66.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 64.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 35.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 35.0</dt>

  <dd></dd><dt>Propionyl-CoA: 34.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 33.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 26.3</dt>

  <dd></dd><dt>Acetyl-CoA: 22.5</dt>

  <dd></dd><dt>CHC-CoA: 22.4</dt>

  <dd></dd><dt>fatty_acid: 19.7</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 18.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 12.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r50c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00050_NODE_9_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-408"></div> <span class="help-tooltip" id="region_svg-help-408">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00050_NODE_9_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r50c1-svg" data-filename="c00050_NODE_9_.._r50c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 90,069 - 117,092 nt. (total: 27,024 nt)
        
        <a class="cluster-rules-header" id="r50c1-rules-header" href="#r50c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r50c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r50c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-409"></div> <span class="help-tooltip" id="focus-panel-help-409">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r50c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r50c1-clusterblast" data-name="r50c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r50c1-knownclusterblast" data-name="r50c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r50c1-subclusterblast" data-name="r50c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r50c1-MIBiG-cluster-compare" data-name="r50c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r50c1-pfam-details" data-name="r50c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r50c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-68"></div> <span class="help-tooltip" id="cb-general-help-68">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r50c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r50c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r50c1_1.svg">
        NZ_SZNQ01000001 (6790867-6816723): Streptomyces lasalocidi strain X-537 Scaff... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_2.svg">
        NZ_CP023699 (1151785-1192294): Streptomyces kanamyceticus strain ATCC 12853 c... (61% of genes show similarity), CDPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_3.svg">
        NZ_CP022685 (8295330-8336075): Streptomyces formicae strain KY5 chromosome, c... (61% of genes show similarity), CDPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_4.svg">
        NC_021985 (7118020-7144778): Streptomyces collinus Tu 365, complete sequence (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_5.svg">
        NZ_CP054939 (6795065-6888848): Streptomyces sp. NA02536 chromosome, complete ... (24% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_6.svg">
        NZ_SOAR01000001 (857516-884094): Streptomyces sp. BK141 Ga0307717 101, whole ... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_7.svg">
        NZ_CP023408 (190271-284108): Streptomyces fungicidicus strain TXX3120 plasmid... (24% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_8.svg">
        NZ_CP048641 (1367335-1391690): Streptomyces aureoverticillatus strain HN6 chr... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_9.svg">
        NZ_RHGC01000006 (163870-188218): Streptomyces alboflavus strain 313 contig006... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r50c1_10.svg">
        NZ_CP015588 (1340144-1364588): Streptomyces alfalfae strain ACCC40021 chromos... (95% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r50c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r50c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r50c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-69"></div> <span class="help-tooltip" id="cb-known-help-69">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r50c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r50c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_1.svg">
        hopene
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_2.svg">
        phosphonoglycans
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_3.svg">
        xiamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_4.svg">
        xiamycin
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_5.svg">
        platencin
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_6.svg">
        platensimycin / platencin
      </option>
      
      <option value = "svg/knownclusterblast_r50c1_7.svg">
        thioplatensimycin / thioplatencin
      </option>
      
    </select>
    <button id="knownclusterblast-r50c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r50c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r50c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-70"></div> <span class="help-tooltip" id="cb-sub-help-70">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r50c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-71"></div> <span class="help-tooltip" id="MIBiG-help-71">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r50c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r50c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r50c1"></div>
    <div id="comparison-MIBiG-r50c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.63, order:0.72, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.47</td>
        <td>Terpene</td>
        
        <td>hopene</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r50c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.63, order:0.72, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Terpene</td>
      <td>hopene</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r50c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-72"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-72">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r50c1-pfam-details-svg" data-filename="c00050_NODE_9_.._r50c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r50c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r53c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00053_NODE_5_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-410"></div> <span class="help-tooltip" id="region_svg-help-410">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00053_NODE_5_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r53c1-svg" data-filename="c00053_NODE_5_.._r53c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 46,732 nt. (total: 46,732 nt)
        
        <a class="cluster-rules-header" id="r53c1-rules-header" href="#r53c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r53c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r53c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-411"></div> <span class="help-tooltip" id="focus-panel-help-411">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r53c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r53c1-nrps_pks" data-name="r53c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r53c1-clusterblast" data-name="r53c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r53c1-knownclusterblast" data-name="r53c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r53c1-subclusterblast" data-name="r53c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r53c1-MIBiG-cluster-compare" data-name="r53c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r53c1-pfam-details" data-name="r53c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r53c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-73"></div> <span class="help-tooltip" id="nrps-domain-help-73">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r53c1-details-svg" data-filename="c00053_NODE_5_.._r53c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r53c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r53c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-76"></div> <span class="help-tooltip" id="cb-general-help-76">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r53c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r53c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r53c1_1.svg">
        NZ_CP030771 (9456411-9505802): Streptomyces sp. YIM 121038 chromosome, comple... (60% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_2.svg">
        NZ_CP040916 (8875660-8924737): Streptomyces spectabilis strain NRRL 2792 chro... (60% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_3.svg">
        NZ_CP023690 (9229626-9279536): Streptomyces spectabilis strain ATCC 27465 chr... (60% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_4.svg">
        NZ_KQ948777 (222024-272956): Streptomyces griseoruber strain DSM 40281 PRJNA2... (45% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_5.svg">
        NZ_CP048641 (8337663-8404745): Streptomyces aureoverticillatus strain HN6 chr... (36% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r53c1_6.svg">
        NZ_JACHGV010000001 (180764-228858): Streptomyces paradoxus strain DSM 43350 G... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_7.svg">
        NZ_CP015098 (8045065-8095973): Streptomyces qaidamensis strain S10 chromosome... (44% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_8.svg">
        NZ_CP021978 (7819822-7868600): Streptomyces hawaiiensis strain ATCC 12236 chr... (45% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_9.svg">
        NZ_CP015588 (2477018-2527753): Streptomyces alfalfae strain ACCC40021 chromos... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r53c1_10.svg">
        NZ_CP030287 (1038032-1086713): Streptomyces sp. ICC1 chromosome (40% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r53c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r53c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r53c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-77"></div> <span class="help-tooltip" id="cb-known-help-77">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r53c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r53c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_1.svg">
        coelichelin
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_2.svg">
        scabichelin
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_3.svg">
        exochelin
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_4.svg">
        salinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_5.svg">
        albachelin
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_6.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_7.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_8.svg">
        erythrochelin
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_9.svg">
        amychelin
      </option>
      
      <option value = "svg/knownclusterblast_r53c1_10.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
    </select>
    <button id="knownclusterblast-r53c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r53c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r53c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-78"></div> <span class="help-tooltip" id="cb-sub-help-78">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r53c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-79"></div> <span class="help-tooltip" id="MIBiG-help-79">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r53c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r53c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r53c1"></div>
    <div id="comparison-MIBiG-r53c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.63, order:1.00, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>monobactam</td>
        
        
        <td>Agrobacterium tumefaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>chaetocin</td>
        
        
        <td>Collariella virescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001791.1: 5526778-5556971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001791/index.html#r1c1" target="_new">BGC0001791.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>sunshinamide</td>
        
        
        <td>Gynuella sunshinyii YC6258</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r53c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.63, order:1.00, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>monobactam</td>
      <td>Agrobacterium tumefaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>chaetocin</td>
      <td>Collariella virescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001791.1: 5526778-5556971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001791/index.html#r1c1" target="_new">BGC0001791.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>sunshinamide</td>
      <td>Gynuella sunshinyii YC6258</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r53c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-80"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-80">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r53c1-pfam-details-svg" data-filename="c00053_NODE_5_.._r53c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r53c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r53c1-nrps_pks_products" data-name="r53c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r53c1-nrps_pks" data-name="r53c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r53c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-74"></div> <span class="help-tooltip" id="nrps-structures-help-74">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 46732:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r53c1-1" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)NC(CCCN)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-orn - D-thr - orn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn,d-thr,orn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn*,d-thr*,orn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r53c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)NC(CCCN)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r53c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-75"></div> <span class="help-tooltip" id="nrps-monomers-help-75">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds15749_26732</span></strong>: orn - thr - orn
      <div class="collapser collapser-target-cds15749_26732 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,thr,orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,thr*,orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (143..541)</strong>: orn
          <div class="collapser collapser-target-cds15749_26732 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds15749_26732-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DINYWGGIGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1623..2038)</strong>: thr
          <div class="collapser collapser-target-cds15749_26732 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds15749_26732-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3067..3479)</strong>: orn
          <div class="collapser collapser-target-cds15749_26732 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds15749_26732-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMwNLGLINK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r56c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00056_NODE_18.. - Region 1 - melanin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-412"></div> <span class="help-tooltip" id="region_svg-help-412">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00056_NODE_18...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r56c1-svg" data-filename="c00056_NODE_18.._r56c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1,049 - 11,453 nt. (total: 10,405 nt)
        
        <a class="cluster-rules-header" id="r56c1-rules-header" href="#r56c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r56c1-rules">melanin: melC</div>
      
      <div class="region-svg-container">
       <div id='r56c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-413"></div> <span class="help-tooltip" id="focus-panel-help-413">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r56c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r56c1-clusterblast" data-name="r56c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r56c1-knownclusterblast" data-name="r56c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r56c1-subclusterblast" data-name="r56c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r56c1-MIBiG-cluster-compare" data-name="r56c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r56c1-pfam-details" data-name="r56c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r56c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-81"></div> <span class="help-tooltip" id="cb-general-help-81">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r56c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r56c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r56c1_1.svg">
        NZ_CP023690 (4000559-&gt;4010190): Streptomyces spectabilis strain ATCC 27465 ch... (77% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_2.svg">
        NZ_CP021748 (3556579-3565604): Streptomyces alboflavus strain MDJK44 chromoso... (54% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_3.svg">
        NZ_RHGC01000004 (127398-137839): Streptomyces alboflavus strain 313 contig004... (54% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_4.svg">
        NZ_CP030771 (3879528-3887116): Streptomyces sp. YIM 121038 chromosome, comple... (75% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_5.svg">
        NZ_CP040916 (3659924-3667602): Streptomyces spectabilis strain NRRL 2792 chro... (75% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_6.svg">
        NZ_CP048641 (5695214-5705622): Streptomyces aureoverticillatus strain HN6 chr... (41% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_7.svg">
        NZ_LWLA01000034 (206221-281420): Streptomyces sp. CB03911 scaffold4, whole ge... (4% of genes show similarity), T2PKS,melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_8.svg">
        NZ_LZRD01000027 (55217-84212): Streptomyces sp. PTY087I2 A3Q37 contig000027, ... (13% of genes show similarity), lanthipeptide,melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_9.svg">
        NZ_CP034352 (1037-312401): Streptomyces sp. W1SF4 plasmid p2, complete sequence (1% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,melanin
      </option>
      
      <option value = "svg/clusterblast_r56c1_10.svg">
        NZ_CP026304 (7642938-7687551): Streptomyces lunaelactis strain MM109 chromoso... (7% of genes show similarity), NRPS-like,melanin
      </option>
      
    </select>
    <button id= "clusterblast-r56c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r56c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r56c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-82"></div> <span class="help-tooltip" id="cb-known-help-82">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r56c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r56c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r56c1_1.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r56c1_2.svg">
        saframycin A / saframycin B
      </option>
      
    </select>
    <button id="knownclusterblast-r56c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r56c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r56c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-83"></div> <span class="help-tooltip" id="cb-sub-help-83">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r56c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-84"></div> <span class="help-tooltip" id="MIBiG-help-84">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r56c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r56c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r56c1"></div>
    <div id="comparison-MIBiG-r56c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">melanin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.52</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.56, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.49, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.48, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r56c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.56, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.49, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.48, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.48, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r56c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-85"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-85">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r56c1-pfam-details-svg" data-filename="c00056_NODE_18.._r56c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r56c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r58c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00058_NODE_85.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-414"></div> <span class="help-tooltip" id="region_svg-help-414">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00058_NODE_85...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r58c1-svg" data-filename="c00058_NODE_85.._r58c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,798 nt. (total: 27,798 nt)
        
        <a class="cluster-rules-header" id="r58c1-rules-header" href="#r58c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r58c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r58c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-415"></div> <span class="help-tooltip" id="focus-panel-help-415">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r58c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r58c1-nrps_pks" data-name="r58c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r58c1-clusterblast" data-name="r58c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r58c1-knownclusterblast" data-name="r58c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r58c1-subclusterblast" data-name="r58c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r58c1-MIBiG-cluster-compare" data-name="r58c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r58c1-pfam-details" data-name="r58c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r58c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-86"></div> <span class="help-tooltip" id="nrps-domain-help-86">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r58c1-details-svg" data-filename="c00058_NODE_85.._r58c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r58c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r58c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-89"></div> <span class="help-tooltip" id="cb-general-help-89">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r58c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r58c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r58c1_1.svg">
        NZ_CP030771 (3381610-3427927): Streptomyces sp. YIM 121038 chromosome, comple... (55% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r58c1_2.svg">
        NZ_CP023690 (3471375-3526897): Streptomyces spectabilis strain ATCC 27465 chr... (42% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r58c1_3.svg">
        NZ_CP040916 (3155110-3211334): Streptomyces spectabilis strain NRRL 2792 chro... (43% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r58c1_4.svg">
        NZ_WPNZ01000002 (401777-448434): Streptomyces sp. p1417 Scaffold2, whole geno... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r58c1_5.svg">
        NZ_CP029617 (2553402-2599318): Streptomyces sp. WAC 01529 chromosome, complet... (41% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r58c1_6.svg">
        NZ_CP058556 (5863882-5936422): Streptomyces sp. CB04723 chromosome, complete ... (13% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r58c1_7.svg">
        NZ_CP020570 (2164530-2237070): Streptomyces violaceoruber strain S21 chromoso... (13% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r58c1_8.svg">
        NZ_WWGL01000034 (641695-714235): Streptomyces sp. SID8369 SID8369.c34, whole ... (12% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r58c1_9.svg">
        NZ_MTKC01000003 (920474-941399): Streptomyces griseofuscus strain NG1-7 NG1-1... (38% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r58c1_10.svg">
        NZ_CP028719 (3007229-3029365): Streptomyces sp. endophyte N2 chromosome, comp... (35% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r58c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r58c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r58c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-90"></div> <span class="help-tooltip" id="cb-known-help-90">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r58c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-91"></div> <span class="help-tooltip" id="cb-sub-help-91">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r58c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-92"></div> <span class="help-tooltip" id="MIBiG-help-92">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r58c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r58c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r58c1"></div>
    <div id="comparison-MIBiG-r58c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>salinichelins</td>
        
        
        <td>Salinispora pacifica CNY331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>erythrochelin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r58c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>salinichelins</td>
      <td>Salinispora pacifica CNY331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>erythrochelin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r58c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-93"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-93">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r58c1-pfam-details-svg" data-filename="c00058_NODE_85.._r58c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r58c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r58c1-nrps_pks_products" data-name="r58c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r58c1-nrps_pks" data-name="r58c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r58c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-87"></div> <span class="help-tooltip" id="nrps-structures-help-87">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 27798:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r58c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-leu - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-leu,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-leu*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r58c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r58c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-88"></div> <span class="help-tooltip" id="nrps-monomers-help-88">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1072_7798</span></strong>: leu - X
      <div class="collapser collapser-target-cds1072_7798 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (22..422)</strong>: leu
          <div class="collapser collapser-target-cds1072_7798 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds1072_7798-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DGFFwGacFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1644..2046)</strong>: X
          <div class="collapser collapser-target-cds1072_7798 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds1072_7798-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYmWGamaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r61c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00061_NODE_20.. - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-416"></div> <span class="help-tooltip" id="region_svg-help-416">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00061_NODE_20...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r61c1-svg" data-filename="c00061_NODE_20.._r61c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 21,176 - 95,463 nt. (total: 74,288 nt)
        
        <a class="cluster-rules-header" id="r61c1-rules-header" href="#r61c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r61c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r61c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-417"></div> <span class="help-tooltip" id="focus-panel-help-417">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r61c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r61c1-nrps_pks" data-name="r61c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r61c1-clusterblast" data-name="r61c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r61c1-knownclusterblast" data-name="r61c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r61c1-subclusterblast" data-name="r61c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r61c1-MIBiG-cluster-compare" data-name="r61c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r61c1-pfam-details" data-name="r61c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r61c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-94"></div> <span class="help-tooltip" id="nrps-domain-help-94">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r61c1-details-svg" data-filename="c00061_NODE_20.._r61c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r61c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r61c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-97"></div> <span class="help-tooltip" id="cb-general-help-97">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r61c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r61c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r61c1_1.svg">
        NZ_CP031742 (1224927-1286185): Streptomyces koyangensis strain VK-A60T chromo... (34% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_2.svg">
        NZ_JABBGE010000004 (88987-167757): Streptomyces sp. R302 contig4, whole genom... (23% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r61c1_3.svg">
        NZ_ML123034 (5519499-5567981): Streptomyces sp. ADI96-02 scaffold001, whole g... (39% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_4.svg">
        NZ_LIWB01000001 (460652-508119): Streptomyces sp. CB02400 scaffold1, whole ge... (33% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_5.svg">
        NZ_CP023700 (6274029-6321403): Streptomyces viridosporus T7A strain ATCC 3911... (31% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_6.svg">
        NZ_JH993790 (199560-245412): Streptomyces viridosporus T7A StrvirDRAFT SVC.4,... (31% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_7.svg">
        NZ_AZUS01000007 (166746-214129): Streptomyces sp. CNH189 B100DRAFT scaffold 6... (34% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_8.svg">
        NZ_DS999641 (1827869-1875244): Streptomyces viridosporus ATCC 14672 supercont... (31% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r61c1_9.svg">
        NC_016582 (4052518-4099778): Streptomyces bingchenggensis BCW-1, complete seq... (28% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r61c1_10.svg">
        NZ_CP024985 (2348052-2396143): Streptomyces lavendulae subsp. lavendulae stra... (27% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r61c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r61c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r61c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-98"></div> <span class="help-tooltip" id="cb-known-help-98">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r61c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r61c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r61c1_1.svg">
        enduracidin
      </option>
      
      <option value = "svg/knownclusterblast_r61c1_2.svg">
        lorneic acid A
      </option>
      
    </select>
    <button id="knownclusterblast-r61c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r61c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r61c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-99"></div> <span class="help-tooltip" id="cb-sub-help-99">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r61c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-100"></div> <span class="help-tooltip" id="MIBiG-help-100">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r61c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r61c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r61c1"></div>
    <div id="comparison-MIBiG-r61c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.90</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001023.1: 0-29111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001023/index.html#r1c1" target="_new">BGC0001023.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:0.55, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.45, order:0.83, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.83, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.82</td>
        <td>NRP, Polyketide</td>
        
        <td>myxochromide S1</td>
        
        
        <td>Stigmatella aurantiaca</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.80</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.80</td>
        <td>NRP</td>
        
        <td>beauvericin</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.80</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.79</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.78</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.75, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.78</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens 3L</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.75, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.78</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens JHB</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.75, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.78</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea bouillonii PNG</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r61c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001023.1: 0-29111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001023/index.html#r1c1" target="_new">BGC0001023.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.45, order:0.83, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Polyketide</td>
      <td>myxochromide S1</td>
      <td>Stigmatella aurantiaca</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058.1: 29690-42292">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>Reutericyclin A, Reutericyclin B, Reutericyclin C, Reutericyclin D</td>
      <td>Streptococcus mutans B04Sm5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252.1: 0-7958">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>UNII-YC2Q1O94PT</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606.1: 2066368-2088912">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>gibepyrone-A</td>
      <td>Fusarium fujikuroi IMI 58289</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.75, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.75, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r61c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-101"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-101">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r61c1-pfam-details-svg" data-filename="c00061_NODE_20.._r61c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r61c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r61c1-nrps_pks_products" data-name="r61c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r61c1-nrps_pks" data-name="r61c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r61c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-95"></div> <span class="help-tooltip" id="nrps-structures-help-95">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 21175 - 95463:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r61c1-1" class="smiles-canvas" data-smiles="CC(O)NC([*])C(=O)NC(CC(=O)N)C(=O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal) + (X) + (asn) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal&amp;nrps2=x&amp;nrps3=asn&amp;nrps4=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*&amp;nrps2=x&amp;nrps3=asn*&amp;nrps4=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r61c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)NC([*])C(=O)NC(CC(=O)N)C(=O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 21175 - 66192:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r61c1-2" class="smiles-canvas" data-smiles="NC(CC(=O)N)C(=O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(asn) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asn&amp;nrps2=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asn*&amp;nrps2=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r61c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(=O)N)C(=O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 39238 - 89742:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r61c1-3" class="smiles-canvas" data-smiles="CC(O)NC([*])C(=O)NC(CC(=O)N)C(=O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal) + (X) + (asn) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal&amp;nrps2=x&amp;nrps3=asn&amp;nrps4=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*&amp;nrps2=x&amp;nrps3=asn*&amp;nrps4=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r61c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)NC([*])C(=O)NC(CC(=O)N)C(=O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 49781 - 95463:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r61c1-4" class="smiles-canvas" data-smiles="CC(O)NC([*])C(=O)NC(CC(=O)N)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal) + (X) + (asn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal&amp;nrps2=x&amp;nrps3=asn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*&amp;nrps2=x&amp;nrps3=asn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r61c1-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)NC([*])C(=O)NC(CC(=O)N)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r61c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-96"></div> <span class="help-tooltip" id="nrps-monomers-help-96">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds40412_48158</span></strong>: pk
      <div class="collapser collapser-target-cds40412_48158 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (543..841)</strong>: pk
          <div class="collapser collapser-target-cds40412_48158 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds40412_48158-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>CE-Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds40412_48158-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 93.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 86.3</dt>

  <dd></dd><dt>Malonyl-CoA: 73.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 56.9</dt>

  <dd></dd><dt>Propionyl-CoA: 50.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 46.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 42.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 31.4</dt>

  <dd></dd><dt>CHC-CoA: 29.7</dt>

  <dd></dd><dt>fatty_acid: 25.3</dt>

  <dd></dd><dt>inactive: 17.3</dt>

  <dd></dd><dt>Acetyl-CoA: 16.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 13.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 6.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds59238_64680</span></strong>: pk - asn
      <div class="collapser collapser-target-cds59238_64680 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>CAL_domain (52..498)</strong>: pk
          <div class="collapser collapser-target-cds59238_64680 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  Acetyl-CoA
              <div class="collapser collapser-target-cds59238_64680-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Acetyl-CoA: 34.7</dt>

  <dd></dd><dt>AHBA: 15.6</dt>

  <dd></dd><dt>NH2: 9.5</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>fatty_acid: -3.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1206..1605)</strong>: asn
          <div class="collapser collapser-target-cds59238_64680 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds59238_64680-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DATKVGEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds66253_69742</span></strong>: X
      <div class="collapser collapser-target-cds66253_69742 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (509..905)</strong>: X
          <div class="collapser collapser-target-cds66253_69742 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds66253_69742-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DSaLiAEVwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>his</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds69781_75463</span></strong>: mal
      <div class="collapser collapser-target-cds69781_75463 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (537..833)</strong>: mal
          <div class="collapser collapser-target-cds69781_75463 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds69781_75463-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds69781_75463-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 138.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 77.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 75.7</dt>

  <dd></dd><dt>inactive: 66.8</dt>

  <dd></dd><dt>Propionyl-CoA: 53.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 41.4</dt>

  <dd></dd><dt>fatty_acid: 40.3</dt>

  <dd></dd><dt>CHC-CoA: 39.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 32.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 20.4</dt>

  <dd></dd><dt>Acetyl-CoA: 18.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 11.3</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 7.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r70c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00070_NODE_10.. - Region 1 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-418"></div> <span class="help-tooltip" id="region_svg-help-418">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00070_NODE_10...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r70c1-svg" data-filename="c00070_NODE_10.._r70c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 33,038 nt. (total: 33,038 nt)
        
        <a class="cluster-rules-header" id="r70c1-rules-header" href="#r70c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r70c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r70c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-419"></div> <span class="help-tooltip" id="focus-panel-help-419">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r70c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r70c1-nrps_pks" data-name="r70c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r70c1-clusterblast" data-name="r70c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r70c1-knownclusterblast" data-name="r70c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r70c1-subclusterblast" data-name="r70c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r70c1-MIBiG-cluster-compare" data-name="r70c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r70c1-pfam-details" data-name="r70c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r70c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-102"></div> <span class="help-tooltip" id="nrps-domain-help-102">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r70c1-details-svg" data-filename="c00070_NODE_10.._r70c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r70c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r70c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-105"></div> <span class="help-tooltip" id="cb-general-help-105">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r70c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r70c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r70c1_1.svg">
        NZ_PVTF01000005 (360532-527607): Umezawaea tangerina strain DSM 44720 Ga01809... (11% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r70c1_2.svg">
        NZ_KZ819182 (237576-378380): Streptomyces sp. CS147 scaffold00001, whole geno... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r70c1_3.svg">
        NZ_JACHJG010000009 (132384-208075): Streptomyces netropsis strain CECT 3265 G... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r70c1_4.svg">
        NZ_KB894403 (49036-178532): Actinokineospora enzanensis DSM 44649 C503DRAFT s... (12% of genes show similarity), LAP,NRPS
      </option>
      
      <option value = "svg/clusterblast_r70c1_5.svg">
        NZ_CP056041 (139180-208432): Streptomyces chartreusis strain NA02069 chromoso... (13% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r70c1_6.svg">
        NZ_LIXJ01000001 (186219-276128): Streptomyces sp. CB01580 scaffold1, whole ge... (15% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r70c1_7.svg">
        NZ_CP029617 (71628-143380): Streptomyces sp. WAC 01529 chromosome, complete g... (14% of genes show similarity), NRPS,lassopeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r70c1_8.svg">
        NZ_CP048397 (7441299-7554123): Streptomyces sp. S4.7 chromosome, complete genome (15% of genes show similarity), NRPS,linaridin,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r70c1_9.svg">
        NZ_CP018047 (239407-353926): Streptomyces niveus strain SCSIO 3406 chromosome... (16% of genes show similarity), NRPS,linaridin,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r70c1_10.svg">
        NZ_CP026692 (302346-394968): Nostoc sp. &#39;Lobaria pulmonaria (5183) cyanobiont... (17% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r70c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r70c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r70c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-106"></div> <span class="help-tooltip" id="cb-known-help-106">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r70c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r70c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_1.svg">
        enduracidin
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_2.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_3.svg">
        actinomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_4.svg">
        WS9326
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_5.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_6.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_7.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_8.svg">
        polyoxypeptin
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_9.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r70c1_10.svg">
        triostin A
      </option>
      
    </select>
    <button id="knownclusterblast-r70c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r70c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r70c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-107"></div> <span class="help-tooltip" id="cb-sub-help-107">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r70c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r70c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r70c1_1.svg">
        AF235504_c2: FK520 methoxymalonyl-ACP (66% of genes show similarity), methoxymalonyl-ACP
      </option>
      
    </select>
    <button id="subclusterblast-r70c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r70c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r70c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-108"></div> <span class="help-tooltip" id="MIBiG-help-108">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r70c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r70c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r70c1"></div>
    <div id="comparison-MIBiG-r70c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.64</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.62</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.62</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.60</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.59</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:0.68, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:0.68, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r70c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r70c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-109"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-109">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r70c1-pfam-details-svg" data-filename="c00070_NODE_10.._r70c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r70c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r70c1-nrps_pks_products" data-name="r70c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r70c1-nrps_pks" data-name="r70c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r70c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-103"></div> <span class="help-tooltip" id="nrps-structures-help-103">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 33038:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r70c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC(C(C)C)C(=O)CC(=O)NC([*])C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - X - val) + (mal - X) + (X) + (thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,x,val&amp;nrps2=mal,x&amp;nrps3=x&amp;nrps4=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,x,val*&amp;nrps2=mal*,x&amp;nrps3=x&amp;nrps4=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r70c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC(C(C)C)C(=O)CC(=O)NC([*])C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 95 - 16880:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r70c1-2" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(thr) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r70c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r70c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-104"></div> <span class="help-tooltip" id="nrps-monomers-help-104">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds95_2960</span></strong>: thr
      <div class="collapser collapser-target-cds95_2960 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (372..769)</strong>: thr
          <div class="collapser collapser-target-cds95_2960 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds95_2960-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds5851_9661</span></strong>: X
      <div class="collapser collapser-target-cds5851_9661 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (467..840)</strong>: X
          <div class="collapser collapser-target-cds5851_9661 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds5851_9661-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAthLGVTwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds16880_23948</span></strong>: X
      <div class="collapser collapser-target-cds16880_23948 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1317..1715)</strong>: X
          <div class="collapser collapser-target-cds16880_23948 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds16880_23948-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMLcvGiVcK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser-thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds23944_33037</span></strong>: X - X - val
      <div class="collapser collapser-target-cds23944_33037 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X,val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*,val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (9..224)</strong>: X
          <div class="collapser collapser-target-cds23944_33037 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp
              <div class="collapser collapser-target-cds23944_33037-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAfTtgGVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1387..1776)</strong>: X
          <div class="collapser collapser-target-cds23944_33037 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds23944_33037-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvFsVAAyaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2432..2831)</strong>: val
          <div class="collapser collapser-target-cds23944_33037 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds23944_33037-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DiyFLGgVFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r73c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00073_NODE_16.. - Region 1 - NRPS,NRPS-like,T1PKS,T3PKS,betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-420"></div> <span class="help-tooltip" id="region_svg-help-420">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00073_NODE_16...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r73c1-svg" data-filename="c00073_NODE_16.._r73c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 82,739 nt. (total: 82,739 nt)
        
        <a class="cluster-rules-header" id="r73c1-rules-header" href="#r73c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r73c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>betalactone: (HMGL-like and AMP-binding)<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r73c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-421"></div> <span class="help-tooltip" id="focus-panel-help-421">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r73c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r73c1-nrps_pks" data-name="r73c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r73c1-clusterblast" data-name="r73c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r73c1-knownclusterblast" data-name="r73c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r73c1-subclusterblast" data-name="r73c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r73c1-MIBiG-cluster-compare" data-name="r73c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r73c1-pfam-details" data-name="r73c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r73c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-110"></div> <span class="help-tooltip" id="nrps-domain-help-110">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r73c1-details-svg" data-filename="c00073_NODE_16.._r73c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r73c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r73c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-113"></div> <span class="help-tooltip" id="cb-general-help-113">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r73c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r73c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r73c1_1.svg">
        NZ_KZ819159 (36962-127179): Streptomyces sp. CS090A Scaffold00001, whole geno... (16% of genes show similarity), NRPS,T3PKS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_2.svg">
        NZ_WWIF01000005 (66211-156815): Streptomyces sp. SID5594 SID5594.c5, whole ge... (17% of genes show similarity), NRPS,T3PKS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_3.svg">
        NZ_CP009438 (194524-235255): Streptomyces glaucescens strain GLA.O chromosome... (40% of genes show similarity), NRPS-like,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_4.svg">
        NZ_CP047144 (164722-205840): Streptomyces sp. HF10 chromosome, complete genome (25% of genes show similarity), NRPS-like,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_5.svg">
        NZ_SMSN01000037 (56450-98277): Streptomyces lavendulae strain YAKB-15 scaffol... (25% of genes show similarity), NRPS-like,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_6.svg">
        NZ_CP045740 (8034118-8075903): Streptomyces sp. SUK 48 chromosome, complete g... (26% of genes show similarity), NRPS-like,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_7.svg">
        NZ_NNBJ01000022 (35665-77468): Streptomyces sp. CB01201 scaffold22, whole gen... (25% of genes show similarity), NRPS-like,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_8.svg">
        NZ_CP022752 (2990870-3085953): Actinopolyspora erythraea strain YIM 90600 chr... (20% of genes show similarity), NRPS-like,T1PKS,betalactone,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r73c1_9.svg">
        NZ_CP023865 (2121650-2173443): Actinoplanes teichomyceticus ATCC 31121 chromo... (25% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r73c1_10.svg">
        NZ_FPAT01000001 (599450-681015): Actinopolyspora righensis strain DSM 45501, ... (16% of genes show similarity), NRPS-like,betalactone,hglE-KS
      </option>
      
    </select>
    <button id= "clusterblast-r73c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r73c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r73c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-114"></div> <span class="help-tooltip" id="cb-known-help-114">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r73c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r73c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_1.svg">
        jawsamycin
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_2.svg">
        flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_3.svg">
        flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_4.svg">
        napyradiomycin
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_5.svg">
        napyradiomycin A80915C
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_6.svg">
        merochlorin A / merochlorin B / deschloro-merochlorin A / deschloro-merochlorin B / isochloro-merochlorin B / dichloro-merochlorin B / merochlorin D / merochlorin C
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_7.svg">
        furaquinocin B
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_8.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_9.svg">
        lipstatin
      </option>
      
      <option value = "svg/knownclusterblast_r73c1_10.svg">
        ansamitocin P-3
      </option>
      
    </select>
    <button id="knownclusterblast-r73c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r73c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r73c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-115"></div> <span class="help-tooltip" id="cb-sub-help-115">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r73c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-116"></div> <span class="help-tooltip" id="MIBiG-help-116">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r73c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r73c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r73c1"></div>
    <div id="comparison-MIBiG-r73c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">betalactone</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>1.29</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>1.29</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.50, order:0.65, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.58, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>1.22</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>1.14</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>1.11</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000254.1: 133-22098">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000254/index.html#r1c1" target="_new">BGC0000254.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>1.08</td>
        <td>Polyketide</td>
        
        <td>oxytetracycline</td>
        
        
        <td>Streptomyces rimosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145.1: 1-32637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>1.05</td>
        <td>Polyketide</td>
        
        <td>salinosporamide A</td>
        
        
        <td>Salinispora tropica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001041.1: 1140546-1182556">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001041/index.html#r1c1" target="_new">BGC0001041.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>1.04</td>
        <td>NRP, Polyketide</td>
        
        <td>salinosporamide A</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>1.03</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>1.03</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r73c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001041.1: 1140546-1182556">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001041/index.html#r1c1" target="_new">BGC0001041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Polyketide</td>
      <td>salinosporamide A</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145.1: 1-32637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>salinosporamide A</td>
      <td>Salinispora tropica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921.1: 4334-20040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other, Polyketide, NRP,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>promysalin</td>
      <td>Pseudomonas putida</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r73c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-117"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-117">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r73c1-pfam-details-svg" data-filename="c00073_NODE_16.._r73c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r73c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r73c1-nrps_pks_products" data-name="r73c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r73c1-nrps_pks" data-name="r73c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r73c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-111"></div> <span class="help-tooltip" id="nrps-structures-help-111">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 82739:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r73c1-1" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (X) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=x&amp;nrps3=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=x&amp;nrps3=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r73c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 0 - 25980:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r73c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r73c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 14288 - 56136:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r73c1-3" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r73c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 24295 - 68264:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r73c1-4" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r73c1-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 41542 - 82739:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r73c1-5" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r73c1-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r73c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-112"></div> <span class="help-tooltip" id="nrps-monomers-help-112">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1_5980</span></strong>: X
      <div class="collapser collapser-target-cds1_5980 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (468..856)</strong>: X
          <div class="collapser collapser-target-cds1_5980 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds1_5980-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSvAsfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds30982_32509</span></strong>: X
      <div class="collapser collapser-target-cds30982_32509 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (52..411)</strong>: X
          <div class="collapser collapser-target-cds30982_32509 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds30982_32509-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DgFLfGLfaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds34288_36136</span></strong>: X
      <div class="collapser collapser-target-cds34288_36136 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (15..439)</strong>: X
          <div class="collapser collapser-target-cds34288_36136 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds34288_36136-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DgTKVGaVcK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds44295_48264</span></strong>: mal
      <div class="collapser collapser-target-cds44295_48264 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (534..826)</strong>: mal
          <div class="collapser collapser-target-cds44295_48264 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds44295_48264-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>inactive: 79.2%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds44295_48264-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 163.2</dt>

  <dd></dd><dt>inactive: 97.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 75.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 71.4</dt>

  <dd></dd><dt>Propionyl-CoA: 53.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 32.4</dt>

  <dd></dd><dt>CHC-CoA: 28.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 21.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 19.6</dt>

  <dd></dd><dt>Acetyl-CoA: 18.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 16.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 16.8</dt>

  <dd></dd><dt>fatty_acid: 16.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 7.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r77c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00077_NODE_73.. - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-422"></div> <span class="help-tooltip" id="region_svg-help-422">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00077_NODE_73...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r77c1-svg" data-filename="c00077_NODE_73.._r77c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 23,132 nt. (total: 23,132 nt)
        
        <a class="cluster-rules-header" id="r77c1-rules-header" href="#r77c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r77c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r77c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-423"></div> <span class="help-tooltip" id="focus-panel-help-423">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r77c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r77c1-clusterblast" data-name="r77c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r77c1-knownclusterblast" data-name="r77c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r77c1-subclusterblast" data-name="r77c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r77c1-MIBiG-cluster-compare" data-name="r77c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r77c1-pfam-details" data-name="r77c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r77c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-118"></div> <span class="help-tooltip" id="cb-general-help-118">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r77c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r77c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r77c1_1.svg">
        NZ_CP023976 (45536-86634): Streptomyces alboflavus strain MDJK44 plasmid pMDJ... (11% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r77c1_2.svg">
        NZ_CP023699 (10045095-10128068): Streptomyces kanamyceticus strain ATCC 12853... (8% of genes show similarity), NRPS-like,T3PKS,lassopeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r77c1_3.svg">
        NZ_CP023690 (9707265-9798331): Streptomyces spectabilis strain ATCC 27465 chr... (4% of genes show similarity), T1PKS,T3PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r77c1_4.svg">
        NZ_CP040916 (9346846-9416805): Streptomyces spectabilis strain NRRL 2792 chro... (3% of genes show similarity), T3PKS,butyrolactone,terpene
      </option>
      
      <option value = "svg/clusterblast_r77c1_5.svg">
        NZ_CP031455 (1966080-2004745): Streptomyces olivoreticuli subsp. olivoreticul... (5% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r77c1_6.svg">
        NZ_JACHJG010000012 (14037-55093): Streptomyces netropsis strain CECT 3265 Ga0... (5% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r77c1_7.svg">
        NZ_JTHL01000001 (8508115-8548360): Streptomyces sp. 150FB scaffold00001, whol... (5% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r77c1_8.svg">
        NZ_KI547049 (152955-194020): Streptomycetaceae bacterium MP113-05 scaffold000... (5% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r77c1_9.svg">
        NZ_KE384522 (254-57920): Streptomyces sp. CNS606 K304DRAFT scaffold00002.2, w... (5% of genes show similarity), NRPS,T1PKS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r77c1_10.svg">
        NZ_NNBL01000003 (728023-769064): Streptomyces sp. CB01635 scaffold3, whole ge... (4% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r77c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r77c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r77c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-119"></div> <span class="help-tooltip" id="cb-known-help-119">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r77c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-120"></div> <span class="help-tooltip" id="cb-sub-help-120">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r77c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-121"></div> <span class="help-tooltip" id="MIBiG-help-121">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r77c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r77c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r77c1"></div>
    <div id="comparison-MIBiG-r77c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001454.1: 0-1185">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001454/index.html#r1c1" target="_new">BGC0001454.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>germicidin</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001905.1: 0-3803">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001905/index.html#r1c1" target="_new">BGC0001905.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>violapyrone B</td>
        
        
        <td>Streptomyces somaliensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>alkylresorcinol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
        <td>0.10</td>
        <td>Terpene, Polyketide</td>
        
        <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
        
        
        <td>Actinoplanes missouriensis 431</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r77c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001454.1: 0-1185">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001454/index.html#r1c1" target="_new">BGC0001454.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>germicidin</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001905.1: 0-3803">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001905/index.html#r1c1" target="_new">BGC0001905.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>violapyrone B</td>
      <td>Streptomyces somaliensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>alkylresorcinol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Polyketide</td>
      <td>alkylpyrone-407, alkylpyrone-393</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
      <td>0.35</td>
      <td>Terpene, Polyketide</td>
      <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
      <td>Actinoplanes missouriensis 431</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r77c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-122"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-122">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r77c1-pfam-details-svg" data-filename="c00077_NODE_73.._r77c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r77c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r81c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00081_NODE_20.. - Region 1 - other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-424"></div> <span class="help-tooltip" id="region_svg-help-424">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00081_NODE_20...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r81c1-svg" data-filename="c00081_NODE_20.._r81c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,409 nt. (total: 6,409 nt)
        
        <a class="cluster-rules-header" id="r81c1-rules-header" href="#r81c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r81c1-rules">other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r81c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-425"></div> <span class="help-tooltip" id="focus-panel-help-425">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r81c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r81c1-clusterblast" data-name="r81c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r81c1-knownclusterblast" data-name="r81c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r81c1-subclusterblast" data-name="r81c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r81c1-MIBiG-cluster-compare" data-name="r81c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r81c1-pfam-details" data-name="r81c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r81c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-123"></div> <span class="help-tooltip" id="cb-general-help-123">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r81c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r81c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r81c1_1.svg">
        NZ_CP023690 (7874786-7942911): Streptomyces spectabilis strain ATCC 27465 chr... (9% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_2.svg">
        NZ_CP040916 (7561982-7630130): Streptomyces spectabilis strain NRRL 2792 chro... (9% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_3.svg">
        NZ_CP030771 (7834424-7901028): Streptomyces sp. YIM 121038 chromosome, comple... (9% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_4.svg">
        NZ_RHGC01000006 (59286-149148): Streptomyces alboflavus strain 313 contig006,... (9% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_5.svg">
        NZ_WPNZ01000004 (400429-487232): Streptomyces sp. p1417 Scaffold4, whole geno... (9% of genes show similarity), NRPS-like,T1PKS,hglE-KS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_6.svg">
        NZ_CP021748 (8069033-&gt;8156625): Streptomyces alboflavus strain MDJK44 chromos... (9% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_7.svg">
        NZ_CP048641 (1255717-1355003): Streptomyces aureoverticillatus strain HN6 chr... (9% of genes show similarity), NRPS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_8.svg">
        NZ_PTJS01000001 (6280945-6362003): Streptomyces sp. QL37 scaffold1 size877290... (7% of genes show similarity), NRPS,betalactone,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_9.svg">
        NZ_CP016279 (2486499-2611542): Streptomyces griseochromogenes strain ATCC 145... (4% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r81c1_10.svg">
        NZ_CP042266 (6918091-7295107): Streptomyces qinzhouensis strain SSL-25 chromo... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,other,terpene,transAT-PKS
      </option>
      
    </select>
    <button id= "clusterblast-r81c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r81c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r81c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-124"></div> <span class="help-tooltip" id="cb-known-help-124">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r81c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r81c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r81c1_1.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r81c1_2.svg">
        kanglemycin A / kanglemycin V1 / kanglemycin V2
      </option>
      
      <option value = "svg/knownclusterblast_r81c1_3.svg">
        rifamorpholine A / rifamorpholine B / rifamorpholine C / rifamorpholine D / rifamorpholine E
      </option>
      
      <option value = "svg/knownclusterblast_r81c1_4.svg">
        rifamycin
      </option>
      
      <option value = "svg/knownclusterblast_r81c1_5.svg">
        rifamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r81c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r81c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r81c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-125"></div> <span class="help-tooltip" id="cb-sub-help-125">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r81c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-126"></div> <span class="help-tooltip" id="MIBiG-help-126">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r81c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r81c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r81c1"></div>
    <div id="comparison-MIBiG-r81c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">other</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
        
        
        <td>Streptomyces sp. F001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene, Alkaloid</td>
        
        <td>xiamycin</td>
        
        
        <td>Streptomyces sp. HKI0576</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene</td>
        
        <td>xiamycin A</td>
        
        
        <td>Streptomyces sp. SCSIO 02999</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>nybomycin</td>
        
        
        <td>Streptomyces albus subsp. chlorinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001534.1: 799-5264">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001534/index.html#r1c1" target="_new">BGC0001534.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>branched-chain fatty acids</td>
        
        
        <td>Streptomyces filamentosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361.1: 2257657-2262237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
         
        
        <td>0.11</td>
        <td>Terpene</td>
        
        <td>sodorifen</td>
        
        
        <td>Serratia plymuthica 4Rx13</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651.1: 740-35007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
         
        
        <td>0.07</td>
        <td>Terpene</td>
        
        <td>KS-505a</td>
        
        
        <td>Streptomyces argenteolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.50, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>Other</td>
        
        <td>lincomycin</td>
        
        
        <td>Streptomyces lincolnensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r81c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
      <td>Streptomyces sp. F001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Terpene, Alkaloid</td>
      <td>xiamycin</td>
      <td>Streptomyces sp. HKI0576</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Terpene</td>
      <td>xiamycin A</td>
      <td>Streptomyces sp. SCSIO 02999</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>nybomycin</td>
      <td>Streptomyces albus subsp. chlorinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001534.1: 799-5264">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001534/index.html#r1c1" target="_new">BGC0001534.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Other</td>
      <td>branched-chain fatty acids</td>
      <td>Streptomyces filamentosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361.1: 2257657-2262237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
      <td>0.37</td>
      <td>Terpene</td>
      <td>sodorifen</td>
      <td>Serratia plymuthica 4Rx13</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651.1: 740-35007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
      <td>0.28</td>
      <td>Terpene</td>
      <td>KS-505a</td>
      <td>Streptomyces argenteolus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.50, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>Other</td>
      <td>lincomycin</td>
      <td>Streptomyces lincolnensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r81c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-127"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-127">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r81c1-pfam-details-svg" data-filename="c00081_NODE_20.._r81c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r81c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r84c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00084_NODE_8_.. - Region 1 - NRPS,NRPS-like,ladderane <div class="help-container"> <div class="help-icon" data-id="region_svg-help-426"></div> <span class="help-tooltip" id="region_svg-help-426">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00084_NODE_8_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r84c1-svg" data-filename="c00084_NODE_8_.._r84c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 2 - 65,431 nt. (total: 65,430 nt)
        
        <a class="cluster-rules-header" id="r84c1-rules-header" href="#r84c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r84c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>ladderane: ladderane</div>
      
      <div class="region-svg-container">
       <div id='r84c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-427"></div> <span class="help-tooltip" id="focus-panel-help-427">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r84c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r84c1-nrps_pks" data-name="r84c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r84c1-clusterblast" data-name="r84c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r84c1-knownclusterblast" data-name="r84c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r84c1-subclusterblast" data-name="r84c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r84c1-MIBiG-cluster-compare" data-name="r84c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r84c1-pfam-details" data-name="r84c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r84c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-128"></div> <span class="help-tooltip" id="nrps-domain-help-128">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r84c1-details-svg" data-filename="c00084_NODE_8_.._r84c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r84c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r84c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-131"></div> <span class="help-tooltip" id="cb-general-help-131">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r84c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r84c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r84c1_1.svg">
        NZ_CP048641 (5809936-5896435): Streptomyces aureoverticillatus strain HN6 chr... (70% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r84c1_2.svg">
        NZ_CP023701 (3767226-3853956): Streptomyces subrutilus strain ATCC 27467 chro... (52% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r84c1_3.svg">
        NZ_KQ948457 (24024-111107): Streptomyces olivochromogenes strain DSM 40451 PR... (50% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r84c1_4.svg">
        NZ_NNBN01000002 (486326-584750): Streptomyces sp. CB02613 scaffold2, whole ge... (43% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r84c1_5.svg">
        NZ_KZ819182 (4724034-4779057): Streptomyces sp. CS147 scaffold00001, whole ge... (70% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r84c1_6.svg">
        NZ_FNHV01000006 (458279-528382): Streptomyces sp. cf386, whole genome shotgun... (56% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r84c1_7.svg">
        NZ_CP029195 (1401410-1576477): Streptomyces venezuelae strain ATCC 10595 chro... (26% of genes show similarity), NRPS,NRPS-like,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r84c1_8.svg">
        NZ_CP029196 (6369955-6503494): Streptomyces venezuelae strain ATCC 21113 chro... (29% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r84c1_9.svg">
        NC_018750 (6721250-6855167): Streptomyces venezuelae ATCC 10712, complete genome (29% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r84c1_10.svg">
        NZ_CP032698 (3157450-3248693): Streptomyces hundungensis strain BH38 chromoso... (33% of genes show similarity), NRPS,bacteriocin,ladderane,thiopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r84c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r84c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r84c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-132"></div> <span class="help-tooltip" id="cb-known-help-132">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r84c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r84c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_1.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_2.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_3.svg">
        ulleungmycin
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_4.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_5.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_6.svg">
        pepticinnamin E
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_7.svg">
        WS9326
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_8.svg">
        malacidin A / malacidin B
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_9.svg">
        ishigamide
      </option>
      
      <option value = "svg/knownclusterblast_r84c1_10.svg">
        acyldepsipeptide 1
      </option>
      
    </select>
    <button id="knownclusterblast-r84c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r84c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r84c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-133"></div> <span class="help-tooltip" id="cb-sub-help-133">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r84c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r84c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r84c1_1.svg">
        NC_009380_c1: salinosporamide cyclohexenylalanine (50% of genes show similarity), cyclohexenylalanine
      </option>
      
    </select>
    <button id="subclusterblast-r84c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r84c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r84c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-134"></div> <span class="help-tooltip" id="MIBiG-help-134">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r84c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r84c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r84c1"></div>
    <div id="comparison-MIBiG-r84c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">ladderane</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.83</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.83</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000371.1: 2888227-2910756">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000371/index.html#r1c1" target="_new">BGC0000371.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.75</td>
        <td>NRP</td>
        
        <td>heterobactin A, heterobactin S2</td>
        
        
        <td>Rhodococcus erythropolis PR4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.72</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.69</td>
        <td>Terpene</td>
        
        <td>eremophilene</td>
        
        
        <td>Sorangium cellulosum So ce56</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001688.1: 66-30223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001688/index.html#r1c1" target="_new">BGC0001688.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.68</td>
        <td>Other</td>
        
        <td>N-tetradecanoyl tyrosine</td>
        
        
        <td>uncultured bacterium CSLC2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591.1: 66-30223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.68</td>
        <td>Other</td>
        
        <td>fatty acid enol ester</td>
        
        
        <td>uncultured bacterium CSLC2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000340.1: 200-18539">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000340/index.html#r1c1" target="_new">BGC0000340.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.47, order:0.37, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>echoside A, echoside B, echoside C, echoside D, echoside E</td>
        
        
        <td>Streptomyces sp. LZ35</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r84c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.57, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>esmeraldin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.54, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000371.1: 2888227-2910756">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000371/index.html#r1c1" target="_new">BGC0000371.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>heterobactin A, heterobactin S2</td>
      <td>Rhodococcus erythropolis PR4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002019.1: 0-24752">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002019/index.html#r1c1" target="_new">BGC0002019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.54, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>tiancilactone</td>
      <td>Streptomyces sp. CB03234</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene, Polyketide</td>
      <td>aurachin RE</td>
      <td>Rhodococcus erythropolis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r84c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-135"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-135">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r84c1-pfam-details-svg" data-filename="c00084_NODE_8_.._r84c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r84c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r84c1-nrps_pks_products" data-name="r84c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r84c1-nrps_pks" data-name="r84c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r84c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-129"></div> <span class="help-tooltip" id="nrps-structures-help-129">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 1 - 65431:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r84c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-X - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r84c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 1 - 29781:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r84c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-X - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r84c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 9777 - 61327:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r84c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-X - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r84c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r84c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-130"></div> <span class="help-tooltip" id="nrps-monomers-help-130">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds10059_13410</span></strong>: X
      <div class="collapser collapser-target-cds10059_13410 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (233..641)</strong>: X
          <div class="collapser collapser-target-cds10059_13410 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophilic
              <div class="collapser collapser-target-cds10059_13410-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DiWQstadDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds29777_41327</span></strong>: X - X - X
      <div class="collapser collapser-target-cds29777_41327 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (473..881)</strong>: X
          <div class="collapser collapser-target-cds29777_41327 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aromatic
              <div class="collapser collapser-target-cds29777_41327-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWeInLdDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2066..2468)</strong>: X
          <div class="collapser collapser-target-cds29777_41327 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp
              <div class="collapser collapser-target-cds29777_41327-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFaVtgVgK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3164..3419)</strong>: X
          <div class="collapser collapser-target-cds29777_41327 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds29777_41327-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">--E---T--K</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys-b</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds59641_61225</span></strong>: X
      <div class="collapser collapser-target-cds59641_61225 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (27..430)</strong>: X
          <div class="collapser collapser-target-cds59641_61225 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds59641_61225-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKMGHiGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r91c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00091_NODE_14.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-428"></div> <span class="help-tooltip" id="region_svg-help-428">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00091_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r91c1-svg" data-filename="c00091_NODE_14.._r91c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 52,735 - 67,819 nt. (total: 15,085 nt)
        
        <a class="cluster-rules-header" id="r91c1-rules-header" href="#r91c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r91c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r91c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-429"></div> <span class="help-tooltip" id="focus-panel-help-429">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r91c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r91c1-clusterblast" data-name="r91c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r91c1-knownclusterblast" data-name="r91c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r91c1-subclusterblast" data-name="r91c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r91c1-MIBiG-cluster-compare" data-name="r91c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r91c1-pfam-details" data-name="r91c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r91c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-136"></div> <span class="help-tooltip" id="cb-general-help-136">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r91c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r91c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r91c1_1.svg">
        NZ_RHGC01000012 (69736-84869): Streptomyces alboflavus strain 313 contig012, ... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_2.svg">
        NZ_WPNZ01000026 (66129-81479): Streptomyces sp. p1417 Scaffold26, whole genom... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_3.svg">
        NZ_FONR01000013 (245532-260515): Streptomyces mirabilis strain OK461, whole g... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_4.svg">
        NZ_CP045643 (2743469-2757798): Streptomyces sp. QMT-28 chromosome, complete g... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_5.svg">
        NZ_AUBE01000020 (78618-93218): Streptomyces flavidovirens DSM 40150 G412DRAFT... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_6.svg">
        NZ_AMPN02000002 (1343959-1357113): Streptomyces sp. SM8 Scaffold 2, whole gen... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_7.svg">
        NZ_CP045178 (6495506-6510843): Streptomyces tsukubensis strain AT3 chromosome... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_8.svg">
        NZ_LFBV01000004 (54802-69612): Streptomyces uncialis strain DCA2648 scaffold4... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_9.svg">
        NZ_AQRJ01000027 (27432-42242): Streptomyces sp. AmelKG-E11A YW7DRAFT scaffold... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r91c1_10.svg">
        NZ_JTIY01000001 (247642-262152): Streptomyces sp. AcH 505 scaffold00001, whol... (88% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r91c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r91c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r91c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-137"></div> <span class="help-tooltip" id="cb-known-help-137">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r91c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-138"></div> <span class="help-tooltip" id="cb-sub-help-138">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r91c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-139"></div> <span class="help-tooltip" id="MIBiG-help-139">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r91c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r91c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r91c1"></div>
    <div id="comparison-MIBiG-r91c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>xanthoferrin</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944.1: 2257888-2263981">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , s, i, d, e, r, o, p, h, o, r, e, )</td>
        
        <td>staphyloferrin A</td>
        
        
        <td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r91c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>xanthoferrin</td>
      <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944.1: 2257888-2263981">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , s, i, d, e, r, o, p, h, o, r, e, )</td>
      <td>staphyloferrin A</td>
      <td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r91c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-140"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-140">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r91c1-pfam-details-svg" data-filename="c00091_NODE_14.._r91c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r91c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r93c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00093_NODE_4_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-430"></div> <span class="help-tooltip" id="region_svg-help-430">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00093_NODE_4_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r93c1-svg" data-filename="c00093_NODE_4_.._r93c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 14,840 nt. (total: 14,840 nt)
        
        <a class="cluster-rules-header" id="r93c1-rules-header" href="#r93c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r93c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r93c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-431"></div> <span class="help-tooltip" id="focus-panel-help-431">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r93c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r93c1-clusterblast" data-name="r93c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r93c1-knownclusterblast" data-name="r93c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r93c1-subclusterblast" data-name="r93c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r93c1-MIBiG-cluster-compare" data-name="r93c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r93c1-pfam-details" data-name="r93c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r93c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-141"></div> <span class="help-tooltip" id="cb-general-help-141">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r93c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r93c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r93c1_1.svg">
        NZ_CP023690 (6274671-6294044): Streptomyces spectabilis strain ATCC 27465 chr... (63% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_2.svg">
        NZ_CP030771 (6289208-6308643): Streptomyces sp. YIM 121038 chromosome, comple... (63% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_3.svg">
        NZ_RHGC01000001 (639216-660332): Streptomyces alboflavus strain 313 contig001... (63% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_4.svg">
        NZ_CP021748 (6294340-6313764): Streptomyces alboflavus strain MDJK44 chromoso... (63% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_5.svg">
        NZ_CP040916 (5945623-5965445): Streptomyces spectabilis strain NRRL 2792 chro... (60% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_6.svg">
        NZ_WPNZ01000001 (50610-71675): Streptomyces sp. p1417 Scaffold1, whole genome... (64% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_7.svg">
        NC_020504 (3567624-3614705): Streptomyces davaonensis JCM 4913, complete genome (27% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_8.svg">
        NZ_KQ947992 (105247-126261): Streptomyces curacoi strain DSM 40107 PRJNA29921... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_9.svg">
        NZ_CP048641 (3149763-3168570): Streptomyces aureoverticillatus strain HN6 chr... (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_10.svg">
        NZ_SLZH01000003 (230084-251098): Streptomyces sp. BK042 Ga0307683 103, whole ... (52% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r93c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r93c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r93c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-142"></div> <span class="help-tooltip" id="cb-known-help-142">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r93c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r93c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r93c1_1.svg">
        albaflavenone
      </option>
      
    </select>
    <button id="knownclusterblast-r93c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r93c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r93c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-143"></div> <span class="help-tooltip" id="cb-sub-help-143">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r93c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-144"></div> <span class="help-tooltip" id="MIBiG-help-144">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r93c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r93c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r93c1"></div>
    <div id="comparison-MIBiG-r93c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.82 (id:0.54, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
         
        
        <td>0.55</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>(-)--cadinene</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677.1: 120-4940">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>cyclooctatin</td>
        
        
        <td>Streptomyces melanosporofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>CC-1065</td>
        
        
        <td>Streptomyces zelensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000671.1: 5310551-5478920">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000671/index.html#r1c1" target="_new">BGC0000671.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene</td>
        
        <td>momilactone</td>
        
        
        <td>Oryza sativa Japonica Group</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r93c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.82 (id:0.54, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
      <td>0.82</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>(-)--cadinene</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677.1: 120-4940">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>cyclooctatin</td>
      <td>Streptomyces melanosporofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>CC-1065</td>
      <td>Streptomyces zelensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000671.1: 5310551-5478920">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000671/index.html#r1c1" target="_new">BGC0000671.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Terpene</td>
      <td>momilactone</td>
      <td>Oryza sativa Japonica Group</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r93c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-145"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-145">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r93c1-pfam-details-svg" data-filename="c00093_NODE_4_.._r93c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r93c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r94c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00094_NODE_22.. - Region 1 - NRPS,PKS-like,T1PKS,transAT-PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-432"></div> <span class="help-tooltip" id="region_svg-help-432">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00094_NODE_22...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r94c1-svg" data-filename="c00094_NODE_22.._r94c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 67,707 nt. (total: 67,707 nt)
        
        <a class="cluster-rules-header" id="r94c1-rules-header" href="#r94c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r94c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>transAT-PKS: (cds(PKS_AT and not (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS)) and (cds(ATd and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS)) or cds(tra_KS and PP-binding)))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>PKS-like: (t2pks2 or ksIII)</div>
      
      <div class="region-svg-container">
       <div id='r94c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-433"></div> <span class="help-tooltip" id="focus-panel-help-433">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r94c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r94c1-nrps_pks" data-name="r94c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r94c1-clusterblast" data-name="r94c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r94c1-knownclusterblast" data-name="r94c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r94c1-subclusterblast" data-name="r94c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r94c1-MIBiG-cluster-compare" data-name="r94c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r94c1-pfam-details" data-name="r94c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r94c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-146"></div> <span class="help-tooltip" id="nrps-domain-help-146">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r94c1-details-svg" data-filename="c00094_NODE_22.._r94c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r94c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r94c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-149"></div> <span class="help-tooltip" id="cb-general-help-149">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r94c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r94c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r94c1_1.svg">
        NZ_CP023690 (285783-351455): Streptomyces spectabilis strain ATCC 27465 chrom... (14% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_2.svg">
        NC_021177 (361553-478840): Streptomyces fulvissimus DSM 40593, complete sequence (8% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_3.svg">
        NZ_LZQS01000004 (126611-243705): Streptomyces sp. IB2014 011-1 STIB 4, whole ... (7% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_4.svg">
        NZ_CP011522 (7970151-8088157): Streptomyces sp. CFMR 7 strain CFMR-7 chromoso... (8% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_5.svg">
        NZ_KZ819160 (182811-298798): Streptomyces sp. CS090A Scaffold00002, whole gen... (7% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_6.svg">
        NZ_WWIF01000005 (279694-397190): Streptomyces sp. SID5594 SID5594.c5, whole g... (7% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_7.svg">
        NZ_CP054926 (233409-350060): Streptomyces fulvissimus strain NA06532 chromoso... (7% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_8.svg">
        NZ_KL647090 (198809-315466): Streptomyces sp. NRRL S-623 Doro1 scaffold1, who... (7% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_9.svg">
        NZ_CP030930 (108461-226163): Streptomyces cavourensis strain TJ430 chromosome... (8% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r94c1_10.svg">
        NZ_WWGS01000014 (126482-241157): Streptomyces sp. SID8359 SID8359.c14, whole ... (8% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
    </select>
    <button id= "clusterblast-r94c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r94c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r94c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-150"></div> <span class="help-tooltip" id="cb-known-help-150">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r94c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r94c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r94c1_1.svg">
        actinomycin D
      </option>
      
    </select>
    <button id="knownclusterblast-r94c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r94c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r94c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-151"></div> <span class="help-tooltip" id="cb-sub-help-151">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r94c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-152"></div> <span class="help-tooltip" id="MIBiG-help-152">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r94c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r94c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r94c1"></div>
    <div id="comparison-MIBiG-r94c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">transAT-PKS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002057.1: 10296-37416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002057/index.html#r1c1" target="_new">BGC0002057.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>1.15</td>
        <td>Polyketide</td>
        
        <td>pateamine</td>
        
        
        <td>Kiritimatiellaeota bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257.1: 399-7066">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>1.14</td>
        <td>Polyketide</td>
        
        <td>1,3,6,8-tetrahydroxynaphthalene</td>
        
        
        <td>Nodulisporium sp. ATCC74245</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107.1: 1136198-1142873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>1.12</td>
        <td>Polyketide</td>
        
        <td>naphthopyrone</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854.1: 27635-56388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>1.07</td>
        <td>Polyketide</td>
        
        <td>tricholignan A</td>
        
        
        <td>Trichoderma harzianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001901.1: 22313-56388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001901/index.html#r1c1" target="_new">BGC0001901.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>1.01</td>
        <td>Polyketide</td>
        
        <td>tricholignan A</td>
        
        
        <td>Trichoderma harzianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266.1: 10707-26494">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.98</td>
        <td>Polyketide</td>
        
        <td>grayanic acid</td>
        
        
        <td>Cladonia grayi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000205.1: 1474-9885">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000205/index.html#r1c1" target="_new">BGC0000205.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.91</td>
        <td>Polyketide</td>
        
        <td>bryostatin</td>
        
        
        <td>Candidatus Endobugula sertula</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583.1: 3200-15837">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.91</td>
        <td>Polyketide</td>
        
        <td>emodin</td>
        
        
        <td>Escovopsis weberi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001704.1: 0-37663">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001704/index.html#r1c1" target="_new">BGC0001704.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.90</td>
        <td>Other</td>
        
        <td>nocuolin A</td>
        
        
        <td>Nodularia sp. HBU26</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000121.1: 4103-40345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000121/index.html#r1c1" target="_new">BGC0000121.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.89</td>
        <td>Polyketide</td>
        
        <td>RES-1214-2</td>
        
        
        <td>Pestalotiopsis fici</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r94c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002057.1: 10296-37416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002057/index.html#r1c1" target="_new">BGC0002057.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>pateamine</td>
      <td>Kiritimatiellaeota bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.75, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257.1: 399-7066">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>1,3,6,8-tetrahydroxynaphthalene</td>
      <td>Nodulisporium sp. ATCC74245</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001502.1: 2741865-2757479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001502/index.html#r1c1" target="_new">BGC0001502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>amonabactin P 750</td>
      <td>Aeromonas hydrophila subsp. hydrophila ATCC 7966</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r94c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-153"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-153">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r94c1-pfam-details-svg" data-filename="c00094_NODE_22.._r94c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r94c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r94c1-nrps_pks_products" data-name="r94c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r94c1-nrps_pks" data-name="r94c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r94c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-147"></div> <span class="help-tooltip" id="nrps-structures-help-147">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 67707:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r94c1-1" class="smiles-canvas" data-smiles="CC(=O)C(C)C(=O)C=CCC(=O)NCC(=O)CC(=O)C=CNC(Cc1ccccc1)C(=O)NC([*])C(=O)NC(C(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (Me-mal) + (ccmal - mal) + (gly - mal - ccmal) + (D-phe - X - val)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=me-mal&amp;nrps3=ccmal,mal&amp;nrps4=gly,mal,ccmal&amp;nrps5=d-phe,x,val" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=me-mal*&amp;nrps3=ccmal*,mal*&amp;nrps4=gly*,mal*,ccmal*&amp;nrps5=d-phe*,x,val*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r94c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(C)C(=O)C=CCC(=O)NCC(=O)CC(=O)C=CNC(Cc1ccccc1)C(=O)NC([*])C(=O)NC(C(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r94c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-148"></div> <span class="help-tooltip" id="nrps-monomers-help-148">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_12636</span></strong>: phe - X - val
      <div class="collapser collapser-target-cds0_12636 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,X,val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,X*,val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (779..1174)</strong>: phe
          <div class="collapser collapser-target-cds0_12636 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe
              <div class="collapser collapser-target-cds0_12636-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>phe</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWTaAAVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2329..2724)</strong>: X
          <div class="collapser collapser-target-cds0_12636 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala
              <div class="collapser collapser-target-cds0_12636-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSvAAfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3382..3780)</strong>: val
          <div class="collapser collapser-target-cds0_12636 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds0_12636-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYFWGaTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21638_32423</span></strong>: gly
      <div class="collapser collapser-target-cds21638_32423 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (494..898)</strong>: gly
          <div class="collapser collapser-target-cds21638_32423 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds21638_32423-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds36951_45435</span></strong>: mmal
      <div class="collapser collapser-target-cds36951_45435 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (544..840)</strong>: mmal
          <div class="collapser collapser-target-cds36951_45435 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds36951_45435-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 87.5%</dd>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methoxymalonyl-CoA: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds36951_45435-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 184.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 180.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 149.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 127.9</dt>

  <dd></dd><dt>Malonyl-CoA: 117.4</dt>

  <dd></dd><dt>Propionyl-CoA: 111.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 97.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 85.0</dt>

  <dd></dd><dt>CHC-CoA: 82.9</dt>

  <dd></dd><dt>Acetyl-CoA: 61.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 59.2</dt>

  <dd></dd><dt>fatty_acid: 43.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 33.7</dt>

  <dd></dd><dt>inactive: 30.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds46459_47707</span></strong>: mal
      <div class="collapser collapser-target-cds46459_47707 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (4..285)</strong>: mal
          <div class="collapser collapser-target-cds46459_47707 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds46459_47707-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds46459_47707-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 98.4</dt>

  <dd></dd><dt>Propionyl-CoA: 36.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 30.1</dt>

  <dd></dd><dt>inactive: 30.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 19.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 12.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 10.8</dt>

  <dd></dd><dt>CHC-CoA: 10.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 8.8</dt>

  <dd></dd><dt>Acetyl-CoA: 8.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r96c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00096_NODE_88.. - Region 1 - T1PKS,other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-434"></div> <span class="help-tooltip" id="region_svg-help-434">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00096_NODE_88...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r96c1-svg" data-filename="c00096_NODE_88.._r96c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 39,339 nt. (total: 39,339 nt)
        
        <a class="cluster-rules-header" id="r96c1-rules-header" href="#r96c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r96c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r96c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-435"></div> <span class="help-tooltip" id="focus-panel-help-435">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r96c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r96c1-nrps_pks" data-name="r96c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r96c1-clusterblast" data-name="r96c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r96c1-knownclusterblast" data-name="r96c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r96c1-subclusterblast" data-name="r96c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r96c1-MIBiG-cluster-compare" data-name="r96c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r96c1-pfam-details" data-name="r96c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r96c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-154"></div> <span class="help-tooltip" id="nrps-domain-help-154">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r96c1-details-svg" data-filename="c00096_NODE_88.._r96c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r96c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r96c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-157"></div> <span class="help-tooltip" id="cb-general-help-157">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r96c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r96c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r96c1_1.svg">
        NZ_CP010849 (181793-248590): Streptomyces cyaneogriseus subsp. noncyanogenus ... (61% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_2.svg">
        NZ_JH993791 (611-62373): Streptomyces viridosporus T7A StrvirDRAFT SVC.5, who... (63% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_3.svg">
        NZ_CP026490 (20189-159129): Streptomyces sp. 604F chromosome, complete genome (40% of genes show similarity), NRPS,T1PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_4.svg">
        NZ_CM002271 (3165-135288): Streptomyces sp. GBA 94-10 chromosome, whole genom... (40% of genes show similarity), NRPS,T1PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_5.svg">
        NZ_CM002271 (6889180-7023245): Streptomyces sp. GBA 94-10 chromosome, whole g... (40% of genes show similarity), NRPS,T1PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_6.svg">
        NZ_CM002273 (730-128956): Streptomyces sp. PVA 94-07 strain PVA 94-07 chromos... (42% of genes show similarity), NRPS,T1PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_7.svg">
        NZ_LN831790 (324783-457035): Streptomyces leeuwenhoekii strain type strain (C... (32% of genes show similarity), NRPS,T1PKS,T3PKS,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_8.svg">
        NC_016109 (3025555-3093773): Kitasatospora setae KM-6054, complete genome (45% of genes show similarity), LAP,PKS-like,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_9.svg">
        NZ_KB900614 (724882-774662): Salinispora pacifica DSM 45549 = CNT-150 strain ... (46% of genes show similarity), T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r96c1_10.svg">
        NZ_FMZZ01000008 (43978-113656): Alloactinosynnema iranicum strain IBRC-M 1040... (32% of genes show similarity), PKS-like,T1PKS,butyrolactone,other
      </option>
      
    </select>
    <button id= "clusterblast-r96c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r96c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r96c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-158"></div> <span class="help-tooltip" id="cb-known-help-158">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r96c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r96c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r96c1_1.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r96c1_2.svg">
        pyrrolomycin A / pyrrolomycin B / pyrrolomycin C / pyrrolomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r96c1_3.svg">
        ,-epoxyketone
      </option>
      
      <option value = "svg/knownclusterblast_r96c1_4.svg">
        eponemycin
      </option>
      
      <option value = "svg/knownclusterblast_r96c1_5.svg">
        diisonitrile antibiotic SF2768
      </option>
      
      <option value = "svg/knownclusterblast_r96c1_6.svg">
        salinomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r96c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r96c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r96c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-159"></div> <span class="help-tooltip" id="cb-sub-help-159">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r96c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-160"></div> <span class="help-tooltip" id="MIBiG-help-160">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r96c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r96c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r96c1"></div>
    <div id="comparison-MIBiG-r96c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">other</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.58</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.54</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.50</td>
        <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>esmeraldin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.50</td>
        <td>Polyketide</td>
        
        <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
        
        
        <td>Streptomyces sp. A2991200</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.49</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>fluostatins M-Q</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000194.1: 5513808-5535091">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000194/index.html#r1c1" target="_new">BGC0000194.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>actinorhodin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.51, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.47</td>
        <td>Other</td>
        
        <td>A33853</td>
        
        
        <td>Streptomyces sp. NRRL 12068</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.47</td>
        <td>Terpene, Polyketide</td>
        
        <td>aurachin RE</td>
        
        
        <td>Rhodococcus erythropolis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r96c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>esmeraldin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000194.1: 5513808-5535091">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000194/index.html#r1c1" target="_new">BGC0000194.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>actinorhodin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
      <td>Streptomyces sp. A2991200</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>fluostatins M-Q</td>
      <td>Streptomyces albus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
      <td>Streptomyces sp. CB09001</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r96c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-161"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-161">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r96c1-pfam-details-svg" data-filename="c00096_NODE_88.._r96c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r96c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r96c1-nrps_pks_products" data-name="r96c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r96c1-nrps_pks" data-name="r96c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r96c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-155"></div> <span class="help-tooltip" id="nrps-structures-help-155">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 39339:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r96c1-1" class="smiles-canvas" data-smiles="CC(O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r96c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 0 - 36418:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r96c1-2" class="smiles-canvas" data-smiles="CC(O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r96c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r96c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-156"></div> <span class="help-tooltip" id="nrps-monomers-help-156">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds8661_10278</span></strong>: X
      <div class="collapser collapser-target-cds8661_10278 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (13..430)</strong>: X
          <div class="collapser collapser-target-cds8661_10278 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds8661_10278-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">gigFaGisVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds10457_16418</span></strong>: mal
      <div class="collapser collapser-target-cds10457_16418 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (565..860)</strong>: mal
          <div class="collapser collapser-target-cds10457_16418 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds10457_16418-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 91.7%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds10457_16418-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 169.3</dt>

  <dd></dd><dt>inactive: 101.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 53.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 50.6</dt>

  <dd></dd><dt>Propionyl-CoA: 34.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 26.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 18.4</dt>

  <dd></dd><dt>CHC-CoA: 15.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 12.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 12.1</dt>

  <dd></dd><dt>Acetyl-CoA: 11.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r99c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00099_NODE_13.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-436"></div> <span class="help-tooltip" id="region_svg-help-436">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00099_NODE_13...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r99c1-svg" data-filename="c00099_NODE_13.._r99c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 24,329 nt. (total: 24,329 nt)
        
        <a class="cluster-rules-header" id="r99c1-rules-header" href="#r99c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r99c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r99c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-437"></div> <span class="help-tooltip" id="focus-panel-help-437">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r99c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r99c1-nrps_pks" data-name="r99c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r99c1-clusterblast" data-name="r99c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r99c1-knownclusterblast" data-name="r99c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r99c1-subclusterblast" data-name="r99c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r99c1-MIBiG-cluster-compare" data-name="r99c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r99c1-pfam-details" data-name="r99c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r99c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-162"></div> <span class="help-tooltip" id="nrps-domain-help-162">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r99c1-details-svg" data-filename="c00099_NODE_13.._r99c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r99c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r99c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-165"></div> <span class="help-tooltip" id="cb-general-help-165">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r99c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r99c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r99c1_1.svg">
        NZ_LT629701 (2429365-2651872): Allokutzneria albata strain DSM 44149 chromoso... (5% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r99c1_2.svg">
        NZ_CP023690 (386026-497923): Streptomyces spectabilis strain ATCC 27465 chrom... (5% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r99c1_3.svg">
        NC_015312 (234323-317006): Pseudonocardia dioxanivorans CB1190, complete sequ... (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r99c1_4.svg">
        NZ_CP015098 (7375426-7510438): Streptomyces qaidamensis strain S10 chromosome... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r99c1_5.svg">
        NZ_CP015726 (1909867-1981053): Streptomyces sp. RTd22 chromosome, complete ge... (8% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r99c1_6.svg">
        NZ_JACHJP010000002 (187398-248741): Streptosporangium saharense strain CECT 8... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r99c1_7.svg">
        NZ_CP031516 (4253190-4331778): [Mycobacterium] chelonae subsp. gwanakae strai... (6% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r99c1_8.svg">
        NZ_KQ948675 (29967-86140): Streptomyces canus strain DSM 40017 PRJNA299229 s0... (12% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r99c1_9.svg">
        NZ_CP029189 (415-60850): Streptomyces venezuelae strain ATCC 21018 chromosome... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r99c1_10.svg">
        NZ_CP040916 (42604-117730): Streptomyces spectabilis strain NRRL 2792 chromos... (7% of genes show similarity), NRPS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r99c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r99c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r99c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-166"></div> <span class="help-tooltip" id="cb-known-help-166">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r99c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r99c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_1.svg">
        cysteoamide
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_2.svg">
        polyoxypeptin
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_3.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_4.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_5.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_6.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_7.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_8.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
      <option value = "svg/knownclusterblast_r99c1_9.svg">
        sanglifehrin A
      </option>
      
    </select>
    <button id="knownclusterblast-r99c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r99c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r99c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-167"></div> <span class="help-tooltip" id="cb-sub-help-167">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r99c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-168"></div> <span class="help-tooltip" id="MIBiG-help-168">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r99c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r99c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r99c1"></div>
    <div id="comparison-MIBiG-r99c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.43, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.41, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus niger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r99c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.43, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.41, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus niger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.48, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r99c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-169"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-169">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r99c1-pfam-details-svg" data-filename="c00099_NODE_13.._r99c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r99c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r99c1-nrps_pks_products" data-name="r99c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r99c1-nrps_pks" data-name="r99c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r99c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-163"></div> <span class="help-tooltip" id="nrps-structures-help-163">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 24329:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r99c1-1" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-val - X) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val,x&amp;nrps2=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val*,x&amp;nrps2=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r99c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r99c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-164"></div> <span class="help-tooltip" id="nrps-monomers-help-164">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds10624_17242</span></strong>: val - X
      <div class="collapser collapser-target-cds10624_17242 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (22..425)</strong>: val
          <div class="collapser collapser-target-cds10624_17242 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds10624_17242-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFFwGaVlK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1609..2001)</strong>: X
          <div class="collapser collapser-target-cds10624_17242 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds10624_17242-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvFsVAAyaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds17238_24327</span></strong>: X - X
      <div class="collapser collapser-target-cds17238_24327 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (469..854)</strong>: X
          <div class="collapser collapser-target-cds17238_24327 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds17238_24327-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVFSVGayaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>d-lyserg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2016..2363)</strong>: X
          <div class="collapser collapser-target-cds17238_24327 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds17238_24327-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYlWGamaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r103c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00103_NODE_24.. - Region 1 - lanthipeptide-class-i <div class="help-container"> <div class="help-icon" data-id="region_svg-help-438"></div> <span class="help-tooltip" id="region_svg-help-438">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00103_NODE_24...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r103c1-svg" data-filename="c00103_NODE_24.._r103c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 3,211 - 27,757 nt. (total: 24,547 nt)
        
        <a class="cluster-rules-header" id="r103c1-rules-header" href="#r103c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r103c1-rules">lanthipeptide-class-i: ((LANC_like and (Lant_dehydr_N or Lant_dehydr_C)) and not (YcaO or TIGR03882))</div>
      
      <div class="region-svg-container">
       <div id='r103c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-439"></div> <span class="help-tooltip" id="focus-panel-help-439">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r103c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r103c1-lanthipeptides" data-name="r103c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r103c1-clusterblast" data-name="r103c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r103c1-knownclusterblast" data-name="r103c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r103c1-subclusterblast" data-name="r103c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r103c1-MIBiG-cluster-compare" data-name="r103c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r103c1-pfam-details" data-name="r103c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r103c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-170"></div> <span class="help-tooltip" id="lanthi-body-help-170">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds13210_16216 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds12986_13139</span> leader / core peptide
          </dt>
          <dd>
            MDATVIAALPLAEDEWVLDITVTDAPR
            -
            VMARDD<span class="cys">C</span>D<span class="dhb">Dhb</span>NDG<span class="cys">C</span>Q<span class="dha">Dha</span><span class="dhb">Dhb</span><span class="cys">C</span>D<span class="dha">Dha</span>A<span class="cys">C</span>L<span class="dha">Dha</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r103c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-172"></div> <span class="help-tooltip" id="cb-general-help-172">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r103c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r103c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r103c1_1.svg">
        NZ_CP015588 (1813949-1834588): Streptomyces alfalfae strain ACCC40021 chromos... (28% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r103c1_2.svg">
        NZ_CP029254 (1877525-1901483): Streptomyces spongiicola strain HNM0071 chromo... (29% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_3.svg">
        NZ_CP028834 (5800616-5827063): Streptomyces sp. M2 chromosome, complete genome (19% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_4.svg">
        NZ_AP018517 (4262237-4287500): Streptomyces rochei strain 7434AN4 (22% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_5.svg">
        NZ_SNZM01000001 (107357-132471): Streptomyces sp. BK561 Ga0307712 101, whole ... (13% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_6.svg">
        NZ_CP054919 (6120447-6145508): Kitasatospora sp. NA04385 chromosome, complete... (15% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_7.svg">
        NZ_CP025394 (1831360-1854916): Kitasatospora sp. MMS16-BH015 chromosome, comp... (15% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_8.svg">
        NZ_CP029618 (4828555-4852942): Streptomyces sp. WAC 06738 chromosome, complet... (23% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r103c1_9.svg">
        NZ_CP054929 (8148828-8275216): Streptomyces sp. NA00687 chromosome, complete ... (6% of genes show similarity), NRPS,linaridin,terpene
      </option>
      
      <option value = "svg/clusterblast_r103c1_10.svg">
        NZ_JAIY01000005 (76583-102830): Kitasatospora sp. MBT66 scaffold5, whole geno... (17% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r103c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r103c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r103c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-173"></div> <span class="help-tooltip" id="cb-known-help-173">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r103c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-174"></div> <span class="help-tooltip" id="cb-sub-help-174">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r103c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-175"></div> <span class="help-tooltip" id="MIBiG-help-175">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r103c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r103c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r103c1"></div>
    <div id="comparison-MIBiG-r103c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-i</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>penisin</td>
        
        
        <td>Paenibacillus ehimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.56, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000541.1: 0-9113">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000541/index.html#r1c1" target="_new">BGC0000541.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>paenicidin A</td>
        
        
        <td>Paenibacillus polymyxa</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000538.1: 238-7610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000538/index.html#r1c1" target="_new">BGC0000538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>nisin Z</td>
        
        
        <td>Lactococcus lactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001618.1: 70-8595">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001618/index.html#r1c1" target="_new">BGC0001618.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>hyicin 3682</td>
        
        
        <td>Staphylococcus hyicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000508.1: 0-8379">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000508/index.html#r1c1" target="_new">BGC0000508.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>RiPP</td>
        
        <td>epidermin</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506.1: 405-13975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>entianin</td>
        
        
        <td>Bacillus subtilis subsp. spizizenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000535.1: 827-14152">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000535/index.html#r1c1" target="_new">BGC0000535.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>nisin A</td>
        
        
        <td>Lactococcus lactis subsp. lactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>paenilan</td>
        
        
        <td>Paenibacillus polymyxa E681</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559.1: 494-13831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>subtilin</td>
        
        
        <td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r103c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>RiPP</td>
      <td>penisin</td>
      <td>Paenibacillus ehimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.56, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000541.1: 0-9113">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000541/index.html#r1c1" target="_new">BGC0000541.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>paenicidin A</td>
      <td>Paenibacillus polymyxa</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000538.1: 238-7610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000538/index.html#r1c1" target="_new">BGC0000538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>nisin Z</td>
      <td>Lactococcus lactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001618.1: 70-8595">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001618/index.html#r1c1" target="_new">BGC0001618.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>hyicin 3682</td>
      <td>Staphylococcus hyicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000508.1: 0-8379">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000508/index.html#r1c1" target="_new">BGC0000508.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>epidermin</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506.1: 405-13975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>entianin</td>
      <td>Bacillus subtilis subsp. spizizenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000535.1: 827-14152">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000535/index.html#r1c1" target="_new">BGC0000535.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>nisin A</td>
      <td>Lactococcus lactis subsp. lactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>paenilan</td>
      <td>Paenibacillus polymyxa E681</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559.1: 494-13831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>subtilin</td>
      <td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r103c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-176"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-176">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r103c1-pfam-details-svg" data-filename="c00103_NODE_24.._r103c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r103c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r103c1-lanthipeptides" data-name="r103c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r103c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-171"></div> <span class="help-tooltip" id="lanthi-side-help-171">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds12986_13139</span> - Class I</dt>
        <dd>Cleavage pHMM score: -11.00</dd>
        <dd>RODEO score: 19</dd>
        <dd>Monoisotopic mass: 2318.9 Da</dd>
        <dd>Molecular weight: 2320.5 Da</dd>
        <dd>Number of bridges: 4</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2338.5 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r104c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00104_NODE_37.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-440"></div> <span class="help-tooltip" id="region_svg-help-440">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00104_NODE_37...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r104c1-svg" data-filename="c00104_NODE_37.._r104c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 77,841 nt. (total: 77,841 nt)
        
        <a class="cluster-rules-header" id="r104c1-rules-header" href="#r104c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r104c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r104c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-441"></div> <span class="help-tooltip" id="focus-panel-help-441">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r104c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r104c1-nrps_pks" data-name="r104c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r104c1-clusterblast" data-name="r104c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r104c1-knownclusterblast" data-name="r104c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r104c1-subclusterblast" data-name="r104c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r104c1-MIBiG-cluster-compare" data-name="r104c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r104c1-pfam-details" data-name="r104c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r104c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-177"></div> <span class="help-tooltip" id="nrps-domain-help-177">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r104c1-details-svg" data-filename="c00104_NODE_37.._r104c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r104c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r104c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-180"></div> <span class="help-tooltip" id="cb-general-help-180">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r104c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r104c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r104c1_1.svg">
        NZ_CP030771 (8015578-8162299): Streptomyces sp. YIM 121038 chromosome, comple... (50% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_2.svg">
        NZ_CP040916 (7733849-7879172): Streptomyces spectabilis strain NRRL 2792 chro... (47% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_3.svg">
        NZ_CP023690 (8047123-8193590): Streptomyces spectabilis strain ATCC 27465 chr... (47% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_4.svg">
        NZ_CP048641 (1021665-1243764): Streptomyces aureoverticillatus strain HN6 chr... (29% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_5.svg">
        NZ_CP021748 (8263860-8399611): Streptomyces alboflavus strain MDJK44 chromoso... (52% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_6.svg">
        NZ_CP016174 (6074368-6223519): Amycolatopsis orientalis strain B-37 chromosom... (22% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_7.svg">
        NZ_GG657754 (6762597-6972752): Streptomyces himastatinicus ATCC 53653 superco... (23% of genes show similarity), NRPS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_8.svg">
        NZ_CP029711 (7772223-7941678): Streptosporangium sp. &#39;caverna&#39; chromosome (26% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_9.svg">
        NZ_FWXV01000001 (119666-310509): Kibdelosporangium aridum strain DSM 43828, w... (16% of genes show similarity), PKS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r104c1_10.svg">
        NZ_AP022871 (1761649-1945724): Phytohabitans suffuscus strain NBRC 105367 (23% of genes show similarity), T1PKS,ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r104c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r104c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r104c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-181"></div> <span class="help-tooltip" id="cb-known-help-181">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r104c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r104c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_1.svg">
        niphimycins C-E
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_2.svg">
        concanamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_3.svg">
        stambomycin A / stambomycin B / stambomycin C / stambomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_4.svg">
        cremimycin
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_5.svg">
        hitachimycin
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_6.svg">
        elaiophylin
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_7.svg">
        fostriecin
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_8.svg">
        lydicamycin
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_9.svg">
        meridamycin
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_10.svg">
        actinoallolide A
      </option>
      
    </select>
    <button id="knownclusterblast-r104c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r104c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r104c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-182"></div> <span class="help-tooltip" id="cb-sub-help-182">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r104c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-183"></div> <span class="help-tooltip" id="MIBiG-help-183">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r104c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r104c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r104c1"></div>
    <div id="comparison-MIBiG-r104c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.48, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.50, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000018.1: 243-42343">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000018/index.html#r1c1" target="_new">BGC0000018.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.50, order:0.63, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160.1: 89033-116126">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>terreic acid</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000113.1: 0-41097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000113/index.html#r1c1" target="_new">BGC0000113.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:0.53, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>niddamycin</td>
        
        
        <td>Streptomyces caelestis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.50, order:0.50, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r104c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.48, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.50, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000018.1: 243-42343">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000018/index.html#r1c1" target="_new">BGC0000018.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.50, order:0.63, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160.1: 89033-116126">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>terreic acid</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000113.1: 0-41097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000113/index.html#r1c1" target="_new">BGC0000113.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.49, order:0.53, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>niddamycin</td>
      <td>Streptomyces caelestis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.50, order:0.50, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r104c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-184"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-184">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r104c1-pfam-details-svg" data-filename="c00104_NODE_37.._r104c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r104c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r104c1-nrps_pks_products" data-name="r104c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r104c1-nrps_pks" data-name="r104c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r104c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-178"></div> <span class="help-tooltip" id="nrps-structures-help-178">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 77841:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r104c1-1" class="smiles-canvas" data-smiles="CC(O)C(C)C(O)CC(O)C(C)=CC(C)C(O)C=CC(C)C(O)C=CCCC(C)C(O)C(C)C(O)CCC(C)=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal - Me-ohmal - ohmal) + (Me-ccmal - Me-ohmal - ccmal) + (Me-ohmal - ccmal) + (redmal - Me-ohmal - Me-ohmal) + (redmal - Me-ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal,me-ohmal,ohmal&amp;nrps2=me-ccmal,me-ohmal,ccmal&amp;nrps3=me-ohmal,ccmal&amp;nrps4=redmal,me-ohmal,me-ohmal&amp;nrps5=redmal,me-ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*,me-ohmal*,ohmal*&amp;nrps2=me-ccmal*,me-ohmal*,ccmal*&amp;nrps3=me-ohmal*,ccmal*&amp;nrps4=redmal*,me-ohmal*,me-ohmal*&amp;nrps5=redmal*,me-ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r104c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)C(C)C(O)CC(O)C(C)=CC(C)C(O)C=CC(C)C(O)C=CCCC(C)C(O)C(C)C(O)CCC(C)=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r104c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-179"></div> <span class="help-tooltip" id="nrps-monomers-help-179">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds401_13154</span></strong>: mal - mmal - mal
      <div class="collapser collapser-target-cds401_13154 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (39..336)</strong>: mal
          <div class="collapser collapser-target-cds401_13154 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds401_13154-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 87.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds401_13154-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 210.4</dt>

  <dd></dd><dt>inactive: 114.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 53.6</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 46.4</dt>

  <dd></dd><dt>Propionyl-CoA: 30.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 26.8</dt>

  <dd></dd><dt>Acetyl-CoA: 20.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 10.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1643..1869)</strong>: mmal
          <div class="collapser collapser-target-cds401_13154 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds401_13154-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds401_13154-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 63.2</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 56.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 54.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 54.0</dt>

  <dd></dd><dt>Propionyl-CoA: 50.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 45.3</dt>

  <dd></dd><dt>CHC-CoA: 41.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 31.0</dt>

  <dd></dd><dt>Malonyl-CoA: 30.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 29.1</dt>

  <dd></dd><dt>Acetyl-CoA: 16.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 9.4</dt>

  <dd></dd><dt>fatty_acid: 6.9</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (3134..3425)</strong>: mal
          <div class="collapser collapser-target-cds401_13154 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds401_13154-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds401_13154-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 210.3</dt>

  <dd></dd><dt>inactive: 113.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 51.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 47.4</dt>

  <dd></dd><dt>Propionyl-CoA: 35.1</dt>

  <dd></dd><dt>Acetyl-CoA: 18.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.4</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 5.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds13397_29252</span></strong>: mmal - mmal - mal
      <div class="collapser collapser-target-cds13397_29252 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (580..874)</strong>: mmal
          <div class="collapser collapser-target-cds13397_29252 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds13397_29252-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds13397_29252-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 215.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 191.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 171.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 154.3</dt>

  <dd></dd><dt>Propionyl-CoA: 118.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 115.5</dt>

  <dd></dd><dt>CHC-CoA: 99.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 96.3</dt>

  <dd></dd><dt>Malonyl-CoA: 96.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 76.0</dt>

  <dd></dd><dt>Acetyl-CoA: 68.6</dt>

  <dd></dd><dt>fatty_acid: 48.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 41.2</dt>

  <dd></dd><dt>inactive: 27.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2414..2706)</strong>: mmal
          <div class="collapser collapser-target-cds13397_29252 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds13397_29252-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds13397_29252-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 220.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 198.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 176.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 156.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 127.3</dt>

  <dd></dd><dt>Propionyl-CoA: 121.1</dt>

  <dd></dd><dt>CHC-CoA: 102.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 98.0</dt>

  <dd></dd><dt>Malonyl-CoA: 95.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 75.1</dt>

  <dd></dd><dt>Acetyl-CoA: 61.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 45.0</dt>

  <dd></dd><dt>fatty_acid: 42.5</dt>

  <dd></dd><dt>inactive: 22.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (3952..4244)</strong>: mal
          <div class="collapser collapser-target-cds13397_29252 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds13397_29252-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds13397_29252-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 200.8</dt>

  <dd></dd><dt>inactive: 109.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 43.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 43.1</dt>

  <dd></dd><dt>Propionyl-CoA: 31.8</dt>

  <dd></dd><dt>Acetyl-CoA: 21.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 13.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 11.6</dt>

  <dd></dd><dt>fatty_acid: 8.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 8.8</dt>

  <dd></dd><dt>CHC-CoA: 6.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 1.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds29252_39518</span></strong>: mmal - mal
      <div class="collapser collapser-target-cds29252_39518 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (578..872)</strong>: mmal
          <div class="collapser collapser-target-cds29252_39518 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds29252_39518-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds29252_39518-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 219.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 200.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 171.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 154.2</dt>

  <dd></dd><dt>Propionyl-CoA: 120.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 118.5</dt>

  <dd></dd><dt>CHC-CoA: 106.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 100.8</dt>

  <dd></dd><dt>Malonyl-CoA: 98.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 72.1</dt>

  <dd></dd><dt>Acetyl-CoA: 71.3</dt>

  <dd></dd><dt>fatty_acid: 48.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 39.2</dt>

  <dd></dd><dt>inactive: 24.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2142..2434)</strong>: mal
          <div class="collapser collapser-target-cds29252_39518 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds29252_39518-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds29252_39518-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 196.9</dt>

  <dd></dd><dt>inactive: 107.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 41.3</dt>

  <dd></dd><dt>Propionyl-CoA: 30.7</dt>

  <dd></dd><dt>Acetyl-CoA: 21.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 12.7</dt>

  <dd></dd><dt>fatty_acid: 9.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.0</dt>

  <dd></dd><dt>CHC-CoA: 7.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 1.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds39822_55812</span></strong>: mal - mmal - mmal
      <div class="collapser collapser-target-cds39822_55812 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (568..860)</strong>: mal
          <div class="collapser collapser-target-cds39822_55812 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds39822_55812-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 79.2%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds39822_55812-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 209.4</dt>

  <dd></dd><dt>inactive: 110.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 59.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 58.5</dt>

  <dd></dd><dt>Propionyl-CoA: 37.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 20.1</dt>

  <dd></dd><dt>fatty_acid: 16.7</dt>

  <dd></dd><dt>Acetyl-CoA: 16.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 14.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 13.9</dt>

  <dd></dd><dt>CHC-CoA: 11.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 2.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2671..2967)</strong>: mmal
          <div class="collapser collapser-target-cds39822_55812 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds39822_55812-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds39822_55812-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 218.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 201.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 172.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 146.7</dt>

  <dd></dd><dt>Propionyl-CoA: 122.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 115.4</dt>

  <dd></dd><dt>CHC-CoA: 103.9</dt>

  <dd></dd><dt>Malonyl-CoA: 98.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 95.1</dt>

  <dd></dd><dt>Acetyl-CoA: 70.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 64.8</dt>

  <dd></dd><dt>fatty_acid: 46.4</dt>

  <dd></dd><dt>inactive: 33.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 30.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (4242..4536)</strong>: mmal
          <div class="collapser collapser-target-cds39822_55812 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds39822_55812-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds39822_55812-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 219.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 200.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 171.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 154.2</dt>

  <dd></dd><dt>Propionyl-CoA: 120.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 118.5</dt>

  <dd></dd><dt>CHC-CoA: 106.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 100.8</dt>

  <dd></dd><dt>Malonyl-CoA: 98.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 72.1</dt>

  <dd></dd><dt>Acetyl-CoA: 71.3</dt>

  <dd></dd><dt>fatty_acid: 48.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 39.2</dt>

  <dd></dd><dt>inactive: 24.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds55945_68806</span></strong>: mal - mmal
      <div class="collapser collapser-target-cds55945_68806 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (571..867)</strong>: mal
          <div class="collapser collapser-target-cds55945_68806 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds55945_68806-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 87.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds55945_68806-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 207.6</dt>

  <dd></dd><dt>inactive: 113.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 51.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 48.3</dt>

  <dd></dd><dt>Propionyl-CoA: 32.9</dt>

  <dd></dd><dt>Acetyl-CoA: 17.2</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 17.1</dt>

  <dd></dd><dt>fatty_acid: 15.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 12.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 11.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2695..2992)</strong>: mmal
          <div class="collapser collapser-target-cds55945_68806 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds55945_68806-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds55945_68806-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 217.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 199.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 173.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 150.6</dt>

  <dd></dd><dt>Propionyl-CoA: 124.1</dt>

  <dd></dd><dt>CHC-CoA: 113.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 112.1</dt>

  <dd></dd><dt>Malonyl-CoA: 97.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 87.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 82.1</dt>

  <dd></dd><dt>Acetyl-CoA: 74.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 45.0</dt>

  <dd></dd><dt>fatty_acid: 43.6</dt>

  <dd></dd><dt>inactive: 24.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds70873_72307</span></strong>: X
      <div class="collapser collapser-target-cds70873_72307 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (8..371)</strong>: X
          <div class="collapser collapser-target-cds70873_72307 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds70873_72307-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">qYsdCGsnIK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds72421_73426</span></strong>: pk
      <div class="collapser collapser-target-cds72421_73426 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (10..287)</strong>: pk
          <div class="collapser collapser-target-cds72421_73426 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds72421_73426-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  CHC-CoA
              <div class="collapser collapser-target-cds72421_73426-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>CHC-CoA: 12.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Malonyl-CoA: -0.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r111c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00111_NODE_23.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-442"></div> <span class="help-tooltip" id="region_svg-help-442">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00111_NODE_23...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r111c1-svg" data-filename="c00111_NODE_23.._r111c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 7,335 - 18,918 nt. (total: 11,584 nt)
        
        <a class="cluster-rules-header" id="r111c1-rules-header" href="#r111c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r111c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r111c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-443"></div> <span class="help-tooltip" id="focus-panel-help-443">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r111c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r111c1-clusterblast" data-name="r111c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r111c1-knownclusterblast" data-name="r111c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r111c1-subclusterblast" data-name="r111c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r111c1-MIBiG-cluster-compare" data-name="r111c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r111c1-pfam-details" data-name="r111c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r111c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-185"></div> <span class="help-tooltip" id="cb-general-help-185">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r111c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r111c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r111c1_1.svg">
        NZ_CP029193 (6504493-6515167): Streptomyces venezuelae strain ATCC 14583 chro... (83% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_2.svg">
        NZ_CP029617 (6504552-6515058): Streptomyces sp. WAC 01529 chromosome, complet... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_3.svg">
        NZ_CP022685 (7219937-7231236): Streptomyces formicae strain KY5 chromosome, c... (81% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_4.svg">
        NZ_CP029191 (6340536-6350722): Streptomyces venezuelae strain ATCC 14585 chro... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_5.svg">
        NZ_CP023695 (6359441-6370498): Streptomyces alboniger strain ATCC 12461 chrom... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_6.svg">
        NZ_CP015588 (2193291-2204119): Streptomyces alfalfae strain ACCC40021 chromos... (81% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_7.svg">
        NZ_KQ948928 (357605-368973): Streptomyces caeruleatus strain NRRL B-24802 PRJ... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_8.svg">
        NZ_KQ947987 (329990-341352): Streptomyces curacoi strain DSM 40107 PRJNA29921... (72% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_9.svg">
        NZ_CP029788 (6438409-6447776): Streptomyces actuosus strain ATCC 25421 chromo... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r111c1_10.svg">
        NZ_KE150426 (2314006-2325314): Streptomyces sp. HGB0020 aczIR-supercont1.1, w... (80% of genes show similarity), bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r111c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r111c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r111c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-186"></div> <span class="help-tooltip" id="cb-known-help-186">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r111c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-187"></div> <span class="help-tooltip" id="cb-sub-help-187">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r111c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-188"></div> <span class="help-tooltip" id="MIBiG-help-188">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r111c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r111c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r111c1"></div>
    <div id="comparison-MIBiG-r111c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>NRP, Polyketide</td>
        
        <td>burkholderic acid</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000783.1: 0-13186">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000783/index.html#r1c1" target="_new">BGC0000783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>O-antigen</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000785.1: 151-19915">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000785/index.html#r1c1" target="_new">BGC0000785.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>O-antigen</td>
        
        
        <td>Burkholderia mallei</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000782.1: 0-23182">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000782/index.html#r1c1" target="_new">BGC0000782.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>O-antigen</td>
        
        
        <td>Burkholderia pseudomallei 1026b</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>CC-1065</td>
        
        
        <td>Streptomyces zelensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
        
        
        <td>Actinomadura sp. ATCC 39365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000775.1: 302-32661">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000775/index.html#r1c1" target="_new">BGC0000775.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>lipopolysaccharide</td>
        
        
        <td>Legionella pneumophila</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000795.1: 2265-27605">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000795/index.html#r1c1" target="_new">BGC0000795.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Aneurinibacillus thermoaerophilus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002005.1: 1285023-1291221">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002005/index.html#r1c1" target="_new">BGC0002005.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>RaxX</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r111c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>NRP, Polyketide</td>
      <td>burkholderic acid</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000783.1: 0-13186">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000783/index.html#r1c1" target="_new">BGC0000783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>O-antigen</td>
      <td>Xanthomonas oryzae pv. oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000785.1: 151-19915">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000785/index.html#r1c1" target="_new">BGC0000785.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>O-antigen</td>
      <td>Burkholderia mallei</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000782.1: 0-23182">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000782/index.html#r1c1" target="_new">BGC0000782.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>O-antigen</td>
      <td>Burkholderia pseudomallei 1026b</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>CC-1065</td>
      <td>Streptomyces zelensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
      <td>Actinomadura sp. ATCC 39365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000775.1: 302-32661">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000775/index.html#r1c1" target="_new">BGC0000775.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>lipopolysaccharide</td>
      <td>Legionella pneumophila</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000795.1: 2265-27605">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000795/index.html#r1c1" target="_new">BGC0000795.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Aneurinibacillus thermoaerophilus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002005.1: 1285023-1291221">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002005/index.html#r1c1" target="_new">BGC0002005.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>RiPP</td>
      <td>RaxX</td>
      <td>Xanthomonas oryzae pv. oryzae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r111c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-189"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-189">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r111c1-pfam-details-svg" data-filename="c00111_NODE_23.._r111c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r111c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r118c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00118_NODE_11.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-444"></div> <span class="help-tooltip" id="region_svg-help-444">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00118_NODE_11...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r118c1-svg" data-filename="c00118_NODE_11.._r118c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 31,017 nt. (total: 31,017 nt)
        
        <a class="cluster-rules-header" id="r118c1-rules-header" href="#r118c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r118c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r118c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-445"></div> <span class="help-tooltip" id="focus-panel-help-445">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r118c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r118c1-nrps_pks" data-name="r118c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r118c1-clusterblast" data-name="r118c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r118c1-knownclusterblast" data-name="r118c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r118c1-subclusterblast" data-name="r118c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r118c1-MIBiG-cluster-compare" data-name="r118c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r118c1-pfam-details" data-name="r118c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r118c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-190"></div> <span class="help-tooltip" id="nrps-domain-help-190">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r118c1-details-svg" data-filename="c00118_NODE_11.._r118c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r118c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r118c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-193"></div> <span class="help-tooltip" id="cb-general-help-193">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r118c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r118c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r118c1_1.svg">
        NZ_CP029617 (414138-539988): Streptomyces sp. WAC 01529 chromosome, complete ... (11% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r118c1_2.svg">
        NZ_JAAMPJ010000010 (96471-149585): Lentzea sp. NEAU-D13 Scaffold10, whole gen... (11% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_3.svg">
        NZ_LIPD01000001 (118656-187598): Streptomyces sp. CB02056 scaffold1, whole ge... (4% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_4.svg">
        NZ_KB889967 (359610-472722): Methyloversatilis thermotolerans strain 3t A3Q7D... (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_5.svg">
        NZ_CP034353 (8030335-8076225): Streptomyces sp. KPB2 chromosome, complete genome (5% of genes show similarity), PKS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_6.svg">
        NZ_JHXQ01000006 (44122-99285): Rhizobium undicola ORS 992 = ATCC 700741 T424D... (8% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_7.svg">
        NZ_NNBP01000001 (366976-470841): Streptomyces sp. CB02959 scaffold1, whole ge... (4% of genes show similarity), NRPS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_8.svg">
        NZ_AP019620 (1490004-1618210): Streptomyces antimycoticus strain NBRC 100767 (23% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_9.svg">
        NZ_AMPN02000002 (6662429-6928181): Streptomyces sp. SM8 Scaffold 2, whole gen... (9% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r118c1_10.svg">
        NZ_AP019620 (7021301-7157676): Streptomyces antimycoticus strain NBRC 100767 (30% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r118c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r118c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r118c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-194"></div> <span class="help-tooltip" id="cb-known-help-194">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r118c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r118c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_1.svg">
        laidlomycin
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_2.svg">
        ECO-02301
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_3.svg">
        filipin
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_4.svg">
        pentamycin
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_5.svg">
        funisamine
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_6.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_7.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_8.svg">
        elaiophylin
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_9.svg">
        hitachimycin
      </option>
      
      <option value = "svg/knownclusterblast_r118c1_10.svg">
        natamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r118c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r118c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r118c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-195"></div> <span class="help-tooltip" id="cb-sub-help-195">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r118c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-196"></div> <span class="help-tooltip" id="MIBiG-help-196">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r118c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r118c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r118c1"></div>
    <div id="comparison-MIBiG-r118c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.51, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164.1: 2029234-2037562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Cyanothece sp. PCC 7822</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.51, order:0.76, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001013.1: 0-116855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001013/index.html#r1c1" target="_new">BGC0001013.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.51, order:1.00, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP, Polyketide</td>
        
        <td>meridamycin</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.56, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP, Polyketide</td>
        
        <td>antalid</td>
        
        
        <td>Polyangium spumosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens JHB</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001012.1: 7792-87277">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001012/index.html#r1c1" target="_new">BGC0001012.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.52, order:0.76, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP, Polyketide</td>
        
        <td>meridamycin</td>
        
        
        <td>Streptomyces violaceusniger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001051.1: 124-81344">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001051/index.html#r1c1" target="_new">BGC0001051.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.51, order:0.76, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP, Polyketide</td>
        
        <td>thuggacin A</td>
        
        
        <td>Sorangium cellulosum</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r118c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.51, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164.1: 2029234-2037562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Cyanothece sp. PCC 7822</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.51, order:0.76, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001013.1: 0-116855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001013/index.html#r1c1" target="_new">BGC0001013.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.51, order:1.00, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>meridamycin</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.56, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>antalid</td>
      <td>Polyangium spumosum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens JHB</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001012.1: 7792-87277">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001012/index.html#r1c1" target="_new">BGC0001012.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.52, order:0.76, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP, Polyketide</td>
      <td>meridamycin</td>
      <td>Streptomyces violaceusniger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001051.1: 124-81344">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001051/index.html#r1c1" target="_new">BGC0001051.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.51, order:0.76, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP, Polyketide</td>
      <td>thuggacin A</td>
      <td>Sorangium cellulosum</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r118c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-197"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-197">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r118c1-pfam-details-svg" data-filename="c00118_NODE_11.._r118c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r118c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r118c1-nrps_pks_products" data-name="r118c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r118c1-nrps_pks" data-name="r118c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r118c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-191"></div> <span class="help-tooltip" id="nrps-structures-help-191">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 31017:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r118c1-1" class="smiles-canvas" data-smiles="C=CCC(O)CC(O)C(C)=CC(C)C(O)CC(O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (ohmal - ohmal) + (Me-ccmal - Me-ohmal - ohmal)</dd>
                
                    <dt><b>Non-colinear ordering:</b></dt><dd>
                    
                        <span class="serif">cds0_3885</span>
                    
                         + <span class="serif">cds4188_14049</span>
                    
                         + <span class="serif">cds14089_29326</span>
                    
                         + <span class="serif">cds29534_31016</span>
                    
                    </dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal&amp;nrps2=ohmal,ohmal&amp;nrps3=me-ccmal,me-ohmal,ohmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=ohmal*,ohmal*&amp;nrps3=me-ccmal*,me-ohmal*,ohmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r118c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C=CCC(O)CC(O)C(C)=CC(C)C(O)CC(O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r118c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-192"></div> <span class="help-tooltip" id="nrps-monomers-help-192">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_3885</span></strong>: mal
      <div class="collapser collapser-target-cds0_3885 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (1..265)</strong>: mal
          <div class="collapser collapser-target-cds0_3885 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds0_3885-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>inactive: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds0_3885-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 185.2</dt>

  <dd></dd><dt>inactive: 100.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 45.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 42.6</dt>

  <dd></dd><dt>Propionyl-CoA: 29.1</dt>

  <dd></dd><dt>Acetyl-CoA: 21.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 14.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.7</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 9.3</dt>

  <dd></dd><dt>fatty_acid: 8.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 6.8</dt>

  <dd></dd><dt>CHC-CoA: 6.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds4188_14049</span></strong>: mal - mal
      <div class="collapser collapser-target-cds4188_14049 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (566..858)</strong>: mal
          <div class="collapser collapser-target-cds4188_14049 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds4188_14049-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds4188_14049-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 207.4</dt>

  <dd></dd><dt>inactive: 111.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 49.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 48.4</dt>

  <dd></dd><dt>Propionyl-CoA: 37.0</dt>

  <dd></dd><dt>Acetyl-CoA: 22.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.5</dt>

  <dd></dd><dt>CHC-CoA: 9.8</dt>

  <dd></dd><dt>fatty_acid: 9.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 4.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2144..2433)</strong>: mal
          <div class="collapser collapser-target-cds4188_14049 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds4188_14049-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds4188_14049-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 197.0</dt>

  <dd></dd><dt>inactive: 98.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 43.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 40.9</dt>

  <dd></dd><dt>Propionyl-CoA: 28.1</dt>

  <dd></dd><dt>Acetyl-CoA: 20.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 14.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 8.8</dt>

  <dd></dd><dt>fatty_acid: 8.0</dt>

  <dd></dd><dt>CHC-CoA: 4.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds14089_29326</span></strong>: mmal - mmal - mal
      <div class="collapser collapser-target-cds14089_29326 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (578..872)</strong>: mmal
          <div class="collapser collapser-target-cds14089_29326 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds14089_29326-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds14089_29326-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 218.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 199.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 173.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 152.9</dt>

  <dd></dd><dt>Propionyl-CoA: 120.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 116.0</dt>

  <dd></dd><dt>CHC-CoA: 107.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 98.5</dt>

  <dd></dd><dt>Malonyl-CoA: 94.4</dt>

  <dd></dd><dt>Acetyl-CoA: 75.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 74.2</dt>

  <dd></dd><dt>fatty_acid: 49.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 39.5</dt>

  <dd></dd><dt>inactive: 25.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2431..2727)</strong>: mmal
          <div class="collapser collapser-target-cds14089_29326 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds14089_29326-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds14089_29326-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 217.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 198.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 168.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 141.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 124.0</dt>

  <dd></dd><dt>Propionyl-CoA: 121.7</dt>

  <dd></dd><dt>CHC-CoA: 101.4</dt>

  <dd></dd><dt>Malonyl-CoA: 101.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 92.9</dt>

  <dd></dd><dt>Acetyl-CoA: 71.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 69.5</dt>

  <dd></dd><dt>fatty_acid: 45.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 33.5</dt>

  <dd></dd><dt>inactive: 21.6</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (4000..4292)</strong>: mal
          <div class="collapser collapser-target-cds14089_29326 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds14089_29326-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds14089_29326-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 203.0</dt>

  <dd></dd><dt>inactive: 110.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 45.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 43.2</dt>

  <dd></dd><dt>Propionyl-CoA: 31.2</dt>

  <dd></dd><dt>Acetyl-CoA: 21.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.2</dt>

  <dd></dd><dt>fatty_acid: 9.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.0</dt>

  <dd></dd><dt>CHC-CoA: 7.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r123c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00123_NODE_15.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-446"></div> <span class="help-tooltip" id="region_svg-help-446">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00123_NODE_15...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r123c1-svg" data-filename="c00123_NODE_15.._r123c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 52,666 nt. (total: 52,666 nt)
        
        <a class="cluster-rules-header" id="r123c1-rules-header" href="#r123c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r123c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r123c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-447"></div> <span class="help-tooltip" id="focus-panel-help-447">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r123c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r123c1-nrps_pks" data-name="r123c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r123c1-clusterblast" data-name="r123c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r123c1-knownclusterblast" data-name="r123c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r123c1-subclusterblast" data-name="r123c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r123c1-MIBiG-cluster-compare" data-name="r123c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r123c1-pfam-details" data-name="r123c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r123c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-198"></div> <span class="help-tooltip" id="nrps-domain-help-198">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r123c1-details-svg" data-filename="c00123_NODE_15.._r123c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r123c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r123c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-201"></div> <span class="help-tooltip" id="cb-general-help-201">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r123c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r123c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r123c1_1.svg">
        NZ_GG657754 (6762597-6972752): Streptomyces himastatinicus ATCC 53653 superco... (29% of genes show similarity), NRPS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r123c1_2.svg">
        NZ_CP015726 (4048739-4258212): Streptomyces sp. RTd22 chromosome, complete ge... (31% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r123c1_3.svg">
        NZ_CP029541 (3625311-3671319): Streptomyces sp. NEAU-S7GS2 chromosome, comple... (31% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r123c1_4.svg">
        NZ_CP031455 (3920196-3994462): Streptomyces olivoreticuli subsp. olivoreticul... (16% of genes show similarity), T2PKS,melanin
      </option>
      
      <option value = "svg/clusterblast_r123c1_5.svg">
        NZ_CP029191 (5397372-5464392): Streptomyces venezuelae strain ATCC 14585 chro... (18% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r123c1_6.svg">
        NZ_CP029193 (5528262-5590885): Streptomyces venezuelae strain ATCC 14583 chro... (24% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r123c1_7.svg">
        NZ_KB897778 (287279-386421): Streptomyces sp. CNT372 C593DRAFT scaffold00003.... (14% of genes show similarity), NRPS-like,PKS-like,T2PKS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r123c1_8.svg">
        NZ_WWIF01000008 (210998-283559): Streptomyces sp. SID5594 SID5594.c8, whole g... (15% of genes show similarity), T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r123c1_9.svg">
        NZ_KZ819159 (2024473-2097034): Streptomyces sp. CS090A Scaffold00001, whole g... (15% of genes show similarity), T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r123c1_10.svg">
        NZ_CP029190 (4877974-4935017): Streptomyces venezuelae strain ATCC 21782 chro... (21% of genes show similarity), NRPS,other
      </option>
      
    </select>
    <button id= "clusterblast-r123c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r123c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r123c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-202"></div> <span class="help-tooltip" id="cb-known-help-202">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r123c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r123c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_1.svg">
        incednine
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_2.svg">
        apoptolidin
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_3.svg">
        lasalocid
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_4.svg">
        lasalocid
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_5.svg">
        coelimycin P1
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_6.svg">
        tiacumicin B
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_7.svg">
        simocyclinone D8
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_8.svg">
        oligomycin
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_9.svg">
        ECO-02301
      </option>
      
      <option value = "svg/knownclusterblast_r123c1_10.svg">
        nystatin A1
      </option>
      
    </select>
    <button id="knownclusterblast-r123c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r123c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r123c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-203"></div> <span class="help-tooltip" id="cb-sub-help-203">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r123c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-204"></div> <span class="help-tooltip" id="MIBiG-help-204">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r123c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r123c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r123c1"></div>
    <div id="comparison-MIBiG-r123c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>setomimycin</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001892.1: 5625-69449">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001892/index.html#r1c1" target="_new">BGC0001892.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>botcinic acid</td>
        
        
        <td>Botrytis cinerea B05.10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>4-epi-15-epi-brefeldin A</td>
        
        
        <td>Penicillium brefeldianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001334.1: 9570-30347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001334/index.html#r1c1" target="_new">BGC0001334.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Alkaloid</td>
        
        <td>caldoniamide A, cladoniamide B, cladoniamide C7, cladoniamide D, cladoniamide E, cladoniamide F, cladoniamide G</td>
        
        
        <td>Streptomyces uncialis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r123c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001892.1: 5625-69449">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001892/index.html#r1c1" target="_new">BGC0001892.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>botcinic acid</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>4-epi-15-epi-brefeldin A</td>
      <td>Penicillium brefeldianum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001334.1: 9570-30347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001334/index.html#r1c1" target="_new">BGC0001334.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Alkaloid</td>
      <td>caldoniamide A, cladoniamide B, cladoniamide C7, cladoniamide D, cladoniamide E, cladoniamide F, cladoniamide G</td>
      <td>Streptomyces uncialis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r123c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-205"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-205">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r123c1-pfam-details-svg" data-filename="c00123_NODE_15.._r123c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r123c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r123c1-nrps_pks_products" data-name="r123c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r123c1-nrps_pks" data-name="r123c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r123c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-199"></div> <span class="help-tooltip" id="nrps-structures-help-199">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 52666:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r123c1-1" class="smiles-canvas" data-smiles="CC(=O)C(C)=CC(C)=CCCC(C)=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal - Me-ccmal - Me-ccmal - redmal) + (Me-ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal,me-ccmal,me-ccmal,redmal&amp;nrps2=me-ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*,me-ccmal*,me-ccmal*,redmal*&amp;nrps2=me-ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r123c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(C)=CC(C)=CCCC(C)=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r123c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-200"></div> <span class="help-tooltip" id="nrps-monomers-help-200">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds4180_24166</span></strong>: mal - mmal - mmal - mal
      <div class="collapser collapser-target-cds4180_24166 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (604..882)</strong>: mal
          <div class="collapser collapser-target-cds4180_24166 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 141.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 59.6</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 55.1</dt>

  <dd></dd><dt>inactive: 54.1</dt>

  <dd></dd><dt>Propionyl-CoA: 36.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 28.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 24.8</dt>

  <dd></dd><dt>Acetyl-CoA: 23.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 14.6</dt>

  <dd></dd><dt>CHC-CoA: 14.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 6.7</dt>

  <dd></dd><dt>fatty_acid: 5.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1612..1907)</strong>: mmal
          <div class="collapser collapser-target-cds4180_24166 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 218.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 217.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 175.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 151.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 149.0</dt>

  <dd></dd><dt>CHC-CoA: 131.1</dt>

  <dd></dd><dt>Propionyl-CoA: 118.2</dt>

  <dd></dd><dt>Malonyl-CoA: 106.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 76.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 68.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 54.5</dt>

  <dd></dd><dt>Acetyl-CoA: 53.3</dt>

  <dd></dd><dt>fatty_acid: 48.8</dt>

  <dd></dd><dt>inactive: 32.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (3320..3615)</strong>: mmal
          <div class="collapser collapser-target-cds4180_24166 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 213.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 207.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 178.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 153.4</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 139.8</dt>

  <dd></dd><dt>Propionyl-CoA: 121.4</dt>

  <dd></dd><dt>CHC-CoA: 107.9</dt>

  <dd></dd><dt>Malonyl-CoA: 101.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 75.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 72.4</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 50.2</dt>

  <dd></dd><dt>Acetyl-CoA: 47.0</dt>

  <dd></dd><dt>fatty_acid: 44.2</dt>

  <dd></dd><dt>inactive: 25.4</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (5037..5344)</strong>: mal
          <div class="collapser collapser-target-cds4180_24166 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>inactive: 75.0%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds4180_24166-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 185.3</dt>

  <dd></dd><dt>inactive: 99.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 65.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 48.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 23.4</dt>

  <dd></dd><dt>fatty_acid: 22.7</dt>

  <dd></dd><dt>Propionyl-CoA: 19.2</dt>

  <dd></dd><dt>Acetyl-CoA: 18.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 17.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 16.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 7.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds24176_32666</span></strong>: mmal
      <div class="collapser collapser-target-cds24176_32666 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (550..845)</strong>: mmal
          <div class="collapser collapser-target-cds24176_32666 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds24176_32666-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds24176_32666-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 215.7</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 208.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 173.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 158.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 146.9</dt>

  <dd></dd><dt>Propionyl-CoA: 117.3</dt>

  <dd></dd><dt>CHC-CoA: 115.4</dt>

  <dd></dd><dt>Malonyl-CoA: 103.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 77.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 70.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 55.5</dt>

  <dd></dd><dt>fatty_acid: 50.0</dt>

  <dd></dd><dt>Acetyl-CoA: 42.0</dt>

  <dd></dd><dt>inactive: 33.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r125c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00125_NODE_15.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-448"></div> <span class="help-tooltip" id="region_svg-help-448">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00125_NODE_15...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r125c1-svg" data-filename="c00125_NODE_15.._r125c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 5,094 - 22,215 nt. (total: 17,122 nt)
        
        <a class="cluster-rules-header" id="r125c1-rules-header" href="#r125c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r125c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r125c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-449"></div> <span class="help-tooltip" id="focus-panel-help-449">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r125c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r125c1-clusterblast" data-name="r125c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r125c1-knownclusterblast" data-name="r125c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r125c1-subclusterblast" data-name="r125c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r125c1-MIBiG-cluster-compare" data-name="r125c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r125c1-pfam-details" data-name="r125c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r125c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-206"></div> <span class="help-tooltip" id="cb-general-help-206">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r125c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r125c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r125c1_1.svg">
        NZ_CP023690 (3125709-3144567): Streptomyces spectabilis strain ATCC 27465 chr... (72% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r125c1_2.svg">
        NZ_CP040916 (2904513-2925585): Streptomyces spectabilis strain NRRL 2792 chro... (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r125c1_3.svg">
        NC_020504 (6741826-6817879): Streptomyces davaonensis JCM 4913, complete genome (22% of genes show similarity), NRPS,PKS-like
      </option>
      
      <option value = "svg/clusterblast_r125c1_4.svg">
        NZ_CP021744 (&lt;5418662-5445565): Streptomyces albireticuli strain MDJK11 chrom... (56% of genes show similarity), lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r125c1_5.svg">
        NZ_CP022685 (2833866-2856248): Streptomyces formicae strain KY5 chromosome, c... (55% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r125c1_6.svg">
        NZ_AUBE01000003 (115783-151323): Streptomyces flavidovirens DSM 40150 G412DRA... (31% of genes show similarity), CDPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r125c1_7.svg">
        NC_015957 (8459430-8479625): Streptomyces violaceusniger Tu 4113, complete se... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r125c1_8.svg">
        NZ_CP019458 (3722032-3742281): Streptomyces autolyticus strain CGMCC0516 chro... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r125c1_9.svg">
        NZ_LJIW01000001 (3746237-3766620): Streptomyces malaysiensis strain F913 SMF9... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r125c1_10.svg">
        NZ_CP029823 (7747063-7767570): Streptomyces malaysiensis strain DSM 4137 chro... (66% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r125c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r125c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r125c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-207"></div> <span class="help-tooltip" id="cb-known-help-207">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r125c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-208"></div> <span class="help-tooltip" id="cb-sub-help-208">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r125c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-209"></div> <span class="help-tooltip" id="MIBiG-help-209">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r125c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r125c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r125c1"></div>
    <div id="comparison-MIBiG-r125c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene, Polyketide</td>
        
        <td>aurachin RE</td>
        
        
        <td>Rhodococcus erythropolis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000823.1: 1-16912">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000823/index.html#r1c1" target="_new">BGC0000823.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Alkaloid</td>
        
        <td>rebeccamycin</td>
        
        
        <td>Lechevalieria aerocolonigenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
        
        
        <td>Synechocystis salina LEGE 06099</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525.1: 0-16392">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>bartoloside 2, bartoloside 3, bartoloside 4</td>
        
        
        <td>Synechocystis salina LEGE 06155</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>citrulassin E</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000822.1: 1-25681">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000822/index.html#r1c1" target="_new">BGC0000822.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Alkaloid</td>
        
        <td>rebeccamycin</td>
        
        
        <td>Lechevalieria aerocolonigenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866.1: 467-5165">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>polyhydroxyalkanoate</td>
        
        
        <td>Burkholderia sp. DSM 9242</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r125c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene, Saccharide</td>
      <td>brasilicardin A</td>
      <td>Nocardia terpenica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene, Polyketide</td>
      <td>aurachin RE</td>
      <td>Rhodococcus erythropolis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000823.1: 1-16912">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000823/index.html#r1c1" target="_new">BGC0000823.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Alkaloid</td>
      <td>rebeccamycin</td>
      <td>Lechevalieria aerocolonigenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
      <td>Synechocystis salina LEGE 06099</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525.1: 0-16392">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>bartoloside 2, bartoloside 3, bartoloside 4</td>
      <td>Synechocystis salina LEGE 06155</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>citrulassin E</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000822.1: 1-25681">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000822/index.html#r1c1" target="_new">BGC0000822.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Alkaloid</td>
      <td>rebeccamycin</td>
      <td>Lechevalieria aerocolonigenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866.1: 467-5165">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>polyhydroxyalkanoate</td>
      <td>Burkholderia sp. DSM 9242</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r125c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-210"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-210">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r125c1-pfam-details-svg" data-filename="c00125_NODE_15.._r125c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r125c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r137c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00137_NODE_61.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-450"></div> <span class="help-tooltip" id="region_svg-help-450">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00137_NODE_61...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r137c1-svg" data-filename="c00137_NODE_61.._r137c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 23,305 nt. (total: 23,305 nt)
        
        <a class="cluster-rules-header" id="r137c1-rules-header" href="#r137c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r137c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r137c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-451"></div> <span class="help-tooltip" id="focus-panel-help-451">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r137c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r137c1-nrps_pks" data-name="r137c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r137c1-clusterblast" data-name="r137c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r137c1-knownclusterblast" data-name="r137c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r137c1-subclusterblast" data-name="r137c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r137c1-MIBiG-cluster-compare" data-name="r137c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r137c1-pfam-details" data-name="r137c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r137c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-211"></div> <span class="help-tooltip" id="nrps-domain-help-211">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r137c1-details-svg" data-filename="c00137_NODE_61.._r137c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r137c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r137c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-212"></div> <span class="help-tooltip" id="cb-general-help-212">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r137c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r137c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r137c1_1.svg">
        NZ_CP027022 (3980567-4051552): Streptomyces sp. WAC00288 chromosome, complete... (11% of genes show similarity), NRPS,phenazine
      </option>
      
      <option value = "svg/clusterblast_r137c1_2.svg">
        NZ_SMBL01000016 (50153-104530): Streptomyces sp. BK215 Ga0307669 116, whole g... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r137c1_3.svg">
        NZ_AP019620 (6652484-6695770): Streptomyces antimycoticus strain NBRC 100767 (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r137c1_4.svg">
        NZ_CP046173 (4130273-4254163): Nocardia terpenica strain AUSMDU00012715 chrom... (3% of genes show similarity), NRPS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r137c1_5.svg">
        NZ_QQAZ01000006 (479639-531911): Nocardia mexicana strain DSM 44952 Ga0244508... (5% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r137c1_6.svg">
        NZ_CP046905 (211266-&gt;272801): Streptomyces sp. QHH-9511 chromosome, complete ... (3% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r137c1_7.svg">
        NZ_CP030073 (551432-624943): Streptomyces sp. ZFG47 chromosome, complete genome (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_8.svg">
        NZ_CP010519 (118830-165724): Streptomyces albus strain DSM 41398 chromosome, ... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r137c1_9.svg">
        NZ_GG657750 (3585311-3627340): Streptomyces sp. C supercont1.1, whole genome ... (9% of genes show similarity), butyrolactone,lanthidin
      </option>
      
      <option value = "svg/clusterblast_r137c1_10.svg">
        NZ_LISY01000001 (867422-912543): Streptomyces sp. TSRI0261 scaffold1, whole g... (12% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r137c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r137c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r137c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-213"></div> <span class="help-tooltip" id="cb-known-help-213">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r137c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r137c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_1.svg">
        incednine
      </option>
      
    </select>
    <button id="knownclusterblast-r137c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r137c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r137c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-214"></div> <span class="help-tooltip" id="cb-sub-help-214">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r137c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-215"></div> <span class="help-tooltip" id="MIBiG-help-215">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r137c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r137c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r137c1"></div>
    <div id="comparison-MIBiG-r137c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r137c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r137c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-216"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-216">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r137c1-pfam-details-svg" data-filename="c00137_NODE_61.._r137c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r137c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r146c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00146_NODE_40.. - Region 1 - NRPS,other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-452"></div> <span class="help-tooltip" id="region_svg-help-452">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00146_NODE_40...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r146c1-svg" data-filename="c00146_NODE_40.._r146c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 11,957 - 74,423 nt. (total: 62,467 nt)
        
        <a class="cluster-rules-header" id="r146c1-rules-header" href="#r146c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r146c1-rules">other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r146c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-453"></div> <span class="help-tooltip" id="focus-panel-help-453">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r146c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r146c1-nrps_pks" data-name="r146c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r146c1-clusterblast" data-name="r146c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r146c1-knownclusterblast" data-name="r146c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r146c1-subclusterblast" data-name="r146c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r146c1-MIBiG-cluster-compare" data-name="r146c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r146c1-pfam-details" data-name="r146c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r146c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-217"></div> <span class="help-tooltip" id="nrps-domain-help-217">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r146c1-details-svg" data-filename="c00146_NODE_40.._r146c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r146c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r146c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-220"></div> <span class="help-tooltip" id="cb-general-help-220">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r146c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r146c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r146c1_1.svg">
        NZ_CP016279 (2486499-2611542): Streptomyces griseochromogenes strain ATCC 145... (30% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_2.svg">
        NZ_KB913036 (1804528-1951486): Salinispora arenicola DSM 45545 strain CNS-991... (20% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_3.svg">
        NZ_CP029617 (414138-539988): Streptomyces sp. WAC 01529 chromosome, complete ... (19% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_4.svg">
        NZ_FNVU01000001 (224642-328704): Streptomyces yanglinensis strain CGMCC 4.202... (25% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_5.svg">
        NZ_CP017248 (2722891-2862053): Streptomyces fodineus strain TW1S1 chromosome,... (16% of genes show similarity), NRPS,T1PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_6.svg">
        NZ_CP020700 (307-229249): Streptomyces tsukubensis strain NRRL 18488 chromoso... (12% of genes show similarity), NRPS,T1PKS,butyrolactone,ectoine,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_7.svg">
        NZ_KB900614 (3242399-3312809): Salinispora pacifica DSM 45549 = CNT-150 strai... (26% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_8.svg">
        NZ_CP011862 (7712-223580): Pseudonocardia sp. AL041005-10 chromosome, complet... (12% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_9.svg">
        NC_008278 (2737802-2835803): Frankia alni ACN14a, complete genome (23% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r146c1_10.svg">
        NZ_CP042266 (6918091-7295107): Streptomyces qinzhouensis strain SSL-25 chromo... (11% of genes show similarity), NRPS,NRPS-like,T1PKS,other,terpene,transAT-PKS
      </option>
      
    </select>
    <button id= "clusterblast-r146c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r146c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r146c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-221"></div> <span class="help-tooltip" id="cb-known-help-221">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r146c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r146c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_1.svg">
        polyoxypeptin
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_2.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_3.svg">
        salinosporamide A
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_4.svg">
        cinnabaramide A
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_5.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_6.svg">
        sarpeptin A / sarpeptin B
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_7.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_8.svg">
        SW-163C / UK-63598 / SW-163E / SW-163F / SW-163G
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_9.svg">
        triostin A
      </option>
      
      <option value = "svg/knownclusterblast_r146c1_10.svg">
        glycinocin A
      </option>
      
    </select>
    <button id="knownclusterblast-r146c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r146c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r146c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-222"></div> <span class="help-tooltip" id="cb-sub-help-222">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r146c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r146c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r146c1_1.svg">
        NC_009380_c1: salinosporamide cyclohexenylalanine (50% of genes show similarity), cyclohexenylalanine
      </option>
      
    </select>
    <button id="subclusterblast-r146c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r146c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r146c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-223"></div> <span class="help-tooltip" id="MIBiG-help-223">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r146c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r146c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r146c1"></div>
    <div id="comparison-MIBiG-r146c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">other</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.43, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.43, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.74</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.62</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.60</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.60</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:0.83, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>nematophin</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001698.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001698/index.html#r1c1" target="_new">BGC0001698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:0.83, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>nevaltophin A, nevaltophin B, nevaltophin C, nevaltophin D</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:0.83, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>Jessenipeptin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000416.1: 2154723-2170060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000416/index.html#r1c1" target="_new">BGC0000416.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:0.83, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.56</td>
        <td>NRP</td>
        
        <td>rhabdopeptide 1, rhabdopeptide 2, rhabdopeptide 3, rhabdopeptide 4</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:0.69, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.54</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.54</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r146c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.43, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>mirubactin</td>
      <td>Actinosynnema mirum DSM 43827</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>nematophin</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001698.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001698/index.html#r1c1" target="_new">BGC0001698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>nevaltophin A, nevaltophin B, nevaltophin C, nevaltophin D</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000416.1: 2154723-2170060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000416/index.html#r1c1" target="_new">BGC0000416.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>rhabdopeptide 1, rhabdopeptide 2, rhabdopeptide 3, rhabdopeptide 4</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.50, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>erythrochelin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:0.83, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>Jessenipeptin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r146c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-224"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-224">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r146c1-pfam-details-svg" data-filename="c00146_NODE_40.._r146c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r146c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r146c1-nrps_pks_products" data-name="r146c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r146c1-nrps_pks" data-name="r146c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r146c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-218"></div> <span class="help-tooltip" id="nrps-structures-help-218">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 11956 - 74423:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r146c1-1" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-val) + (D-X - ser) + (D-X - ala)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val&amp;nrps2=d-x,ser&amp;nrps3=d-x,ala" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val*&amp;nrps2=d-x*,ser*&amp;nrps3=d-x*,ala*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r146c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 16016 - 74423:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r146c1-3" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-val) + (D-X - ser) + (D-X - ala)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val&amp;nrps2=d-x,ser&amp;nrps3=d-x,ala" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-val*&amp;nrps2=d-x*,ser*&amp;nrps3=d-x*,ala*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r146c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r146c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-219"></div> <span class="help-tooltip" id="nrps-monomers-help-219">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds32624_34211</span></strong>: X
      <div class="collapser collapser-target-cds32624_34211 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (19..425)</strong>: X
          <div class="collapser collapser-target-cds32624_34211 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp, asn
              <div class="collapser collapser-target-cds32624_34211-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DlRHlGqdvK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds36016_42415</span></strong>: val
      <div class="collapser collapser-target-cds36016_42415 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (475..877)</strong>: val
          <div class="collapser collapser-target-cds36016_42415 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds36016_42415-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAkWWGGvFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds42461_50246</span></strong>: X - ser
      <div class="collapser collapser-target-cds42461_50246 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,ser" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,ser*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (465..862)</strong>: X
          <div class="collapser collapser-target-cds42461_50246 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds42461_50246-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSvAsfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2017..2396)</strong>: ser
          <div class="collapser collapser-target-cds42461_50246 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds42461_50246-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVpdeAlVeK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds50242_59200</span></strong>: X - ala
      <div class="collapser collapser-target-cds50242_59200 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,ala" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,ala*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (465..864)</strong>: X
          <div class="collapser collapser-target-cds50242_59200 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp
              <div class="collapser collapser-target-cds50242_59200-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DqWTVggVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2401..2790)</strong>: ala
          <div class="collapser collapser-target-cds50242_59200 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds50242_59200-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DiFhVAAfyK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds59994_61422</span></strong>: X
      <div class="collapser collapser-target-cds59994_61422 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (98..354)</strong>: X
          <div class="collapser collapser-target-cds59994_61422 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds59994_61422-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">ct-MnGlVlK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r147c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00147_NODE_65.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-454"></div> <span class="help-tooltip" id="region_svg-help-454">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00147_NODE_65...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r147c1-svg" data-filename="c00147_NODE_65.._r147c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 45,354 nt. (total: 45,354 nt)
        
        <a class="cluster-rules-header" id="r147c1-rules-header" href="#r147c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r147c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r147c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-455"></div> <span class="help-tooltip" id="focus-panel-help-455">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r147c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r147c1-nrps_pks" data-name="r147c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r147c1-clusterblast" data-name="r147c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r147c1-knownclusterblast" data-name="r147c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r147c1-subclusterblast" data-name="r147c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r147c1-MIBiG-cluster-compare" data-name="r147c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r147c1-pfam-details" data-name="r147c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r147c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-225"></div> <span class="help-tooltip" id="nrps-domain-help-225">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r147c1-details-svg" data-filename="c00147_NODE_65.._r147c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r147c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r147c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-228"></div> <span class="help-tooltip" id="cb-general-help-228">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r147c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r147c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r147c1_1.svg">
        NZ_CP040916 (1044929-1137636): Streptomyces spectabilis strain NRRL 2792 chro... (11% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_2.svg">
        NZ_CP023690 (1228599-1289798): Streptomyces spectabilis strain ATCC 27465 chr... (18% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r147c1_3.svg">
        NZ_CP030771 (1151977-1233127): Streptomyces sp. YIM 121038 chromosome, comple... (14% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r147c1_4.svg">
        NZ_CP014485 (284615-359723): Streptomyces albidoflavus strain SM254 chromosom... (12% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_5.svg">
        NZ_CP047147 (7014845-7092276): Streptomyces sp. GF20 chromosome, complete genome (12% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_6.svg">
        NZ_KB898227 (22855-99639): Streptomyces sp. CNY228 D330DRAFT scaffold00007.7,... (12% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_7.svg">
        NZ_CP056773 (227001-302717): Streptomyces violascens strain YIM 100212 chromo... (12% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_8.svg">
        NZ_CP016824 (6776442-6853141): Streptomyces sampsonii strain KJ40 chromosome,... (12% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_9.svg">
        NZ_CM002273 (276381-349293): Streptomyces sp. PVA 94-07 strain PVA 94-07 chro... (13% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r147c1_10.svg">
        NZ_AZYD01000013 (750907-828267): Streptomyces sp. LaPpAH-201 K372DRAFT scaffo... (12% of genes show similarity), NRPS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r147c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r147c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r147c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-229"></div> <span class="help-tooltip" id="cb-known-help-229">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r147c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r147c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_1.svg">
        butyrolactol A
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_2.svg">
        chondrochloren A
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_3.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_4.svg">
        pellasoren
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_5.svg">
        nocathiacin
      </option>
      
    </select>
    <button id="knownclusterblast-r147c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r147c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r147c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-230"></div> <span class="help-tooltip" id="cb-sub-help-230">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r147c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-231"></div> <span class="help-tooltip" id="MIBiG-help-231">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r147c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r147c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r147c1"></div>
    <div id="comparison-MIBiG-r147c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.43, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>salinichelins</td>
        
        
        <td>Salinispora pacifica CNY331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001562.1: 314040-333483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001562/index.html#r1c1" target="_new">BGC0001562.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>monobactam</td>
        
        
        <td>Agrobacterium tumefaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>erythrochelin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000396.1: 320-14288">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000396/index.html#r1c1" target="_new">BGC0000396.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>nodularin</td>
        
        
        <td>Nostoc sp. 73.1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r147c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.43, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>salinichelins</td>
      <td>Salinispora pacifica CNY331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>mirubactin</td>
      <td>Actinosynnema mirum DSM 43827</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001562.1: 314040-333483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001562/index.html#r1c1" target="_new">BGC0001562.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>monobactam</td>
      <td>Agrobacterium tumefaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>erythrochelin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000396.1: 320-14288">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000396/index.html#r1c1" target="_new">BGC0000396.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>nodularin</td>
      <td>Nostoc sp. 73.1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r147c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-232"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-232">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r147c1-pfam-details-svg" data-filename="c00147_NODE_65.._r147c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r147c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r147c1-nrps_pks_products" data-name="r147c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r147c1-nrps_pks" data-name="r147c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r147c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-226"></div> <span class="help-tooltip" id="nrps-structures-help-226">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 45354:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r147c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC(C)C(=O)NC(C(C)C)C(=O)CC(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - D-X - ala - val) + (mal - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,d-x,ala,val&amp;nrps2=mal,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,d-x*,ala*,val*&amp;nrps2=mal*,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r147c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC(C)C(=O)NC(C(C)C)C(=O)CC(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r147c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-227"></div> <span class="help-tooltip" id="nrps-monomers-help-227">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2639_18263</span></strong>: X - X - ala - val
      <div class="collapser collapser-target-cds2639_18263 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X,ala,val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*,ala*,val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (810..1209)</strong>: X
          <div class="collapser collapser-target-cds2639_18263 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds2639_18263-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvyFaGaAiK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1868..2278)</strong>: X
          <div class="collapser collapser-target-cds2639_18263 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds2639_18263-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAyLwGAVaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3484..3893)</strong>: ala
          <div class="collapser collapser-target-cds2639_18263 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds2639_18263-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DifSiAgyYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4559..4964)</strong>: val
          <div class="collapser collapser-target-cds2639_18263 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds2639_18263-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DmFLWGgvfK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds18259_25354</span></strong>: X
      <div class="collapser collapser-target-cds18259_25354 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1325..1725)</strong>: X
          <div class="collapser collapser-target-cds18259_25354 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds18259_25354-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMLcyGLffK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser-thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r151c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00151_NODE_83.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-456"></div> <span class="help-tooltip" id="region_svg-help-456">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00151_NODE_83...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r151c1-svg" data-filename="c00151_NODE_83.._r151c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 40,488 nt. (total: 40,488 nt)
        
        <a class="cluster-rules-header" id="r151c1-rules-header" href="#r151c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r151c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r151c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-457"></div> <span class="help-tooltip" id="focus-panel-help-457">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r151c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r151c1-nrps_pks" data-name="r151c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r151c1-clusterblast" data-name="r151c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r151c1-knownclusterblast" data-name="r151c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r151c1-subclusterblast" data-name="r151c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r151c1-MIBiG-cluster-compare" data-name="r151c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r151c1-pfam-details" data-name="r151c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r151c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-233"></div> <span class="help-tooltip" id="nrps-domain-help-233">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r151c1-details-svg" data-filename="c00151_NODE_83.._r151c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r151c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r151c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-234"></div> <span class="help-tooltip" id="cb-general-help-234">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r151c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r151c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r151c1_1.svg">
        NZ_CP034463 (6731864-6804326): Streptomyces aquilus strain GGCR-6 chromosome,... (21% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r151c1_2.svg">
        NZ_SLZH01000003 (377420-449882): Streptomyces sp. BK042 Ga0307683 103, whole ... (21% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r151c1_3.svg">
        NZ_CP022685 (5330727-5382893): Streptomyces formicae strain KY5 chromosome, c... (24% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r151c1_4.svg">
        NZ_RSFG01000002 (2738628-2793449): Streptomyces sp. CB01881 Scaffold2, whole ... (28% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r151c1_5.svg">
        NZ_CP016438 (8239464-8311926): Streptomyces lincolnensis strain NRRL 2936 chr... (13% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r151c1_6.svg">
        NZ_CP021978 (4989917-5086183): Streptomyces hawaiiensis strain ATCC 12236 chr... (8% of genes show similarity), NRPS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r151c1_7.svg">
        NZ_CP053189 (7556758-7629244): Streptomyces sp. jing01 chromosome, complete g... (16% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r151c1_8.svg">
        NZ_CP011112 (1193322-1265795): Luteipulveratus mongoliensis strain MN07-A0370... (14% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r151c1_9.svg">
        NZ_KB896769 (90164-162650): Salinispora pacifica CNT584 C582DRAFT scaffold000... (12% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r151c1_10.svg">
        NZ_CP010407 (6327232-6440759): Streptomyces vietnamensis strain GIMV4.0001 ch... (13% of genes show similarity), NRPS,T1PKS,T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r151c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r151c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r151c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-235"></div> <span class="help-tooltip" id="cb-known-help-235">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r151c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r151c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_1.svg">
        setomimycin
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_2.svg">
        frigocyclinone
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_3.svg">
        mayamycin
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_4.svg">
        fluostatins M-Q
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_5.svg">
        kinamycin
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_6.svg">
        auricin
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_7.svg">
        lugdunomycin
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_8.svg">
        julichrome Q3-3 / julichrome Q3-5
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_9.svg">
        prejadomycin / rabelomycin / gaudimycin C / gaudimycin D / UWM6 / gaudimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r151c1_10.svg">
        saprolmycin E
      </option>
      
    </select>
    <button id="knownclusterblast-r151c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r151c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r151c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-236"></div> <span class="help-tooltip" id="cb-sub-help-236">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r151c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r151c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r151c1_1.svg">
        AF080235_c1: landomycin t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r151c1_2.svg">
        AF324838_c1: simocyclinone t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r151c1_3.svg">
        GU233672_c1: granaticin t2pks (71% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r151c1_4.svg">
        FJ483966_c1: polyketomycin t2pks (37% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r151c1_5.svg">
        AB103463_c1: medermycin t2pks (75% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r151c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r151c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r151c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-238"></div> <span class="help-tooltip" id="MIBiG-help-238">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r151c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r151c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r151c1"></div>
    <div id="comparison-MIBiG-r151c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.82 (id:0.56, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
         
        
        <td>0.57</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.66, order:0.75, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.49</td>
        <td>Polyketide</td>
        
        <td>setomimycin</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.53, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.85, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Polyketide</td>
        
        <td>oviedomycin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.62, order:0.46, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
        
        
        <td>Streptomyces sp. CB09001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000235.1: 118281-139606">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000235/index.html#r1c1" target="_new">BGC0000235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.56, order:0.46, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>kiamycin</td>
        
        
        <td>Streptomyces sp. W007</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.53, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.51, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r151c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.82 (id:0.56, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
      <td>0.82</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.66, order:0.75, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.53, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.85, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Polyketide</td>
      <td>oviedomycin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.62, order:0.46, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
      <td>Streptomyces sp. CB09001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000235.1: 118281-139606">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000235/index.html#r1c1" target="_new">BGC0000235.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.56, order:0.46, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>kiamycin</td>
      <td>Streptomyces sp. W007</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.53, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.51, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r151c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-239"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-239">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r151c1-pfam-details-svg" data-filename="c00151_NODE_83.._r151c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r151c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r151c1-t2pks" data-name="r151c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r151c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-237"></div> <span class="help-tooltip" id="t2pks-help-237">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 40488</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 680.5; E-value: 1.5e-207)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				angucycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			acetyl-CoA_8: 333.36 Da<br>
			
			acetyl-CoA_9: 375.39 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds30740_31697</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 442.8; E-value: 1.2e-135)<br>
			</dd>
			
		
		<dt><strong>cds31750_32548</strong></dt>
			
			<dd>
				KR
				
					C9
				
				(Score: 468.2; E-value: 1.6e-143)<br>
			</dd>
			
		
		<dt><strong>cds32750_33017</strong></dt>
			
			<dd>
				ACP
				
				(Score: 122.6; E-value: 1.1e-38)<br>
			</dd>
			
		
		<dt><strong>cds33130_34324</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 680.5; E-value: 1.5e-207)<br>
			</dd>
			
		
		<dt><strong>cds34320_35592</strong></dt>
			
			<dd>
				KS
				
				(Score: 777.1; E-value: 8.8e-237)<br>
			</dd>
			
		
		<dt><strong>cds35588_36236</strong></dt>
			
			<dd>
				CYC
				
					C4-C17/C2-C19
				
				(Score: 166.1; E-value: 1.8e-51)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r155c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00155_NODE_7_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-458"></div> <span class="help-tooltip" id="region_svg-help-458">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00155_NODE_7_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r155c1-svg" data-filename="c00155_NODE_7_.._r155c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 49,163 - 99,464 nt. (total: 50,302 nt)
        
        <a class="cluster-rules-header" id="r155c1-rules-header" href="#r155c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r155c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r155c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-459"></div> <span class="help-tooltip" id="focus-panel-help-459">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r155c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r155c1-nrps_pks" data-name="r155c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r155c1-clusterblast" data-name="r155c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r155c1-knownclusterblast" data-name="r155c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r155c1-subclusterblast" data-name="r155c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r155c1-MIBiG-cluster-compare" data-name="r155c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r155c1-pfam-details" data-name="r155c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r155c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-240"></div> <span class="help-tooltip" id="nrps-domain-help-240">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r155c1-details-svg" data-filename="c00155_NODE_7_.._r155c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r155c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r155c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-243"></div> <span class="help-tooltip" id="cb-general-help-243">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r155c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r155c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r155c1_1.svg">
        NZ_CP023690 (9088805-9182750): Streptomyces spectabilis strain ATCC 27465 chr... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_2.svg">
        NZ_QQBC01000012 (69683-164300): Nocardia pseudobrasiliensis strain DSM 44290 ... (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_3.svg">
        NC_021252 (3843911-3910910): Amycolatopsis keratiniphila, complete sequence (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_4.svg">
        NZ_LT629789 (2455944-2523039): Amycolatopsis keratiniphila strain FH 1893 chr... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_5.svg">
        NZ_LIWC01000001 (483262-550545): Amycolatopsis sp. CB00013 scaffold1, whole g... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_6.svg">
        NZ_JPLW01000002 (303447-370284): Amycolatopsis sp. MJM2582 contig00002, whole... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_7.svg">
        NZ_CP008953 (4531236-4599825): Amycolatopsis japonica strain DSM 44213 chromo... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r155c1_8.svg">
        NZ_CP045740 (1446009-1590356): Streptomyces sp. SUK 48 chromosome, complete g... (10% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r155c1_9.svg">
        NZ_CP029193 (516648-621296): Streptomyces venezuelae strain ATCC 14583 chromo... (11% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r155c1_10.svg">
        NZ_CP017780 (3436357-3502028): Amycolatopsis sp. BJA-103 chromosome, complete... (16% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r155c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r155c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r155c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-244"></div> <span class="help-tooltip" id="cb-known-help-244">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r155c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r155c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_1.svg">
        mirubactin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_2.svg">
        thiolutin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_3.svg">
        heterobactin A / heterobactin S2
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_4.svg">
        herboxidiene
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_5.svg">
        streptobactin
      </option>
      
    </select>
    <button id="knownclusterblast-r155c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r155c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r155c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-245"></div> <span class="help-tooltip" id="cb-sub-help-245">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r155c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-246"></div> <span class="help-tooltip" id="MIBiG-help-246">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r155c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r155c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r155c1"></div>
    <div id="comparison-MIBiG-r155c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.58, order:0.57, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.64, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441.1: 0-27983">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>belactosin A, belactosin C</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343.1: 0-34054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>enterobactin</td>
        
        
        <td>Pseudomonas sp. J465</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>erythrochelin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000294.1: 181-31792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000294/index.html#r1c1" target="_new">BGC0000294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>acinetobactin</td>
        
        
        <td>Acinetobacter baumannii</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r155c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.58, order:0.57, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>mirubactin</td>
      <td>Actinosynnema mirum DSM 43827</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.64, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441.1: 0-27983">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>belactosin A, belactosin C</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343.1: 0-34054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>enterobactin</td>
      <td>Pseudomonas sp. J465</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>erythrochelin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000294.1: 181-31792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000294/index.html#r1c1" target="_new">BGC0000294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>acinetobactin</td>
      <td>Acinetobacter baumannii</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r155c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-247"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-247">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r155c1-pfam-details-svg" data-filename="c00155_NODE_7_.._r155c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r155c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r155c1-nrps_pks_products" data-name="r155c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r155c1-nrps_pks" data-name="r155c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r155c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-241"></div> <span class="help-tooltip" id="nrps-structures-help-241">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 49162 - 99464:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r155c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r155c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r155c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-242"></div> <span class="help-tooltip" id="nrps-monomers-help-242">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds54403_55912</span></strong>: X
      <div class="collapser collapser-target-cds54403_55912 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (13..404)</strong>: X
          <div class="collapser collapser-target-cds54403_55912 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds54403_55912-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">sahFyGcPLK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds56770_58072</span></strong>: X
      <div class="collapser collapser-target-cds56770_58072 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (90..335)</strong>: X
          <div class="collapser collapser-target-cds56770_58072 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds56770_58072-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">------IGpK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>4ppro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds69162_79464</span></strong>: X - X
      <div class="collapser collapser-target-cds69162_79464 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (512..914)</strong>: X
          <div class="collapser collapser-target-cds69162_79464 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds69162_79464-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DsfDvGMVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2040..2434)</strong>: X
          <div class="collapser collapser-target-cds69162_79464 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophilic
              <div class="collapser collapser-target-cds69162_79464-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVwnVGAInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r157c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00157_NODE_6_.. - Region 1 - NRPS,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-460"></div> <span class="help-tooltip" id="region_svg-help-460">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00157_NODE_6_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r157c1-svg" data-filename="c00157_NODE_6_.._r157c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 46,435 - 90,829 nt. (total: 44,395 nt)
        
        <a class="cluster-rules-header" id="r157c1-rules-header" href="#r157c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r157c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r157c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-461"></div> <span class="help-tooltip" id="focus-panel-help-461">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r157c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r157c1-nrps_pks" data-name="r157c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r157c1-clusterblast" data-name="r157c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r157c1-knownclusterblast" data-name="r157c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r157c1-subclusterblast" data-name="r157c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r157c1-MIBiG-cluster-compare" data-name="r157c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r157c1-pfam-details" data-name="r157c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r157c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-248"></div> <span class="help-tooltip" id="nrps-domain-help-248">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r157c1-details-svg" data-filename="c00157_NODE_6_.._r157c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r157c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r157c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-251"></div> <span class="help-tooltip" id="cb-general-help-251">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r157c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r157c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r157c1_1.svg">
        NZ_RHGC01000002 (632535-724404): Streptomyces alboflavus strain 313 contig002... (22% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r157c1_2.svg">
        NZ_CP048641 (425666-521124): Streptomyces aureoverticillatus strain HN6 chrom... (20% of genes show similarity), NRPS,lassopeptide,other,terpene
      </option>
      
      <option value = "svg/clusterblast_r157c1_3.svg">
        NZ_WPNZ01000006 (285583-377458): Streptomyces sp. p1417 Scaffold6, whole geno... (24% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r157c1_4.svg">
        NC_003155 (1877247-1931603): Streptomyces avermitilis MA-4680 = NBRC 14893, c... (31% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r157c1_5.svg">
        NZ_CP021748 (9001097-9093213): Streptomyces alboflavus strain MDJK44 chromoso... (13% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r157c1_6.svg">
        NZ_CP022685 (547821-626611): Streptomyces formicae strain KY5 chromosome, com... (14% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r157c1_7.svg">
        NZ_CP031742 (475311-573215): Streptomyces koyangensis strain VK-A60T chromoso... (17% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r157c1_8.svg">
        NZ_CP023699 (9241324-9322307): Streptomyces kanamyceticus strain ATCC 12853 c... (16% of genes show similarity), NRPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r157c1_9.svg">
        NZ_FNON01000007 (464089-508445): Amycolatopsis xylanica strain CPCC 202699, w... (23% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r157c1_10.svg">
        NZ_QQVZ01000001 (143716-299926): Streptomyces sp. S816 DV517 scf1, whole geno... (12% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r157c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r157c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r157c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-252"></div> <span class="help-tooltip" id="cb-known-help-252">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r157c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r157c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_1.svg">
        oxalomycin B
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_2.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_3.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_4.svg">
        yatakemycin
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_5.svg">
        dynemicin A
      </option>
      
    </select>
    <button id="knownclusterblast-r157c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r157c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r157c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-253"></div> <span class="help-tooltip" id="cb-sub-help-253">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r157c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-254"></div> <span class="help-tooltip" id="MIBiG-help-254">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r157c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r157c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r157c1"></div>
    <div id="comparison-MIBiG-r157c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.71</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.64</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685.1: 127-25775">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>Terpene</td>
        
        <td>brassicicene C</td>
        
        
        <td>Alternaria brassicicola ATCC 96836</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.52</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001085.1: 3683-14329">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001085/index.html#r1c1" target="_new">BGC0001085.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.47</td>
        <td>NRP, Terpene</td>
        
        <td>teleocidin B1</td>
        
        
        <td>Streptomyces blastmyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000444.1: 12535-33158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000444/index.html#r1c1" target="_new">BGC0000444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.45</td>
        <td>NRP</td>
        
        <td>thaxtomin A</td>
        
        
        <td>Streptomyces turgidiscabies Car8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.43</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.43</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001782.1: 1050-39207">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001782/index.html#r1c1" target="_new">BGC0001782.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.55, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.55, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.43</td>
        <td>Other</td>
        
        <td>spiroindimicin A, spiroindimicin B, spiroindimicin C, spiroindimicin D, indimicin A, indimicin B, indimicin C, indimicin D, indimicin E, lynamicin A, lynamicin D, lynamicin F, lynamicin G</td>
        
        
        <td>Streptomyces sp. SCSIO 03032</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r157c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685.1: 127-25775">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>brassicicene C</td>
      <td>Alternaria brassicicola ATCC 96836</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene, Polyketide</td>
      <td>aurachin RE</td>
      <td>Rhodococcus erythropolis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001782.1: 1050-39207">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001782/index.html#r1c1" target="_new">BGC0001782.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.55, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>spiroindimicin A, spiroindimicin B, spiroindimicin C, spiroindimicin D, indimicin A, indimicin B, indimicin C, indimicin D, indimicin E, lynamicin A, lynamicin D, lynamicin F, lynamicin G</td>
      <td>Streptomyces sp. SCSIO 03032</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000653.1: 0-21693">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000653/index.html#r1c1" target="_new">BGC0000653.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001085.1: 3683-14329">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001085/index.html#r1c1" target="_new">BGC0001085.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP, Terpene</td>
      <td>teleocidin B1</td>
      <td>Streptomyces blastmyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.55, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>A-94964</td>
      <td>Streptomyces sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r157c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-255"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-255">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r157c1-pfam-details-svg" data-filename="c00157_NODE_6_.._r157c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r157c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r157c1-nrps_pks_products" data-name="r157c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r157c1-nrps_pks" data-name="r157c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r157c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-249"></div> <span class="help-tooltip" id="nrps-structures-help-249">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 46434 - 90829:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r157c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r157c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r157c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-250"></div> <span class="help-tooltip" id="nrps-monomers-help-250">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds66434_70829</span></strong>: X
      <div class="collapser collapser-target-cds66434_70829 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (27..419)</strong>: X
          <div class="collapser collapser-target-cds66434_70829 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds66434_70829-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DApvVAAVwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r162c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00162_NODE_35.. - Region 1 - redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-462"></div> <span class="help-tooltip" id="region_svg-help-462">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00162_NODE_35...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r162c1-svg" data-filename="c00162_NODE_35.._r162c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 13,362 nt. (total: 13,362 nt)
        
        <a class="cluster-rules-header" id="r162c1-rules-header" href="#r162c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r162c1-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))</div>
      
      <div class="region-svg-container">
       <div id='r162c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-463"></div> <span class="help-tooltip" id="focus-panel-help-463">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r162c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r162c1-clusterblast" data-name="r162c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r162c1-knownclusterblast" data-name="r162c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r162c1-subclusterblast" data-name="r162c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r162c1-MIBiG-cluster-compare" data-name="r162c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r162c1-pfam-details" data-name="r162c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r162c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-256"></div> <span class="help-tooltip" id="cb-general-help-256">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r162c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r162c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r162c1_1.svg">
        NC_017765 (725898-783980): Streptomyces hygroscopicus subsp. jinggangensis 50... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r162c1_2.svg">
        NZ_CP013220 (955574-1016688): Streptomyces hygroscopicus subsp. limoneus stra... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r162c1_3.svg">
        NC_021985 (1457128-1499119): Streptomyces collinus Tu 365, complete sequence (14% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r162c1_4.svg">
        NZ_SOAR01000003 (544142-625786): Streptomyces sp. BK141 Ga0307717 103, whole ... (7% of genes show similarity), NRPS-like,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r162c1_5.svg">
        NZ_BBUY01000011 (27669-95595): Streptomyces sp. NBRC 110611, whole genome sho... (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r162c1_6.svg">
        NZ_CP007699 (155034-187474): Streptomyces lydicus strain A02 chromosome, comp... (12% of genes show similarity), thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r162c1_7.svg">
        NZ_CP028369 (6400892-6516036): Streptomyces sp. P3 chromosome, complete genome (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r162c1_8.svg">
        NZ_CP011125 (1625339-1665211): Sandaracinus amylolyticus strain DSM 53668 chr... (10% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r162c1_9.svg">
        NC_019678 (8012143-8050659): Rivularia sp. PCC 7116, complete genome (16% of genes show similarity), LAP,bacteriocin,cyanobactin,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r162c1_10.svg">
        NZ_LT985361 (5008717-5063929): Nocardia suismassiliense strain S-137, whole g... (11% of genes show similarity), CDPS,NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r162c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r162c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r162c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-257"></div> <span class="help-tooltip" id="cb-known-help-257">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r162c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-258"></div> <span class="help-tooltip" id="cb-sub-help-258">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r162c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-260"></div> <span class="help-tooltip" id="MIBiG-help-260">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r162c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r162c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r162c1"></div>
    <div id="comparison-MIBiG-r162c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">redox-cofactor</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000277.1: 237-11479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000277/index.html#r1c1" target="_new">BGC0000277.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>urdamycin</td>
        
        
        <td>Streptomyces fradiae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>cepacin A</td>
        
        
        <td>Burkholderia ambifaria IOP40-10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>chartreusin</td>
        
        
        <td>Streptomyces chartreusis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>nybomycin</td>
        
        
        <td>Streptomyces albus subsp. chlorinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000466.1: 0-35805">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000466/index.html#r1c1" target="_new">BGC0000466.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>NRP</td>
        
        <td>yatakemycin</td>
        
        
        <td>Streptomyces sp. TP-A2060</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r162c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000277.1: 237-11479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000277/index.html#r1c1" target="_new">BGC0000277.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>urdamycin</td>
      <td>Streptomyces fradiae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>cepacin A</td>
      <td>Burkholderia ambifaria IOP40-10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>chartreusin</td>
      <td>Streptomyces chartreusis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>nybomycin</td>
      <td>Streptomyces albus subsp. chlorinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000466.1: 0-35805">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000466/index.html#r1c1" target="_new">BGC0000466.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>NRP</td>
      <td>yatakemycin</td>
      <td>Streptomyces sp. TP-A2060</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r162c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-261"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-261">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r162c1-pfam-details-svg" data-filename="c00162_NODE_35.._r162c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r162c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r162c1-RREfinder" data-name="r162c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r162c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-259"></div> <span class="help-tooltip" id="rre-side-help-259">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>redox-cofactor protocluster</strong> (0...13362)</span>
      <dt></dt>
      
      
      
        
        <strong>cds1818_2175</strong> <div class="collapser collapser-target-cds1818_2175 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: PqqD_RRE</dd>
          <dd>Location: 32 - 117</dd>
          <dd>Score: 56.0</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r162c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00162_NODE_35.. - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-464"></div> <span class="help-tooltip" id="region_svg-help-464">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00162_NODE_35...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r162c2-svg" data-filename="c00162_NODE_35.._r162c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 53,927 - 76,032 nt. (total: 22,106 nt)
        
        <a class="cluster-rules-header" id="r162c2-rules-header" href="#r162c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r162c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r162c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-465"></div> <span class="help-tooltip" id="focus-panel-help-465">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r162c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r162c2-clusterblast" data-name="r162c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r162c2-knownclusterblast" data-name="r162c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r162c2-subclusterblast" data-name="r162c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r162c2-MIBiG-cluster-compare" data-name="r162c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r162c2-pfam-details" data-name="r162c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r162c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-262"></div> <span class="help-tooltip" id="cb-general-help-262">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r162c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r162c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r162c2_1.svg">
        NZ_CP023690 (7723521-7743821): Streptomyces spectabilis strain ATCC 27465 chr... (64% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_2.svg">
        NZ_CP040916 (7409433-7429772): Streptomyces spectabilis strain NRRL 2792 chro... (64% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_3.svg">
        NZ_CP030771 (7681187-7701996): Streptomyces sp. YIM 121038 chromosome, comple... (56% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_4.svg">
        NZ_WPNZ01000011 (108147-130322): Streptomyces sp. p1417 Scaffold11, whole gen... (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_5.svg">
        NZ_RHGC01000006 (284646-306863): Streptomyces alboflavus strain 313 contig006... (46% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_6.svg">
        NZ_CP021748 (7858722-7879276): Streptomyces alboflavus strain MDJK44 chromoso... (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_7.svg">
        NZ_CP048641 (1499240-1520328): Streptomyces aureoverticillatus strain HN6 chr... (33% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_8.svg">
        NZ_CP031455 (1608103-1831027): Streptomyces olivoreticuli subsp. olivoreticul... (4% of genes show similarity), NRPS,NRPS-like,T1PKS,terpene,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r162c2_9.svg">
        NZ_JACHJG010000011 (35758-121129): Streptomyces netropsis strain CECT 3265 Ga... (3% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r162c2_10.svg">
        NZ_CP045178 (8316059-8338228): Streptomyces tsukubensis strain AT3 chromosome... (10% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r162c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r162c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r162c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-263"></div> <span class="help-tooltip" id="cb-known-help-263">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r162c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r162c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r162c2_1.svg">
        geosmin
      </option>
      
    </select>
    <button id="knownclusterblast-r162c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r162c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r162c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-264"></div> <span class="help-tooltip" id="cb-sub-help-264">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r162c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-265"></div> <span class="help-tooltip" id="MIBiG-help-265">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r162c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r162c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r162c2"></div>
    <div id="comparison-MIBiG-r162c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
        
        
        <td>Streptomyces sp. F001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000022.1: 111992-153232">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000022/index.html#r1c1" target="_new">BGC0000022.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
         
        
        <td>0.08</td>
        <td>Polyketide</td>
        
        <td>asperfuranone</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000134.1: 1391-30251">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000134/index.html#r1c1" target="_new">BGC0000134.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
         
        
        <td>0.07</td>
        <td>Polyketide</td>
        
        <td>radicicol</td>
        
        
        <td>Pochonia chlamydosporia</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353.1: 60-28528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>NRP</td>
        
        <td>tacrolimus</td>
        
        
        <td>Streptomyces sp. MA6548</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000103.1: 30053-137271">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000103/index.html#r1c1" target="_new">BGC0000103.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>Polyketide</td>
        
        <td>mycolactone B</td>
        
        
        <td>Mycobacterium ulcerans Agy99</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001030.1: 0-22759">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001030/index.html#r1c1" target="_new">BGC0001030.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.06</td>
        <td>NRP, Polyketide</td>
        
        <td>ochratoxin A</td>
        
        
        <td>Penicillium nordicum</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r162c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
      <td>Streptomyces sp. F001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000022.1: 111992-153232">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000022/index.html#r1c1" target="_new">BGC0000022.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
      <td>0.30</td>
      <td>Polyketide</td>
      <td>asperfuranone</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000134.1: 1391-30251">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000134/index.html#r1c1" target="_new">BGC0000134.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
      <td>0.29</td>
      <td>Polyketide</td>
      <td>radicicol</td>
      <td>Pochonia chlamydosporia</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353.1: 60-28528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>NRP</td>
      <td>tacrolimus</td>
      <td>Streptomyces sp. MA6548</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000103.1: 30053-137271">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000103/index.html#r1c1" target="_new">BGC0000103.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>Polyketide</td>
      <td>mycolactone B</td>
      <td>Mycobacterium ulcerans Agy99</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.44, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001030.1: 0-22759">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001030/index.html#r1c1" target="_new">BGC0001030.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>NRP, Polyketide</td>
      <td>ochratoxin A</td>
      <td>Penicillium nordicum</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r162c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-266"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-266">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r162c2-pfam-details-svg" data-filename="c00162_NODE_35.._r162c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r162c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r167c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00167_NODE_50.. - Region 1 - NRPS,NRPS-like,other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-466"></div> <span class="help-tooltip" id="region_svg-help-466">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00167_NODE_50...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r167c1-svg" data-filename="c00167_NODE_50.._r167c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 53,641 nt. (total: 53,641 nt)
        
        <a class="cluster-rules-header" id="r167c1-rules-header" href="#r167c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r167c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r167c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-467"></div> <span class="help-tooltip" id="focus-panel-help-467">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r167c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r167c1-nrps_pks" data-name="r167c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r167c1-clusterblast" data-name="r167c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r167c1-knownclusterblast" data-name="r167c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r167c1-subclusterblast" data-name="r167c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r167c1-MIBiG-cluster-compare" data-name="r167c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r167c1-pfam-details" data-name="r167c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r167c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-267"></div> <span class="help-tooltip" id="nrps-domain-help-267">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r167c1-details-svg" data-filename="c00167_NODE_50.._r167c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r167c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r167c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-270"></div> <span class="help-tooltip" id="cb-general-help-270">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r167c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r167c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r167c1_1.svg">
        NZ_CP030771 (8466519-8568365): Streptomyces sp. YIM 121038 chromosome, comple... (30% of genes show similarity), NRPS,bacteriocin,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_2.svg">
        NZ_CP054932 (3704966-3817330): Actinomadura sp. NAK00032 chromosome, complete... (28% of genes show similarity), NRPS,TfuA-related,bacteriocin,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_3.svg">
        NZ_GG657754 (8993871-9064633): Streptomyces himastatinicus ATCC 53653 superco... (36% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_4.svg">
        NZ_CP015726 (1909867-1981053): Streptomyces sp. RTd22 chromosome, complete ge... (34% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_5.svg">
        NZ_CP029195 (1401410-1576477): Streptomyces venezuelae strain ATCC 10595 chro... (12% of genes show similarity), NRPS,NRPS-like,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r167c1_6.svg">
        NZ_CP029617 (6991641-7097699): Streptomyces sp. WAC 01529 chromosome, complet... (20% of genes show similarity), NRPS,arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r167c1_7.svg">
        NZ_KK037166 (10019475-10184618): Kutzneria sp. 744 supercont1.1, whole genome... (13% of genes show similarity), NRPS,T1PKS,bacteriocin,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_8.svg">
        NZ_CP015867 (893-391629): Streptomyces parvulus strain 2297 plasmid pSPA1, co... (10% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,betalactone,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_9.svg">
        NZ_LIYH01000003 (603276-802753): Streptomyces sp. CB03234 scaffold3, whole ge... (13% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane,terpene,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r167c1_10.svg">
        NZ_LN850107 (1679270-1852720): Alloactinosynnema sp. L-07 isolate Alloactinos... (13% of genes show similarity), NRPS,other
      </option>
      
    </select>
    <button id= "clusterblast-r167c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r167c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r167c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-271"></div> <span class="help-tooltip" id="cb-known-help-271">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r167c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r167c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_1.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_2.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_3.svg">
        formicamycins A-M
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_4.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_5.svg">
        polyoxypeptin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_6.svg">
        rubrolone A / rubrolone B
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_7.svg">
        JBIR-126
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_8.svg">
        thienodolin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_9.svg">
        JBIR-34 / JBIR-35
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_10.svg">
        WS9326
      </option>
      
    </select>
    <button id="knownclusterblast-r167c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r167c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r167c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-272"></div> <span class="help-tooltip" id="cb-sub-help-272">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r167c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-273"></div> <span class="help-tooltip" id="MIBiG-help-273">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r167c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r167c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r167c1"></div>
    <div id="comparison-MIBiG-r167c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">other</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>1.20</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>1.17</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.99</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.93</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.93</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.89</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.88</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336.1: 2595-15215">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.88</td>
        <td>Alkaloid</td>
        
        <td>erdasporine A, erdasporine B, erdasporine C</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717.1: 2898-22560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.88</td>
        <td>Alkaloid</td>
        
        <td>okaramine B</td>
        
        
        <td>Penicillium simplicissimum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.87</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r167c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518.1: 3489599-3513098">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>astellolide A</td>
      <td>Aspergillus oryzae RIB40</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717.1: 2898-22560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Alkaloid</td>
      <td>okaramine B</td>
      <td>Penicillium simplicissimum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718.1: 582791-598007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>okaramine D</td>
      <td>Aspergillus aculeatus ATCC 16872</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000432.1: 1563-32878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000432/index.html#r1c1" target="_new">BGC0000432.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>streptothricin</td>
      <td>Streptomyces rochei</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r167c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-274"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-274">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r167c1-pfam-details-svg" data-filename="c00167_NODE_50.._r167c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r167c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r167c1-nrps_pks_products" data-name="r167c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r167c1-nrps_pks" data-name="r167c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r167c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-268"></div> <span class="help-tooltip" id="nrps-structures-help-268">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 53641:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r167c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-ala - ser - X) + (thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-ala,ser,x&amp;nrps3=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-ala*,ser*,x&amp;nrps3=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r167c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 0 - 39484:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r167c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-ala - ser - X) + (thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-ala,ser,x&amp;nrps3=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-ala*,ser*,x&amp;nrps3=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r167c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 1 - 23617:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r167c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-ala - ser - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-ala,ser,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-ala*,ser*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r167c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 10743 - 53641:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r167c1-5" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(thr) + (D-ala - ser - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr&amp;nrps2=d-ala,ser,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*&amp;nrps2=d-ala*,ser*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r167c1-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)NC(C)C(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r167c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-269"></div> <span class="help-tooltip" id="nrps-monomers-help-269">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1_3766</span></strong>: X
      <div class="collapser collapser-target-cds1_3766 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (42..482)</strong>: X
          <div class="collapser collapser-target-cds1_3766 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds1_3766-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">vtyWvAcSGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>hiv-d</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds3827_19484</span></strong>: ala - ser - X
      <div class="collapser collapser-target-cds3827_19484 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala,ser,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*,ser*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (464..855)</strong>: ala
          <div class="collapser collapser-target-cds3827_19484 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds3827_19484-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVFSVagyaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>d-lyserg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2012..2426)</strong>: ser
          <div class="collapser collapser-target-cds3827_19484 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds3827_19484-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHVSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4406..4793)</strong>: X
          <div class="collapser collapser-target-cds3827_19484 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn, lys, arg
              <div class="collapser collapser-target-cds3827_19484-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVcmaGiaAK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>N-(1,1-dimethyl-1-allyl)Trp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds30743_33641</span></strong>: thr
      <div class="collapser collapser-target-cds30743_33641 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (291..717)</strong>: thr
          <div class="collapser collapser-target-cds30743_33641 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds30743_33641-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds42530_44507</span></strong>: X
      <div class="collapser collapser-target-cds42530_44507 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (69..504)</strong>: X
          <div class="collapser collapser-target-cds42530_44507 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds42530_44507-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">hvlsaGGCAK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r170c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00170_NODE_18.. - Region 1 - T1PKS,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-468"></div> <span class="help-tooltip" id="region_svg-help-468">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00170_NODE_18...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r170c1-svg" data-filename="c00170_NODE_18.._r170c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 27,073 - 101,633 nt. (total: 74,561 nt)
        
        <a class="cluster-rules-header" id="r170c1-rules-header" href="#r170c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r170c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r170c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-469"></div> <span class="help-tooltip" id="focus-panel-help-469">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r170c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r170c1-nrps_pks" data-name="r170c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r170c1-clusterblast" data-name="r170c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r170c1-knownclusterblast" data-name="r170c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r170c1-subclusterblast" data-name="r170c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r170c1-MIBiG-cluster-compare" data-name="r170c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r170c1-pfam-details" data-name="r170c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r170c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-275"></div> <span class="help-tooltip" id="nrps-domain-help-275">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r170c1-details-svg" data-filename="c00170_NODE_18.._r170c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r170c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r170c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-278"></div> <span class="help-tooltip" id="cb-general-help-278">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r170c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r170c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r170c1_1.svg">
        NZ_FNHV01000005 (136738-183556): Streptomyces sp. cf386, whole genome shotgun... (26% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_2.svg">
        NZ_AUKV01000003 (310647-367094): Streptomyces sp. 142MFCol3.1 H305DRAFT scaff... (17% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_3.svg">
        NZ_CP027306 (8423091-8627676): Streptomyces atratus strain SCSIO ZH16 chromos... (14% of genes show similarity), NRPS,T1PKS,hglE-KS,terpene
      </option>
      
      <option value = "svg/clusterblast_r170c1_4.svg">
        NZ_CP023445 (3028199-3186238): Actinosynnema pretiosum strain X47 chromosome,... (15% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_5.svg">
        NC_013093 (3002519-3157721): Actinosynnema mirum DSM 43827, complete sequence (15% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_6.svg">
        NZ_CP029607 (3093395-3248136): Actinosynnema pretiosum subsp. pretiosum strai... (14% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_7.svg">
        NC_021191 (3208254-3340523): Actinoplanes sp. N902-109, complete genome (13% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_8.svg">
        NZ_MLCE01000003 (184185-240483): Streptomyces sp. Tue6028 Scaffold2, whole ge... (16% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r170c1_9.svg">
        NZ_CP017717 (10087461-10190937): Nonomuraea sp. ATCC 55076 chromosome, comple... (12% of genes show similarity), NRPS,T1PKS,arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r170c1_10.svg">
        NZ_FNHV01000019 (50536-174737): Streptomyces sp. cf386, whole genome shotgun ... (8% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r170c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r170c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r170c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-279"></div> <span class="help-tooltip" id="cb-known-help-279">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r170c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r170c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_1.svg">
        ikarugamycin
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_2.svg">
        A54145
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_3.svg">
        conglobatin
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_4.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_5.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_6.svg">
        ishigamide
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_7.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_8.svg">
        desotamide
      </option>
      
      <option value = "svg/knownclusterblast_r170c1_9.svg">
        actagardine
      </option>
      
    </select>
    <button id="knownclusterblast-r170c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r170c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r170c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-280"></div> <span class="help-tooltip" id="cb-sub-help-280">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r170c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-281"></div> <span class="help-tooltip" id="MIBiG-help-281">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r170c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r170c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r170c1"></div>
    <div id="comparison-MIBiG-r170c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000012.1: 1892-19092">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000012/index.html#r1c1" target="_new">BGC0000012.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
         
        
        <td>0.34</td>
        <td>Polyketide</td>
        
        <td>alternapyrone</td>
        
        
        <td>Alternaria solani</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000528.1: 2216379-2224562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000528/index.html#r1c1" target="_new">BGC0000528.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:0.45, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>michiganin A</td>
        
        
        <td>Clavibacter michiganensis subsp. michiganensis NCPPB 382</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749.1: 3634168-3687903">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide</td>
        
        <td>epipyriculol</td>
        
        
        <td>Pyricularia oryzae 70-15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000088.1: 1812-55428">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000088/index.html#r1c1" target="_new">BGC0000088.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>monacolin K</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146.1: 754-20637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>solanapyrone D</td>
        
        
        <td>Alternaria solani</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436.1: 25-56534">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>cornexistin</td>
        
        
        <td>Paecilomyces divaricatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001557.1: 25-56534">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001557/index.html#r1c1" target="_new">BGC0001557.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>cornexistin</td>
        
        
        <td>Paecilomyces divaricatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001067.1: 79284-114438">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001067/index.html#r1c1" target="_new">BGC0001067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide, Terpene</td>
        
        <td>fumagillin, -trans-bergamotene, fumagillol</td>
        
        
        <td>Aspergillus fumigatus Af293</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r170c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000088.1: 1812-55428">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000088/index.html#r1c1" target="_new">BGC0000088.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>monacolin K</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001617.1: 911831-958327">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001617/index.html#r1c1" target="_new">BGC0001617.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>hirsutene</td>
      <td>Stereum hirsutum FP-91666 SS1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749.1: 3634168-3687903">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>epipyriculol</td>
      <td>Pyricularia oryzae 70-15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436.1: 25-56534">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>cornexistin</td>
      <td>Paecilomyces divaricatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001557.1: 25-56534">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001557/index.html#r1c1" target="_new">BGC0001557.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>cornexistin</td>
      <td>Paecilomyces divaricatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>botrydial</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146.1: 754-20637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>solanapyrone D</td>
      <td>Alternaria solani</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r170c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-282"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-282">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r170c1-pfam-details-svg" data-filename="c00170_NODE_18.._r170c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r170c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r170c1-nrps_pks_products" data-name="r170c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r170c1-nrps_pks" data-name="r170c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r170c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-276"></div> <span class="help-tooltip" id="nrps-structures-help-276">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 27072 - 101633:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r170c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pk) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk&amp;nrps2=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk*&amp;nrps2=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r170c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 27072 - 88910:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r170c1-2" class="smiles-canvas" data-smiles="C([*])C(-O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pk) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk&amp;nrps2=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk*&amp;nrps2=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r170c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r170c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-277"></div> <span class="help-tooltip" id="nrps-monomers-help-277">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds47072_54638</span></strong>: pk
      <div class="collapser collapser-target-cds47072_54638 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (543..837)</strong>: pk
          <div class="collapser collapser-target-cds47072_54638 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds47072_54638-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>CE-Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds47072_54638-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 109.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 100.6</dt>

  <dd></dd><dt>Malonyl-CoA: 87.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 76.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 63.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 50.4</dt>

  <dd></dd><dt>Propionyl-CoA: 45.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 44.1</dt>

  <dd></dd><dt>CHC-CoA: 26.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 23.2</dt>

  <dd></dd><dt>fatty_acid: 23.1</dt>

  <dd></dd><dt>Acetyl-CoA: 17.3</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 17.2</dt>

  <dd></dd><dt>inactive: 11.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds56037_57780</span></strong>: pk
      <div class="collapser collapser-target-cds56037_57780 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (38..476)</strong>: pk
          <div class="collapser collapser-target-cds56037_57780 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  NH2
              <div class="collapser collapser-target-cds56037_57780-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>NH2: 97.2</dt>

  <dd></dd><dt>fatty_acid: 39.0</dt>

  <dd></dd><dt>Acetyl-CoA: 17.6</dt>

  <dd></dd><dt>AHBA: 2.0</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds60966_68910</span></strong>: pk
      <div class="collapser collapser-target-cds60966_68910 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (615..909)</strong>: pk
          <div class="collapser collapser-target-cds60966_68910 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds60966_68910-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>CE-Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds60966_68910-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 102.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 81.6</dt>

  <dd></dd><dt>Malonyl-CoA: 72.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 55.5</dt>

  <dd></dd><dt>Propionyl-CoA: 55.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 52.8</dt>

  <dd></dd><dt>CHC-CoA: 36.9</dt>

  <dd></dd><dt>Acetyl-CoA: 33.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 31.9</dt>

  <dd></dd><dt>fatty_acid: 31.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 30.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 26.1</dt>

  <dd></dd><dt>inactive: 19.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 17.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r179c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00179_NODE_91.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-470"></div> <span class="help-tooltip" id="region_svg-help-470">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00179_NODE_91...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r179c1-svg" data-filename="c00179_NODE_91.._r179c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 38,047 nt. (total: 38,047 nt)
        
        <a class="cluster-rules-header" id="r179c1-rules-header" href="#r179c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r179c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r179c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-471"></div> <span class="help-tooltip" id="focus-panel-help-471">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r179c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r179c1-nrps_pks" data-name="r179c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r179c1-clusterblast" data-name="r179c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r179c1-knownclusterblast" data-name="r179c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r179c1-subclusterblast" data-name="r179c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r179c1-MIBiG-cluster-compare" data-name="r179c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r179c1-pfam-details" data-name="r179c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r179c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-283"></div> <span class="help-tooltip" id="nrps-domain-help-283">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r179c1-details-svg" data-filename="c00179_NODE_91.._r179c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r179c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r179c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-286"></div> <span class="help-tooltip" id="cb-general-help-286">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r179c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r179c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r179c1_1.svg">
        NC_017803 (995236-1073702): Actinoplanes sp. SE50/110, complete genome (30% of genes show similarity), NRPS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r179c1_2.svg">
        NZ_JACHXF010000001 (1139940-1220306): Actinoplanes campanulatus strain CECT 3... (28% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r179c1_3.svg">
        NZ_CM000913 (5516299-5592878): Streptomyces clavuligerus strain ATCC 27064 ch... (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r179c1_4.svg">
        NZ_CP016559 (1302309-1378394): Streptomyces clavuligerus strain F613-1 chromo... (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r179c1_5.svg">
        NZ_JADG01000003 (144085-225361): Actinomadura oligospora ATCC 43269 P696DRAFT... (18% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r179c1_6.svg">
        NZ_SLXR01000002 (591847-681601): Streptomyces sp. BK438 Ga0307687 102, whole ... (14% of genes show similarity), NRPS,arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r179c1_7.svg">
        NZ_KB905359 (1943282-2018825): Salinispora pacifica DSM 45544 Salpac3DRAFT Sc... (18% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r179c1_8.svg">
        NZ_AZVW01000003 (26067-103490): Salinispora pacifica CNT603 B175DRAFT scaffol... (19% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r179c1_9.svg">
        NZ_KQ948937 (33031-153700): Streptomyces caeruleatus strain NRRL B-24802 PRJN... (16% of genes show similarity), NRPS,PKS-like,T1PKS,T3PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r179c1_10.svg">
        NZ_LT607411 (3085042-3148440): Micromonospora viridifaciens strain DSM 43909 ... (15% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r179c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r179c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r179c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-287"></div> <span class="help-tooltip" id="cb-known-help-287">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r179c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r179c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_1.svg">
        ecumicin
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_2.svg">
        teleocidin B1
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_3.svg">
        sporolide A / sporolide B
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_4.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_5.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_6.svg">
        cysteoamide
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_7.svg">
        teicoplanin
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_8.svg">
        avoparcin
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_9.svg">
        ristomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r179c1_10.svg">
        UK-68,597
      </option>
      
    </select>
    <button id="knownclusterblast-r179c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r179c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r179c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-288"></div> <span class="help-tooltip" id="cb-sub-help-288">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r179c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-289"></div> <span class="help-tooltip" id="MIBiG-help-289">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r179c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r179c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r179c1"></div>
    <div id="comparison-MIBiG-r179c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.45</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.44</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r179c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r179c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-290"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-290">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r179c1-pfam-details-svg" data-filename="c00179_NODE_91.._r179c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r179c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r179c1-nrps_pks_products" data-name="r179c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r179c1-nrps_pks" data-name="r179c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r179c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-284"></div> <span class="help-tooltip" id="nrps-structures-help-284">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 38047:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r179c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(=O)N)C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - asn) + (thr - thr - thr - asn - thr - asn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,asn&amp;nrps2=thr,thr,thr,asn,thr,asn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,asn*&amp;nrps2=thr*,thr*,thr*,asn*,thr*,asn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r179c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(=O)N)C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r179c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-285"></div> <span class="help-tooltip" id="nrps-monomers-help-285">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2998_8233</span></strong>: X - asn
      <div class="collapser collapser-target-cds2998_8233 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,asn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,asn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (24..420)</strong>: X
          <div class="collapser collapser-target-cds2998_8233 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp, asn, glu, gln, aad
              <div class="collapser collapser-target-cds2998_8233-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWqaGtVnK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1073..1488)</strong>: asn
          <div class="collapser collapser-target-cds2998_8233 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds2998_8233-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLpKVaEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds8229_28614</span></strong>: thr - thr - thr - asn - thr - asn
      <div class="collapser collapser-target-cds8229_28614 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,thr,thr,asn,thr,asn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,thr*,thr*,asn*,thr*,asn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (512..924)</strong>: thr
          <div class="collapser collapser-target-cds8229_28614 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds8229_28614-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVfsVAMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1581..1993)</strong>: thr
          <div class="collapser collapser-target-cds8229_28614 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds8229_28614-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVfSIAmVhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2648..3060)</strong>: thr
          <div class="collapser collapser-target-cds8229_28614 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds8229_28614-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVfSIAmVhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3717..4114)</strong>: asn
          <div class="collapser collapser-target-cds8229_28614 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds8229_28614-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLpKVaEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4773..5185)</strong>: thr
          <div class="collapser collapser-target-cds8229_28614 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds8229_28614-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVfSIAmVhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (5850..6247)</strong>: asn
          <div class="collapser collapser-target-cds8229_28614 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds8229_28614-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLpKVaEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds35482_38047</span></strong>: val
      <div class="collapser collapser-target-cds35482_38047 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (475..855)</strong>: val
          <div class="collapser collapser-target-cds35482_38047 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds35482_38047-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYflGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r197c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00197_NODE_10.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-472"></div> <span class="help-tooltip" id="region_svg-help-472">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00197_NODE_10...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r197c1-svg" data-filename="c00197_NODE_10.._r197c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 5,076 - 26,368 nt. (total: 21,293 nt)
        
        <a class="cluster-rules-header" id="r197c1-rules-header" href="#r197c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r197c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r197c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-473"></div> <span class="help-tooltip" id="focus-panel-help-473">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r197c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r197c1-clusterblast" data-name="r197c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r197c1-knownclusterblast" data-name="r197c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r197c1-subclusterblast" data-name="r197c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r197c1-MIBiG-cluster-compare" data-name="r197c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r197c1-pfam-details" data-name="r197c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r197c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-291"></div> <span class="help-tooltip" id="cb-general-help-291">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r197c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r197c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r197c1_1.svg">
        NZ_CP032698 (1264119-1371620): Streptomyces hundungensis strain BH38 chromoso... (5% of genes show similarity), NRPS,T1PKS,betalactone,other,terpene
      </option>
      
      <option value = "svg/clusterblast_r197c1_2.svg">
        NZ_CP023692 (542908-620186): Streptomyces vinaceus strain ATCC 27476 chromoso... (9% of genes show similarity), CDPS,T1PKS,hglE-KS,terpene
      </option>
      
      <option value = "svg/clusterblast_r197c1_3.svg">
        NZ_LWLD01000001 (413875-533225): Streptomyces sp. CB03578 scaffold1, whole ge... (6% of genes show similarity), NRPS,T1PKS,terpene,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r197c1_4.svg">
        NZ_CP033581 (772300-892262): Streptomyces sp. ADI95-16 chromosome, complete g... (6% of genes show similarity), NRPS,T1PKS,terpene,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r197c1_5.svg">
        NZ_CP030287 (6827684-&gt;6841819): Streptomyces sp. ICC1 chromosome (23% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r197c1_6.svg">
        NZ_CP011664 (933028-1051615): Streptomyces sp. Mg1 chromosome 1, complete seq... (6% of genes show similarity), NRPS,T1PKS,terpene,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r197c1_7.svg">
        NZ_ML123037 (927891-1046695): Streptomyces sp. ADI91-18 scaffold002, whole ge... (6% of genes show similarity), NRPS,T1PKS,terpene,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r197c1_8.svg">
        NZ_CP054936 (1740918-1761642): Streptomyces seoulensis strain A01 chromosome (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r197c1_9.svg">
        NZ_FRBK01000014 (114773-229129): Streptomyces yunnanensis strain CGMCC 4.3555... (10% of genes show similarity), NRPS-like,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r197c1_10.svg">
        NZ_CP003987 (9774953-9886021): Streptomyces sp. 769 chromosome, complete genome (11% of genes show similarity), NRPS-like,T1PKS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r197c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r197c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r197c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-292"></div> <span class="help-tooltip" id="cb-known-help-292">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r197c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r197c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r197c1_1.svg">
        ebelactone
      </option>
      
      <option value = "svg/knownclusterblast_r197c1_2.svg">
        2-methylisoborneol
      </option>
      
      <option value = "svg/knownclusterblast_r197c1_3.svg">
        2-methylisoborneol
      </option>
      
    </select>
    <button id="knownclusterblast-r197c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r197c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r197c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-293"></div> <span class="help-tooltip" id="cb-sub-help-293">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r197c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-294"></div> <span class="help-tooltip" id="MIBiG-help-294">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r197c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r197c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r197c1"></div>
    <div id="comparison-MIBiG-r197c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.60, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>0.60</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
         
        
        <td>0.51</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.54, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.52, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Pseudanabaena sp. dqh15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000883.1: 0-30199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000883/index.html#r1c1" target="_new">BGC0000883.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>arginomycin</td>
        
        
        <td>Streptomyces arginensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>a201a</td>
        
        
        <td>Marinactinospora thermotolerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000138.1: 1-18550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000138/index.html#r1c1" target="_new">BGC0000138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>Polyketide</td>
        
        <td>rimocidin</td>
        
        
        <td>Streptomyces diastaticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r197c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.60, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.54, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.52, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Pseudanabaena sp. dqh15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000883.1: 0-30199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000883/index.html#r1c1" target="_new">BGC0000883.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>arginomycin</td>
      <td>Streptomyces arginensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>a201a</td>
      <td>Marinactinospora thermotolerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000138.1: 1-18550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000138/index.html#r1c1" target="_new">BGC0000138.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>Polyketide</td>
      <td>rimocidin</td>
      <td>Streptomyces diastaticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r197c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-295"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-295">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r197c1-pfam-details-svg" data-filename="c00197_NODE_10.._r197c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r197c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r201c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00201_NODE_17.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-474"></div> <span class="help-tooltip" id="region_svg-help-474">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00201_NODE_17...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r201c1-svg" data-filename="c00201_NODE_17.._r201c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 15,040 nt. (total: 15,040 nt)
        
        <a class="cluster-rules-header" id="r201c1-rules-header" href="#r201c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r201c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r201c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-475"></div> <span class="help-tooltip" id="focus-panel-help-475">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r201c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r201c1-nrps_pks" data-name="r201c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r201c1-clusterblast" data-name="r201c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r201c1-knownclusterblast" data-name="r201c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r201c1-subclusterblast" data-name="r201c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r201c1-MIBiG-cluster-compare" data-name="r201c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r201c1-pfam-details" data-name="r201c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r201c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-296"></div> <span class="help-tooltip" id="nrps-domain-help-296">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r201c1-details-svg" data-filename="c00201_NODE_17.._r201c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r201c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r201c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-299"></div> <span class="help-tooltip" id="cb-general-help-299">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r201c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r201c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r201c1_1.svg">
        NZ_CP020700 (260319-402649): Streptomyces tsukubensis strain NRRL 18488 chrom... (6% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_2.svg">
        NZ_CP011492 (6798486-6913245): Streptomyces sp. CNQ-509 chromosome, complete ... (2% of genes show similarity), NRPS,PKS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_3.svg">
        NZ_AZWL01000009 (128293-200845): Streptomyces sp. CNH099 B121DRAFT scaffold 7... (5% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_4.svg">
        NZ_CP025407 (5242-77025): Streptomyces sp. CMB-StM0423 chromosome, complete g... (5% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_5.svg">
        NC_020504 (699925-769986): Streptomyces davaonensis JCM 4913, complete genome (4% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_6.svg">
        NZ_WWJS01000022 (11-88943): Streptomyces sp. SID4912 SID4912.c22, whole genom... (3% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_7.svg">
        NZ_LACQ01000016 (75728-164650): Streptomyces sp. OspMP-M45 YUMDRAFT scaffold ... (3% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_8.svg">
        NZ_CP029342 (729660-818772): Streptomyces sp. SM18 chromosome, complete genome (3% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_9.svg">
        NZ_FMYR01000011 (180612-269509): Streptomyces sp. AmelKG-A3, whole genome sho... (3% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r201c1_10.svg">
        NZ_CP015588 (53613-153054): Streptomyces alfalfae strain ACCC40021 chromosome... (3% of genes show similarity), NRPS,PKS-like,T1PKS,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r201c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r201c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r201c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-300"></div> <span class="help-tooltip" id="cb-known-help-300">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r201c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-301"></div> <span class="help-tooltip" id="cb-sub-help-301">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r201c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-302"></div> <span class="help-tooltip" id="MIBiG-help-302">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r201c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r201c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r201c1"></div>
    <div id="comparison-MIBiG-r201c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r201c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r201c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-303"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-303">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r201c1-pfam-details-svg" data-filename="c00201_NODE_17.._r201c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r201c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r201c1-nrps_pks_products" data-name="r201c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r201c1-nrps_pks" data-name="r201c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r201c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-297"></div> <span class="help-tooltip" id="nrps-structures-help-297">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 15040:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r201c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r201c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r201c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-298"></div> <span class="help-tooltip" id="nrps-monomers-help-298">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1_9679</span></strong>: X - X
      <div class="collapser collapser-target-cds1_9679 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (859..1252)</strong>: X
          <div class="collapser collapser-target-cds1_9679 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp
              <div class="collapser collapser-target-cds1_9679-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWmtgAVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2410..2806)</strong>: X
          <div class="collapser collapser-target-cds1_9679 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala
              <div class="collapser collapser-target-cds1_9679-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfsLGCfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r203c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00203_NODE_1_.. - Region 1 - T1PKS,lassopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-476"></div> <span class="help-tooltip" id="region_svg-help-476">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00203_NODE_1_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r203c1-svg" data-filename="c00203_NODE_1_.._r203c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 14,536 - 62,173 nt. (total: 47,638 nt)
        
        <a class="cluster-rules-header" id="r203c1-rules-header" href="#r203c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r203c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>lassopeptide: (PF13471 and Asn_synthase or micJ25 or mcjC)</div>
      
      <div class="region-svg-container">
       <div id='r203c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-477"></div> <span class="help-tooltip" id="focus-panel-help-477">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r203c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r203c1-nrps_pks" data-name="r203c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r203c1-clusterblast" data-name="r203c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r203c1-knownclusterblast" data-name="r203c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r203c1-subclusterblast" data-name="r203c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r203c1-lassopeptides" data-name="r203c1-lassopeptides">
        Lasso peptides
      </div>
     
    
     
    
     
      <div class="body-details-header r203c1-MIBiG-cluster-compare" data-name="r203c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r203c1-pfam-details" data-name="r203c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r203c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-304"></div> <span class="help-tooltip" id="nrps-domain-help-304">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r203c1-details-svg" data-filename="c00203_NODE_1_.._r203c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r203c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r203c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-307"></div> <span class="help-tooltip" id="cb-general-help-307">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r203c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r203c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r203c1_1.svg">
        NZ_JH725387 (1002631-1096628): Streptomyces auratus AGR0001 Scaffold1, whole ... (9% of genes show similarity), butyrolactone,ladderane,lassopeptide,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r203c1_2.svg">
        NZ_LT907981 (1892492-1933922): Streptomyces sp. 2323.1 chromosome I (16% of genes show similarity), ladderane,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_3.svg">
        NZ_CP034539 (8171274-8241558): Streptomyces cyaneochromogenes strain MK-45 ch... (10% of genes show similarity), NRPS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_4.svg">
        NZ_CP054929 (7903970-7926592): Streptomyces sp. NA00687 chromosome, complete ... (31% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_5.svg">
        NC_003155 (6869385-6891957): Streptomyces avermitilis MA-4680 = NBRC 14893, c... (33% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_6.svg">
        NZ_CP029541 (369758-426253): Streptomyces sp. NEAU-S7GS2 chromosome, complete... (12% of genes show similarity), T1PKS,hglE-KS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_7.svg">
        NZ_CP017157 (5149247-5198382): Streptomyces lydicus strain 103 chromosome, co... (12% of genes show similarity), NRPS,lassopeptide,nucleoside
      </option>
      
      <option value = "svg/clusterblast_r203c1_8.svg">
        NZ_LIYH01000003 (225563-248144): Streptomyces sp. CB03234 scaffold3, whole ge... (22% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_9.svg">
        NZ_NBCN01000010 (310506-333070): Streptomyces sp. CB03238 scaffold10, whole g... (21% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r203c1_10.svg">
        NZ_BBNO01000008 (543836-566449): Streptomyces lydicamycinicus strain TP-A0598... (25% of genes show similarity), lassopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r203c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r203c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r203c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-308"></div> <span class="help-tooltip" id="cb-known-help-308">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r203c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r203c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_1.svg">
        citrulassin D
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_2.svg">
        citrulassin B
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_3.svg">
        citrulassin F
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_4.svg">
        citrulassin A
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_5.svg">
        citrulassin E
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_6.svg">
        4-hydroxy-3-nitrosobenzamide
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_7.svg">
        alnumycin A / alnumycin B / alnumycin C / alnumycin P / prealnumycin / thalnumycin A / thalnumycin B / K1115A / 1,6-dihydro-8-propylanthraquinone
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_8.svg">
        murayaquinone
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_9.svg">
        SF2575
      </option>
      
      <option value = "svg/knownclusterblast_r203c1_10.svg">
        endophenazine A / endophenazine B
      </option>
      
    </select>
    <button id="knownclusterblast-r203c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r203c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r203c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-309"></div> <span class="help-tooltip" id="cb-sub-help-309">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r203c1-lassopeptides">
      <div class="details">
    <div class="heading">
      <span>Lasso peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-body-help-310"></div> <span class="help-tooltip" id="lasso-body-help-310">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lassopeptide(s) for cds45888_45999 - Putative Class II</h4>
    <hr>
      <dl class="details-text">
        
          <dt><span class="serif">cds45888_45999</span> leader / core peptide / cleaved tail</dt>
          <dd>
            MKKAYEAPTLARLGAFRQKTG
            -
            <span class ="macrolactam">L</span><span class ="macrolactam">L</span><span class ="macrolactam">Q</span><span class ="macrolactam">R</span><span class ="macrolactam">S</span><span class ="macrolactam">G</span><span class ="macrolactam">N</span><span class ="macrolactam">D</span>RLIFSKN
          </dd>
        
      </dl>
    
  <div class="legend">
    <h4> Legend: </h4>
    <div>
      <span class="macrolactam">Red</span>: Putative macrolactam<br>
      <span class="cut">Purple</span>: Putative cleaved off residues<br>
    </div>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r203c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-313"></div> <span class="help-tooltip" id="MIBiG-help-313">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r203c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r203c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r203c1"></div>
    <div id="comparison-MIBiG-r203c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">lassopeptide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001550.1: 2949-7545">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001550/index.html#r1c1" target="_new">BGC0001550.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
         
        
        <td>1.24</td>
        <td>RiPP</td>
        
        <td>citrulassin D</td>
        
        
        <td>Streptomyces katrae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001552.1: 6877532-6881873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001552/index.html#r1c1" target="_new">BGC0001552.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.60, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.60, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>1.20</td>
        <td>RiPP</td>
        
        <td>citrulassin F</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001549.1: 98590-103146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001549/index.html#r1c1" target="_new">BGC0001549.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>1.18</td>
        <td>RiPP</td>
        
        <td>citrulassin B</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>1.15</td>
        <td>RiPP</td>
        
        <td>citrulassin E</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.59, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.59, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.94</td>
        <td>RiPP</td>
        
        <td>citrulassin A</td>
        
        
        <td>Streptomyces albulus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001781.1: 158349-161251">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001781/index.html#r1c1" target="_new">BGC0001781.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.79</td>
        <td>RiPP</td>
        
        <td>sphaericin</td>
        
        
        <td>Planomonospora sphaerica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.72</td>
        <td>RiPP</td>
        
        <td>ulleungdin</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001493.1: 72757-75568">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001493/index.html#r1c1" target="_new">BGC0001493.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.70</td>
        <td>RiPP</td>
        
        <td>achromosin</td>
        
        
        <td>Streptomyces achromogenes subsp. achromogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001176.1: 9270-14062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001176/index.html#r1c1" target="_new">BGC0001176.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.68</td>
        <td>RiPP</td>
        
        <td>streptomonomicin</td>
        
        
        <td>Streptomonospora alba</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001889.1: 1333039-1337282">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001889/index.html#r1c1" target="_new">BGC0001889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.64</td>
        <td>RiPP</td>
        
        <td>fusilassin</td>
        
        
        <td>Thermobifida fusca YX</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r203c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001550.1: 2949-7545">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001550/index.html#r1c1" target="_new">BGC0001550.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
      <td>0.85</td>
      <td>RiPP</td>
      <td>citrulassin D</td>
      <td>Streptomyces katrae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001552.1: 6877532-6881873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001552/index.html#r1c1" target="_new">BGC0001552.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.60, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>RiPP</td>
      <td>citrulassin F</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001549.1: 98590-103146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001549/index.html#r1c1" target="_new">BGC0001549.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>RiPP</td>
      <td>citrulassin B</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>RiPP</td>
      <td>citrulassin E</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.59, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>RiPP</td>
      <td>citrulassin A</td>
      <td>Streptomyces albulus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001781.1: 158349-161251">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001781/index.html#r1c1" target="_new">BGC0001781.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>RiPP</td>
      <td>sphaericin</td>
      <td>Planomonospora sphaerica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>RiPP</td>
      <td>ulleungdin</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001493.1: 72757-75568">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001493/index.html#r1c1" target="_new">BGC0001493.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>RiPP</td>
      <td>achromosin</td>
      <td>Streptomyces achromogenes subsp. achromogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001176.1: 9270-14062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001176/index.html#r1c1" target="_new">BGC0001176.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>RiPP</td>
      <td>streptomonomicin</td>
      <td>Streptomonospora alba</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001921.1: 10367-22235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001921/index.html#r1c1" target="_new">BGC0001921.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>MS-271</td>
      <td>Streptomyces sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r203c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-314"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-314">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r203c1-pfam-details-svg" data-filename="c00203_NODE_1_.._r203c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r203c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r203c1-nrps_pks_products" data-name="r203c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r203c1-nrps_pks" data-name="r203c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r203c1-lassopeptides" data-name="r203c1-lassopeptides">
        Lasso peptides
      </div>
     
    
     
      <div class="sidepanel-details-header r203c1-RREfinder" data-name="r203c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r203c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-305"></div> <span class="help-tooltip" id="nrps-structures-help-305">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 14535 - 62173:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r203c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r203c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r203c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-306"></div> <span class="help-tooltip" id="nrps-monomers-help-306">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds34535_42173</span></strong>: pk
      <div class="collapser collapser-target-cds34535_42173 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (559..857)</strong>: pk
          <div class="collapser collapser-target-cds34535_42173 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds34535_42173-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>CE-Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds34535_42173-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 111.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 99.8</dt>

  <dd></dd><dt>Malonyl-CoA: 83.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 69.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 67.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 50.9</dt>

  <dd></dd><dt>Propionyl-CoA: 48.3</dt>

  <dd></dd><dt>CHC-CoA: 45.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 42.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 31.5</dt>

  <dd></dd><dt>Acetyl-CoA: 29.7</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 23.5</dt>

  <dd></dd><dt>fatty_acid: 22.1</dt>

  <dd></dd><dt>inactive: 20.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r203c1-lassopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-side-help-311"></div> <span class="help-tooltip" id="lasso-side-help-311">Lists the possible core peptides in the region. Each core peptide lists the number of disulfide bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
    </div>

  <dl class ="prediction-text">
    <dt><span class="serif">cds45888_45999</span> - Class II</dt>
      <dd>Cleavage pHMM score: 0.20</dd>
      <dd>RODEO score: 21</dd>
      <dd>Monoisotopic mass: 1741.9 Da</dd>
      <dd>Molecular weight: 1743.0 Da</dd>
      <dd>Number of bridges: 0</dd>
    
  </dl>

</div>

    </div>
     
    
     
    <div class="sidepanel-details-section r203c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-312"></div> <span class="help-tooltip" id="rre-side-help-312">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lassopeptide protocluster</strong> (36025...58584)</span>
      <dt></dt>
      
      
      
        
        <strong>cds47898_48153</strong> <div class="collapser collapser-target-cds47898_48153 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Stand_Alone_Lasso_RRE</dd>
          <dd>Location: 1 - 83</dd>
          <dd>Score: 84.0</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r203c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00203_NODE_1_.. - Region 2 - phenazine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-478"></div> <span class="help-tooltip" id="region_svg-help-478">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00203_NODE_1_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r203c2-svg" data-filename="c00203_NODE_1_.._r203c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 138,806 - 159,324 nt. (total: 20,519 nt)
        
        <a class="cluster-rules-header" id="r203c2-rules-header" href="#r203c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r203c2-rules">phenazine: phzB</div>
      
      <div class="region-svg-container">
       <div id='r203c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-479"></div> <span class="help-tooltip" id="focus-panel-help-479">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r203c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r203c2-clusterblast" data-name="r203c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r203c2-knownclusterblast" data-name="r203c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r203c2-subclusterblast" data-name="r203c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r203c2-MIBiG-cluster-compare" data-name="r203c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r203c2-pfam-details" data-name="r203c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r203c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-315"></div> <span class="help-tooltip" id="cb-general-help-315">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r203c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r203c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r203c2_1.svg">
        NZ_CP034353 (935367-955793): Streptomyces sp. KPB2 chromosome, complete genome (18% of genes show similarity), phenazine
      </option>
      
      <option value = "svg/clusterblast_r203c2_2.svg">
        NZ_CP054920 (881109-901535): Streptomyces sp. NA03103 chromosome, complete ge... (20% of genes show similarity), phenazine
      </option>
      
      <option value = "svg/clusterblast_r203c2_3.svg">
        NZ_SOPG01000017 (48392-68818): Streptomyces sp. H23 NODE 17 length 183633 cov... (18% of genes show similarity), phenazine
      </option>
      
      <option value = "svg/clusterblast_r203c2_4.svg">
        NZ_FKJI03000002 (236861-257287): Streptomyces sp. F-1 isolate Streptomyces sp... (13% of genes show similarity), phenazine
      </option>
      
      <option value = "svg/clusterblast_r203c2_5.svg">
        NZ_MTKC01000001 (1612561-1657735): Streptomyces griseofuscus strain NG1-7 NG1... (6% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r203c2_6.svg">
        NZ_CP058977 (2836980-2857406): Streptomyces sp. NEAU-sy36 chromosome, complet... (13% of genes show similarity), phenazine
      </option>
      
      <option value = "svg/clusterblast_r203c2_7.svg">
        NZ_CP033073 (7037600-7157320): Streptomyces sp. Z022 chromosome, complete genome (4% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r203c2_8.svg">
        NZ_CP021080 (742915-797414): Streptomyces pluripotens strain MUSC 135 chromos... (6% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r203c2_9.svg">
        NZ_CP007574 (730322-772364): Streptomyces albulus strain NK660 chromosome, co... (6% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r203c2_10.svg">
        NZ_CP026094 (756703-798927): Streptomyces albulus strain CK-15 chromosome, co... (7% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r203c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r203c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r203c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-316"></div> <span class="help-tooltip" id="cb-known-help-316">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r203c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-317"></div> <span class="help-tooltip" id="cb-sub-help-317">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r203c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-318"></div> <span class="help-tooltip" id="MIBiG-help-318">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r203c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r203c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r203c2"></div>
    <div id="comparison-MIBiG-r203c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">phenazine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001241.1: 3-35442">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001241/index.html#r1c1" target="_new">BGC0001241.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>Terpene</td>
        
        <td>ergotamine</td>
        
        
        <td>Claviceps fusiformis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267.1: 3-35442">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>Terpene</td>
        
        <td>lysergic acid, elymoclavine</td>
        
        
        <td>Claviceps fusiformis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000080.1: 1-67322">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000080/index.html#r1c1" target="_new">BGC0000080.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>Polyketide</td>
        
        <td>jerangolid A, jerangolid D</td>
        
        
        <td>Sorangium cellulosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000811.1: 2907011-2929163">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000811/index.html#r1c1" target="_new">BGC0000811.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.18 (id:0.46, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
         
        
        <td>0.04</td>
        <td>Alkaloid</td>
        
        <td>fumigaclavine C</td>
        
        
        <td>Aspergillus fumigatus Af293</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r203c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001241.1: 3-35442">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001241/index.html#r1c1" target="_new">BGC0001241.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>Terpene</td>
      <td>ergotamine</td>
      <td>Claviceps fusiformis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267.1: 3-35442">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>Terpene</td>
      <td>lysergic acid, elymoclavine</td>
      <td>Claviceps fusiformis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000080.1: 1-67322">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000080/index.html#r1c1" target="_new">BGC0000080.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>Polyketide</td>
      <td>jerangolid A, jerangolid D</td>
      <td>Sorangium cellulosum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000811.1: 2907011-2929163">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000811/index.html#r1c1" target="_new">BGC0000811.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.18 (id:0.46, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
      <td>0.18</td>
      <td>Alkaloid</td>
      <td>fumigaclavine C</td>
      <td>Aspergillus fumigatus Af293</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r203c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-319"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-319">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r203c2-pfam-details-svg" data-filename="c00203_NODE_1_.._r203c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r203c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r206c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00206_NODE_11.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-480"></div> <span class="help-tooltip" id="region_svg-help-480">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00206_NODE_11...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r206c1-svg" data-filename="c00206_NODE_11.._r206c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 30,227 nt. (total: 30,227 nt)
        
        <a class="cluster-rules-header" id="r206c1-rules-header" href="#r206c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r206c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r206c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-481"></div> <span class="help-tooltip" id="focus-panel-help-481">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r206c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r206c1-nrps_pks" data-name="r206c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r206c1-clusterblast" data-name="r206c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r206c1-knownclusterblast" data-name="r206c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r206c1-subclusterblast" data-name="r206c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r206c1-MIBiG-cluster-compare" data-name="r206c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r206c1-pfam-details" data-name="r206c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r206c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-320"></div> <span class="help-tooltip" id="nrps-domain-help-320">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r206c1-details-svg" data-filename="c00206_NODE_11.._r206c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r206c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r206c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-323"></div> <span class="help-tooltip" id="cb-general-help-323">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r206c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r206c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r206c1_1.svg">
        NZ_CP015867 (893-391629): Streptomyces parvulus strain 2297 plasmid pSPA1, co... (7% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,betalactone,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r206c1_2.svg">
        NZ_CP043638 (1873822-1977840): Streptomyces sp. INR7 chromosome, complete genome (14% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r206c1_3.svg">
        NZ_FNVU01000001 (224642-328704): Streptomyces yanglinensis strain CGMCC 4.202... (10% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r206c1_4.svg">
        NZ_CP017248 (2722891-2862053): Streptomyces fodineus strain TW1S1 chromosome,... (8% of genes show similarity), NRPS,T1PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r206c1_5.svg">
        NZ_LIVQ01000010 (7862-141901): Streptomyces sp. CB00455 scaffold10, whole gen... (3% of genes show similarity), NRPS,butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r206c1_6.svg">
        NZ_CP015588 (53613-153054): Streptomyces alfalfae strain ACCC40021 chromosome... (6% of genes show similarity), NRPS,PKS-like,T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r206c1_7.svg">
        NZ_CP015588 (8474190-8572774): Streptomyces alfalfae strain ACCC40021 chromos... (6% of genes show similarity), NRPS,PKS-like,T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r206c1_8.svg">
        NZ_CP026305 (3123-72040): Streptomyces lunaelactis strain MM109 plasmid pSLUN... (15% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r206c1_9.svg">
        NZ_CP020555 (2036513-2140511): Streptomyces sp. Sge12 chromosome, complete ge... (12% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r206c1_10.svg">
        NZ_CP020039 (6137705-6243379): Streptomyces sp. 3211 isolate 3 chromosome, co... (16% of genes show similarity), NRPS,T1PKS,other
      </option>
      
    </select>
    <button id= "clusterblast-r206c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r206c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r206c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-324"></div> <span class="help-tooltip" id="cb-known-help-324">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r206c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r206c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r206c1_1.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r206c1_2.svg">
        polyoxypeptin
      </option>
      
      <option value = "svg/knownclusterblast_r206c1_3.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r206c1_4.svg">
        kutzneride 2
      </option>
      
      <option value = "svg/knownclusterblast_r206c1_5.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
    </select>
    <button id="knownclusterblast-r206c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r206c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r206c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-325"></div> <span class="help-tooltip" id="cb-sub-help-325">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r206c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-326"></div> <span class="help-tooltip" id="MIBiG-help-326">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r206c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r206c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r206c1"></div>
    <div id="comparison-MIBiG-r206c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.63, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r206c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.63, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r206c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-327"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-327">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r206c1-pfam-details-svg" data-filename="c00206_NODE_11.._r206c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r206c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r206c1-nrps_pks_products" data-name="r206c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r206c1-nrps_pks" data-name="r206c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r206c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-321"></div> <span class="help-tooltip" id="nrps-structures-help-321">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 30227:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r206c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC(Cc1ccccc1)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(leu) + (D-phe - X) + (D-X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu&amp;nrps2=d-phe,x&amp;nrps3=d-x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*&amp;nrps2=d-phe*,x&amp;nrps3=d-x*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r206c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC(Cc1ccccc1)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r206c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-322"></div> <span class="help-tooltip" id="nrps-monomers-help-322">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1_9667</span></strong>: X - X
      <div class="collapser collapser-target-cds1_9667 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (857..1251)</strong>: X
          <div class="collapser collapser-target-cds1_9667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds1_9667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWmtgAVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2406..2801)</strong>: X
          <div class="collapser collapser-target-cds1_9667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala
              <div class="collapser collapser-target-cds1_9667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSLAcfYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds12610_18739</span></strong>: phe - X
      <div class="collapser collapser-target-cds12610_18739 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (23..408)</strong>: phe
          <div class="collapser collapser-target-cds12610_18739 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe
              <div class="collapser collapser-target-cds12610_18739-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>phe</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWTVAAVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1556..1854)</strong>: X
          <div class="collapser collapser-target-cds12610_18739 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds12610_18739-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>tyr, bht</dd>
   <dt>Single AA prediction:</dt>
   <dd>tyr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">gSva--E--K</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>his</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds24309_27006</span></strong>: leu
      <div class="collapser collapser-target-cds24309_27006 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (307..719)</strong>: leu
          <div class="collapser collapser-target-cds24309_27006 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds24309_27006-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFvLGCaFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r208c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00208_NODE_26.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-482"></div> <span class="help-tooltip" id="region_svg-help-482">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00208_NODE_26...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r208c1-svg" data-filename="c00208_NODE_26.._r208c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 33,174 - 93,594 nt. (total: 60,421 nt)
        
        <a class="cluster-rules-header" id="r208c1-rules-header" href="#r208c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r208c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r208c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-483"></div> <span class="help-tooltip" id="focus-panel-help-483">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r208c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r208c1-nrps_pks" data-name="r208c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r208c1-clusterblast" data-name="r208c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r208c1-knownclusterblast" data-name="r208c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r208c1-subclusterblast" data-name="r208c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r208c1-MIBiG-cluster-compare" data-name="r208c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r208c1-pfam-details" data-name="r208c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r208c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-328"></div> <span class="help-tooltip" id="nrps-domain-help-328">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r208c1-details-svg" data-filename="c00208_NODE_26.._r208c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r208c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r208c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-331"></div> <span class="help-tooltip" id="cb-general-help-331">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r208c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r208c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r208c1_1.svg">
        NZ_WPNZ01000005 (118249-196956): Streptomyces sp. p1417 Scaffold5, whole geno... (24% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r208c1_2.svg">
        NZ_CP029617 (7567083-7657636): Streptomyces sp. WAC 01529 chromosome, complet... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r208c1_3.svg">
        NZ_CP010849 (7255201-7387896): Streptomyces cyaneogriseus subsp. noncyanogenu... (8% of genes show similarity), LAP,NRPS,PKS-like
      </option>
      
      <option value = "svg/clusterblast_r208c1_4.svg">
        NZ_WWHG01000001 (446133-524258): Streptomyces sp. SID7810 SID7810.c1, whole g... (22% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r208c1_5.svg">
        NZ_CP058977 (4848528-4926800): Streptomyces sp. NEAU-sy36 chromosome, complet... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r208c1_6.svg">
        NZ_CP015588 (8231593-8310034): Streptomyces alfalfae strain ACCC40021 chromos... (20% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r208c1_7.svg">
        NZ_CP017316 (5064336-5169973): Streptomyces rubrolavendulae strain MJM4426 ch... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r208c1_8.svg">
        NZ_CP023696 (5270077-5330299): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r208c1_9.svg">
        NZ_CP040916 (9196245-9274865): Streptomyces spectabilis strain NRRL 2792 chro... (14% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r208c1_10.svg">
        NZ_CP023690 (9615788-9694321): Streptomyces spectabilis strain ATCC 27465 chr... (14% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r208c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r208c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r208c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-332"></div> <span class="help-tooltip" id="cb-known-help-332">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r208c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-333"></div> <span class="help-tooltip" id="cb-sub-help-333">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r208c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-334"></div> <span class="help-tooltip" id="MIBiG-help-334">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r208c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r208c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r208c1"></div>
    <div id="comparison-MIBiG-r208c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404.1: 1968-56662">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>penicillin</td>
        
        
        <td>Penicillium chrysogenum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r208c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000404.1: 1968-56662">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000404/index.html#r1c1" target="_new">BGC0000404.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>penicillin</td>
      <td>Penicillium chrysogenum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r208c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-335"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-335">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r208c1-pfam-details-svg" data-filename="c00208_NODE_26.._r208c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r208c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r208c1-nrps_pks_products" data-name="r208c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r208c1-nrps_pks" data-name="r208c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r208c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-329"></div> <span class="help-tooltip" id="nrps-structures-help-329">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 33173 - 93594:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r208c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(C(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)NC(CC(=O)O)C(=O)NC(C(O)C)C(=O)NC(CC(=O)O)C(=O)NC(C(C)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - D-ala - D-val - D-ala - X) + (D-asp - thr - D-asp - val - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,d-ala,d-val,d-ala,x&amp;nrps2=d-asp,thr,d-asp,val,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,d-ala*,d-val*,d-ala*,x&amp;nrps2=d-asp*,thr*,d-asp*,val*,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r208c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C)C(=O)NC(C(C)C)C(=O)NC(C)C(=O)NC([*])C(=O)NC(CC(=O)O)C(=O)NC(C(O)C)C(=O)NC(CC(=O)O)C(=O)NC(C(C)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r208c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-330"></div> <span class="help-tooltip" id="nrps-monomers-help-330">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds53173_72064</span></strong>: X - ala - val - ala - X
      <div class="collapser collapser-target-cds53173_72064 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,ala,val,ala,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,ala*,val*,ala*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (38..420)</strong>: X
          <div class="collapser collapser-target-cds53173_72064 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds53173_72064-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALliGsVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1084..1510)</strong>: ala
          <div class="collapser collapser-target-cds53173_72064 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds53173_72064-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvFTVAiVyK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2658..3055)</strong>: val
          <div class="collapser collapser-target-cds53173_72064 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds53173_72064-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYWWGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4200..4600)</strong>: ala
          <div class="collapser collapser-target-cds53173_72064 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds53173_72064-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSvAiVYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (5716..6100)</strong>: X
          <div class="collapser collapser-target-cds53173_72064 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds53173_72064-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALlTGsVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds72077_92399</span></strong>: asp - thr - asp - val - X
      <div class="collapser collapser-target-cds72077_92399 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp,thr,asp,val,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp*,thr*,asp*,val*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (522..907)</strong>: asp
          <div class="collapser collapser-target-cds72077_92399 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp
              <div class="collapser collapser-target-cds72077_92399-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DteDmGfVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2120..2532)</strong>: thr
          <div class="collapser collapser-target-cds72077_92399 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds72077_92399-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3189..3582)</strong>: asp
          <div class="collapser collapser-target-cds72077_92399 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp
              <div class="collapser collapser-target-cds72077_92399-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtWnLGVVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4696..5076)</strong>: val
          <div class="collapser collapser-target-cds72077_92399 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds72077_92399-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAfWWGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (5732..6113)</strong>: X
          <div class="collapser collapser-target-cds72077_92399 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds72077_92399-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALliGsVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r215c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00215_NODE_16.. - Region 1 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-484"></div> <span class="help-tooltip" id="region_svg-help-484">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00215_NODE_16...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r215c1-svg" data-filename="c00215_NODE_16.._r215c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 17,837 nt. (total: 17,837 nt)
        
        <a class="cluster-rules-header" id="r215c1-rules-header" href="#r215c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r215c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r215c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-485"></div> <span class="help-tooltip" id="focus-panel-help-485">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r215c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r215c1-nrps_pks" data-name="r215c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r215c1-clusterblast" data-name="r215c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r215c1-knownclusterblast" data-name="r215c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r215c1-subclusterblast" data-name="r215c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r215c1-MIBiG-cluster-compare" data-name="r215c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r215c1-pfam-details" data-name="r215c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r215c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-336"></div> <span class="help-tooltip" id="nrps-domain-help-336">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r215c1-details-svg" data-filename="c00215_NODE_16.._r215c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r215c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r215c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-339"></div> <span class="help-tooltip" id="cb-general-help-339">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r215c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r215c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r215c1_1.svg">
        NZ_CP034550 (7985027-8048499): Saccharothrix syringae strain NRRL B-16468 chr... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r215c1_2.svg">
        NZ_CDME01000002 (58394-246440): Kibdelosporangium sp. MJ126-NF4, whole genome... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,betalactone,lassopeptide,other
      </option>
      
      <option value = "svg/clusterblast_r215c1_3.svg">
        NZ_CP050177 (542896-630186): Streptomyces sp. QMT-12 chromosome, complete genome (12% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r215c1_4.svg">
        NZ_JADG01000003 (334520-483052): Actinomadura oligospora ATCC 43269 P696DRAFT... (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r215c1_5.svg">
        NC_016111 (2784744-2856402): Streptomyces cattleya NRRL 8057 = DSM 46488, com... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r215c1_6.svg">
        NZ_CP023691 (4869767-4939963): Streptomyces platensis strain ATCC 23948 chrom... (7% of genes show similarity), NRPS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r215c1_7.svg">
        NZ_BBNO01000002 (123246-195227): Streptomyces lydicamycinicus strain TP-A0598... (8% of genes show similarity), NRPS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r215c1_8.svg">
        NZ_LT629701 (4156233-4226865): Allokutzneria albata strain DSM 44149 chromoso... (8% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r215c1_9.svg">
        NZ_CP029042 (5134933-5217821): Streptomyces lydicus strain WYEC 108 chromosom... (7% of genes show similarity), LAP,NRPS,TfuA-related,lassopeptide,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r215c1_10.svg">
        NZ_JDUZ01000001 (568918-640244): Streptomyces sp. PCS3-D2 isolate P1 scaffold... (5% of genes show similarity), NRPS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r215c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r215c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r215c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-340"></div> <span class="help-tooltip" id="cb-known-help-340">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r215c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r215c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_1.svg">
        WAP-8294A2
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_2.svg">
        salinamide A / salinamide B / salinamide C / salinamide D / salinamide E / salinamide F / desmethylsalinamide C / desmethylsalinamide E
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_3.svg">
        salinamide A
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_4.svg">
        fuscachelin A / fuscachelin B / fuscachelin C
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_5.svg">
        griselimycin
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_6.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_7.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_8.svg">
        lysocin
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_9.svg">
        cysteoamide
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_10.svg">
        sarpeptin A / sarpeptin B
      </option>
      
    </select>
    <button id="knownclusterblast-r215c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r215c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r215c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-341"></div> <span class="help-tooltip" id="cb-sub-help-341">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r215c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-342"></div> <span class="help-tooltip" id="MIBiG-help-342">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r215c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r215c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r215c1"></div>
    <div id="comparison-MIBiG-r215c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.72</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.59</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.48, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.48, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.50</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.49</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.48</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000293.1: 2428386-2441151">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000293/index.html#r1c1" target="_new">BGC0000293.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.47</td>
        <td>NRP</td>
        
        <td>acetylaszonalenin</td>
        
        
        <td>Aspergillus fischeri NRRL 181</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r215c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.48, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000293.1: 2428386-2441151">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000293/index.html#r1c1" target="_new">BGC0000293.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>acetylaszonalenin</td>
      <td>Aspergillus fischeri NRRL 181</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r215c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-343"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-343">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r215c1-pfam-details-svg" data-filename="c00215_NODE_16.._r215c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r215c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r215c1-nrps_pks_products" data-name="r215c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r215c1-nrps_pks" data-name="r215c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r215c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-337"></div> <span class="help-tooltip" id="nrps-structures-help-337">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 17837:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r215c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-ser)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ser" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ser*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r215c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 2 - 17636:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r215c1-2" class="smiles-canvas" data-smiles="NC(CO)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-ser)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ser" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ser*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r215c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r215c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-338"></div> <span class="help-tooltip" id="nrps-monomers-help-338">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds836_6791</span></strong>: ser
      <div class="collapser collapser-target-cds836_6791 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (470..871)</strong>: ser
          <div class="collapser collapser-target-cds836_6791 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds836_6791-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHVSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r217c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00217_NODE_48.. - Region 1 - NRPS,NRPS-like,RiPP-like,betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-486"></div> <span class="help-tooltip" id="region_svg-help-486">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00217_NODE_48...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r217c1-svg" data-filename="c00217_NODE_48.._r217c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 2 - 62,519 nt. (total: 62,518 nt)
        
        <a class="cluster-rules-header" id="r217c1-rules-header" href="#r217c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r217c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)<br>betalactone: (HMGL-like and AMP-binding)<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r217c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-487"></div> <span class="help-tooltip" id="focus-panel-help-487">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r217c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r217c1-nrps_pks" data-name="r217c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r217c1-clusterblast" data-name="r217c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r217c1-knownclusterblast" data-name="r217c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r217c1-subclusterblast" data-name="r217c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r217c1-MIBiG-cluster-compare" data-name="r217c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r217c1-pfam-details" data-name="r217c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r217c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-344"></div> <span class="help-tooltip" id="nrps-domain-help-344">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r217c1-details-svg" data-filename="c00217_NODE_48.._r217c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r217c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r217c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-347"></div> <span class="help-tooltip" id="cb-general-help-347">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r217c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r217c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r217c1_1.svg">
        NZ_KB913032 (779078-848216): Amycolatopsis alba DSM 44262 scaffold1, whole ge... (42% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r217c1_2.svg">
        NZ_LWLE01000009 (18596-83276): Streptomyces sp. TSRI0281 scaffold17, whole ge... (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r217c1_3.svg">
        NC_021055 (7291525-7437851): Streptomyces sp. PAMC 26508, complete sequence (26% of genes show similarity), NRPS,T1PKS,blactam
      </option>
      
      <option value = "svg/clusterblast_r217c1_4.svg">
        NC_016114 (&lt;71079-216979): Streptomyces pratensis ATCC 33331, complete sequence (25% of genes show similarity), NRPS,T1PKS,blactam
      </option>
      
      <option value = "svg/clusterblast_r217c1_5.svg">
        NZ_CP038147 (6938322-7084613): Streptomyces sp. S501 chromosome, complete genome (24% of genes show similarity), NRPS,T1PKS,blactam
      </option>
      
      <option value = "svg/clusterblast_r217c1_6.svg">
        NZ_CP023695 (7682097-7848984): Streptomyces alboniger strain ATCC 12461 chrom... (12% of genes show similarity), LAP,NRPS,T1PKS,bacteriocin,betalactone,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r217c1_7.svg">
        NZ_CP015867 (893-391629): Streptomyces parvulus strain 2297 plasmid pSPA1, co... (15% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,betalactone,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r217c1_8.svg">
        NZ_CP030771 (8466519-8568365): Streptomyces sp. YIM 121038 chromosome, comple... (15% of genes show similarity), NRPS,bacteriocin,other
      </option>
      
      <option value = "svg/clusterblast_r217c1_9.svg">
        NZ_PVTF01000005 (360532-527607): Umezawaea tangerina strain DSM 44720 Ga01809... (12% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r217c1_10.svg">
        NZ_PVZC01000007 (8944-104208): Allonocardiopsis opalescens strain DSM 45601 G... (23% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r217c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r217c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r217c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-348"></div> <span class="help-tooltip" id="cb-known-help-348">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r217c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r217c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_1.svg">
        vazabitide A
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_2.svg">
        (2S,6R)-diamino-(5R,7)-dihydroxy-heptanoic acid
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_3.svg">
        griseoviridin / fijimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_4.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_5.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_6.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_7.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_8.svg">
        ficellomycin
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_9.svg">
        s56-p1
      </option>
      
      <option value = "svg/knownclusterblast_r217c1_10.svg">
        telomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r217c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r217c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r217c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-349"></div> <span class="help-tooltip" id="cb-sub-help-349">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r217c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-350"></div> <span class="help-tooltip" id="MIBiG-help-350">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r217c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r217c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r217c1"></div>
    <div id="comparison-MIBiG-r217c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
         <th><span class="cc-heat-product">betalactone</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>1.31</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>1.23</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>1.21</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>1.20</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>1.18</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:0.71, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:0.75, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:0.75, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:0.75, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>1.15</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001545.1: 7266118-7283606">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001545/index.html#r1c1" target="_new">BGC0001545.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>1.15</td>
        <td>NRP</td>
        
        <td>chrysogine</td>
        
        
        <td>Fusarium graminearum PH-1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.47, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>1.15</td>
        <td>NRP</td>
        
        <td>paenibactin</td>
        
        
        <td>Paenibacillus elgii B69</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>1.12</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>1.10</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r217c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454.1: 24-19884">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>vanchrobactin</td>
      <td>Vibrio anguillarum RV22</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:0.75, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r217c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-351"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-351">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r217c1-pfam-details-svg" data-filename="c00217_NODE_48.._r217c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r217c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r217c1-nrps_pks_products" data-name="r217c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r217c1-nrps_pks" data-name="r217c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r217c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-345"></div> <span class="help-tooltip" id="nrps-structures-help-345">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 1 - 62519:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r217c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X) + (X) + (X) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x&amp;nrps2=x&amp;nrps3=x&amp;nrps4=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x&amp;nrps2=x&amp;nrps3=x&amp;nrps4=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r217c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 1 - 26882:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r217c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r217c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 6969 - 62519:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r217c1-4" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X) + (X) + (X) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x&amp;nrps2=x&amp;nrps3=x&amp;nrps4=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x&amp;nrps2=x&amp;nrps3=x&amp;nrps4=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r217c1-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r217c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-346"></div> <span class="help-tooltip" id="nrps-monomers-help-346">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1008_7404</span></strong>: X
      <div class="collapser collapser-target-cds1008_7404 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (32..489)</strong>: X
          <div class="collapser collapser-target-cds1008_7404 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds1008_7404-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAyMvGlVFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds16969_18487</span></strong>: pro
      <div class="collapser collapser-target-cds16969_18487 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (15..403)</strong>: pro
          <div class="collapser collapser-target-cds16969_18487 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro
              <div class="collapser collapser-target-cds16969_18487-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>pro, pip</dd>
   <dt>Small clusters prediction:</dt>
   <dd>pro</dd>
   <dt>Single AA prediction:</dt>
   <dd>pro</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DmeFSAqVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21160_22960</span></strong>: X
      <div class="collapser collapser-target-cds21160_22960 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (6..389)</strong>: X
          <div class="collapser collapser-target-cds21160_22960 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds21160_22960-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DgTKIGaVcK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds28269_32244</span></strong>: X
      <div class="collapser collapser-target-cds28269_32244 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (468..871)</strong>: X
          <div class="collapser collapser-target-cds28269_32244 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds28269_32244-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtwDLGllDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds34542_42519</span></strong>: ser - X
      <div class="collapser collapser-target-cds34542_42519 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (460..853)</strong>: ser
          <div class="collapser collapser-target-cds34542_42519 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds34542_42519-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHVSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1503..1901)</strong>: X
          <div class="collapser collapser-target-cds34542_42519 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds34542_42519-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtvDmGiVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r222c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00222_NODE_22.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-488"></div> <span class="help-tooltip" id="region_svg-help-488">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00222_NODE_22...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r222c1-svg" data-filename="c00222_NODE_22.._r222c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,228 nt. (total: 4,228 nt)
        
        <a class="cluster-rules-header" id="r222c1-rules-header" href="#r222c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r222c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r222c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-489"></div> <span class="help-tooltip" id="focus-panel-help-489">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r222c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r222c1-clusterblast" data-name="r222c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r222c1-knownclusterblast" data-name="r222c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r222c1-subclusterblast" data-name="r222c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r222c1-MIBiG-cluster-compare" data-name="r222c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r222c1-pfam-details" data-name="r222c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r222c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-352"></div> <span class="help-tooltip" id="cb-general-help-352">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r222c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r222c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r222c1_1.svg">
        NZ_CP030771 (9617892-9629815): Streptomyces sp. YIM 121038 chromosome, comple... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_2.svg">
        NZ_CP023690 (9394817-9406740): Streptomyces spectabilis strain ATCC 27465 chr... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_3.svg">
        NZ_CP040916 (9034332-9046255): Streptomyces spectabilis strain NRRL 2792 chro... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_4.svg">
        NZ_CP045547 (7208836-7220756): Streptomyces sp. SYP-A7193 chromosome, complet... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_5.svg">
        NZ_WXVT01000001 (7133109-7145029): Streptomyces sp. KO7888 A913 AS4 SC1 R3, w... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_6.svg">
        NZ_CP029192 (8622505-8634458): Streptomyces venezuelae strain ATCC 14584 chro... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_7.svg">
        NZ_CP029191 (198832-210291): Streptomyces venezuelae strain ATCC 14585 chromo... (18% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_8.svg">
        NZ_WHZR01000001 (6892134-6904054): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (25% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_9.svg">
        NZ_CP015588 (8231593-8310034): Streptomyces alfalfae strain ACCC40021 chromos... (5% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r222c1_10.svg">
        NZ_CP012949 (41359-52884): Streptomyces ambofaciens strain DSM 40697 chromoso... (18% of genes show similarity), bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r222c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r222c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r222c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-353"></div> <span class="help-tooltip" id="cb-known-help-353">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r222c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-354"></div> <span class="help-tooltip" id="cb-sub-help-354">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r222c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-355"></div> <span class="help-tooltip" id="MIBiG-help-355">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r222c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r222c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r222c1"></div>
    <div id="comparison-MIBiG-r222c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001546.1: 0-10917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001546/index.html#r1c1" target="_new">BGC0001546.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.45, order:0.69, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Alkaloid</td>
        
        <td>chuangxinmycin</td>
        
        
        <td>Actinoplanes tsinanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485.1: 0-18179">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.45, order:0.69, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>3-demethylchuangxinmycin</td>
        
        
        <td>Actinoplanes tsinanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001753.1: 134922-155574">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001753/index.html#r1c1" target="_new">BGC0001753.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.48, order:0.69, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>radamycin, globimycin</td>
        
        
        <td>Streptomyces globisporus subsp. globisporus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000598.1: 5499-20549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000598/index.html#r1c1" target="_new">BGC0000598.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
         
        
        <td>0.06</td>
        <td>RiPP</td>
        
        <td>polytheonamide A, polytheonamide B</td>
        
        
        <td>Streptomyces diastatochromogenes T6028</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001157.1: 6304533-6321700">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001157/index.html#r1c1" target="_new">BGC0001157.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.24 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
         
        
        <td>0.06</td>
        <td>RiPP</td>
        
        <td>bottromycin A2</td>
        
        
        <td>Streptomyces scabiei 87.22</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000468.1: 1376-18956">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000468/index.html#r1c1" target="_new">BGC0000468.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.24 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
         
        
        <td>0.06</td>
        <td>RiPP</td>
        
        <td>bottromycin A2</td>
        
        
        <td>Streptomyces bottropensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000470.1: 12176-26921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000470/index.html#r1c1" target="_new">BGC0000470.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>RiPP</td>
        
        <td>bottromycin D</td>
        
        
        <td>Streptomyces sp. WMMB 272</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000469.1: 13-39901">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000469/index.html#r1c1" target="_new">BGC0000469.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.46, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>RiPP</td>
        
        <td>bottromycin A2</td>
        
        
        <td>Streptomyces sp. BC16019</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001148.1: 62-28868">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001148/index.html#r1c1" target="_new">BGC0001148.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.20 (id:0.46, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
         
        
        <td>0.04</td>
        <td>NRP, RiPP</td>
        
        <td>pheganomycin</td>
        
        
        <td>Streptomyces cirratus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r222c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001546.1: 0-10917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001546/index.html#r1c1" target="_new">BGC0001546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.45, order:0.69, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Alkaloid</td>
      <td>chuangxinmycin</td>
      <td>Actinoplanes tsinanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485.1: 0-18179">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.45, order:0.69, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>3-demethylchuangxinmycin</td>
      <td>Actinoplanes tsinanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001753.1: 134922-155574">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001753/index.html#r1c1" target="_new">BGC0001753.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.48, order:0.69, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>RiPP</td>
      <td>radamycin, globimycin</td>
      <td>Streptomyces globisporus subsp. globisporus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000598.1: 5499-20549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000598/index.html#r1c1" target="_new">BGC0000598.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.25 (id:0.46, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
      <td>0.25</td>
      <td>RiPP</td>
      <td>polytheonamide A, polytheonamide B</td>
      <td>Streptomyces diastatochromogenes T6028</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001157.1: 6304533-6321700">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001157/index.html#r1c1" target="_new">BGC0001157.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.24 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
      <td>0.24</td>
      <td>RiPP</td>
      <td>bottromycin A2</td>
      <td>Streptomyces scabiei 87.22</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000468.1: 1376-18956">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000468/index.html#r1c1" target="_new">BGC0000468.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.24 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
      <td>0.24</td>
      <td>RiPP</td>
      <td>bottromycin A2</td>
      <td>Streptomyces bottropensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000470.1: 12176-26921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000470/index.html#r1c1" target="_new">BGC0000470.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>RiPP</td>
      <td>bottromycin D</td>
      <td>Streptomyces sp. WMMB 272</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000469.1: 13-39901">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000469/index.html#r1c1" target="_new">BGC0000469.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.46, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>RiPP</td>
      <td>bottromycin A2</td>
      <td>Streptomyces sp. BC16019</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001148.1: 62-28868">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001148/index.html#r1c1" target="_new">BGC0001148.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.20 (id:0.46, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
      <td>0.20</td>
      <td>NRP, RiPP</td>
      <td>pheganomycin</td>
      <td>Streptomyces cirratus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r222c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-356"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-356">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r222c1-pfam-details-svg" data-filename="c00222_NODE_22.._r222c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r222c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r226c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00226_NODE_36.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-490"></div> <span class="help-tooltip" id="region_svg-help-490">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00226_NODE_36...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r226c1-svg" data-filename="c00226_NODE_36.._r226c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 52,084 - 73,157 nt. (total: 21,074 nt)
        
        <a class="cluster-rules-header" id="r226c1-rules-header" href="#r226c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r226c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r226c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-491"></div> <span class="help-tooltip" id="focus-panel-help-491">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r226c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r226c1-clusterblast" data-name="r226c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r226c1-knownclusterblast" data-name="r226c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r226c1-subclusterblast" data-name="r226c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r226c1-MIBiG-cluster-compare" data-name="r226c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r226c1-pfam-details" data-name="r226c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r226c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-357"></div> <span class="help-tooltip" id="cb-general-help-357">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r226c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r226c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r226c1_1.svg">
        NZ_CP026121 (5098731-5117349): Streptomyces sp. Go-475 chromosome, complete g... (22% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_2.svg">
        NZ_CP039123 (803279-837373): Streptomyces sp. SS52 chromosome, complete genome (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_3.svg">
        NZ_KZ819184 (3584225-3618334): Streptomyces sp. CS207 scaffold00001, whole ge... (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_4.svg">
        NZ_MQUH01000027 (40055-61117): Streptomyces sp. M1013 M1013 C27, whole genome... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_5.svg">
        NZ_CP012949 (764702-795248): Streptomyces ambofaciens strain DSM 40697 chromo... (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_6.svg">
        NZ_CP012382 (784392-815058): Streptomyces ambofaciens ATCC 23877 chromosome, ... (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_7.svg">
        NZ_LIPF01000007 (539754-574642): Streptomyces sp. CB02414 scaffold7, whole ge... (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_8.svg">
        NZ_LIYH01000002 (28200-49277): Streptomyces sp. CB03234 scaffold2, whole geno... (25% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_9.svg">
        NZ_CP027297 (879867-913753): Streptomyces sp. SGAir0924 chromosome, complete ... (15% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r226c1_10.svg">
        NZ_CP045740 (4547918-4620430): Streptomyces sp. SUK 48 chromosome, complete g... (9% of genes show similarity), T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r226c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r226c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r226c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-358"></div> <span class="help-tooltip" id="cb-known-help-358">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r226c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r226c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r226c1_1.svg">
        zorbamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r226c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r226c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r226c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-359"></div> <span class="help-tooltip" id="cb-sub-help-359">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r226c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-360"></div> <span class="help-tooltip" id="MIBiG-help-360">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r226c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r226c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r226c1"></div>
    <div id="comparison-MIBiG-r226c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>eremophilene</td>
        
        
        <td>Sorangium cellulosum So ce56</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.49, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Terpene, Alkaloid</td>
        
        <td>xiamycin</td>
        
        
        <td>Streptomyces sp. HKI0576</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Terpene</td>
        
        <td>xiamycin A</td>
        
        
        <td>Streptomyces sp. SCSIO 02999</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
        
        
        <td>Synechocystis salina LEGE 06099</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000561.1: 0-16402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000561/index.html#r1c1" target="_new">BGC0000561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>thermophilin 1277</td>
        
        
        <td>Streptococcus thermophilus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000498.1: 1-14995">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000498/index.html#r1c1" target="_new">BGC0000498.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>bovicin HJ50</td>
        
        
        <td>Streptococcus equinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210.1: 4366120-4375355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>pseudomycoicidin</td>
        
        
        <td>Bacillus pseudomycoides DSM 12442</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r226c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Terpene</td>
      <td>eremophilene</td>
      <td>Sorangium cellulosum So ce56</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.49, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Terpene, Alkaloid</td>
      <td>xiamycin</td>
      <td>Streptomyces sp. HKI0576</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Terpene</td>
      <td>xiamycin A</td>
      <td>Streptomyces sp. SCSIO 02999</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
      <td>Synechocystis salina LEGE 06099</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000561.1: 0-16402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000561/index.html#r1c1" target="_new">BGC0000561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>thermophilin 1277</td>
      <td>Streptococcus thermophilus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000498.1: 1-14995">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000498/index.html#r1c1" target="_new">BGC0000498.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>bovicin HJ50</td>
      <td>Streptococcus equinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210.1: 4366120-4375355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>pseudomycoicidin</td>
      <td>Bacillus pseudomycoides DSM 12442</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r226c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-361"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-361">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r226c1-pfam-details-svg" data-filename="c00226_NODE_36.._r226c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r226c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r227c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00227_NODE_92.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-492"></div> <span class="help-tooltip" id="region_svg-help-492">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00227_NODE_92...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r227c1-svg" data-filename="c00227_NODE_92.._r227c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 9,879 - 20,283 nt. (total: 10,405 nt)
        
        <a class="cluster-rules-header" id="r227c1-rules-header" href="#r227c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r227c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r227c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-493"></div> <span class="help-tooltip" id="focus-panel-help-493">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r227c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r227c1-clusterblast" data-name="r227c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r227c1-knownclusterblast" data-name="r227c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r227c1-subclusterblast" data-name="r227c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r227c1-MIBiG-cluster-compare" data-name="r227c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r227c1-pfam-details" data-name="r227c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r227c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-362"></div> <span class="help-tooltip" id="cb-general-help-362">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r227c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r227c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r227c1_1.svg">
        NZ_CP023690 (2555195-2565600): Streptomyces spectabilis strain ATCC 27465 chr... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_2.svg">
        NZ_CP040916 (2321665-2332070): Streptomyces spectabilis strain NRRL 2792 chro... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_3.svg">
        NZ_WPNZ01000010 (78975-89380): Streptomyces sp. p1417 Scaffold10, whole genom... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_4.svg">
        NZ_RHGC01000007 (604677-615082): Streptomyces alboflavus strain 313 contig007... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_5.svg">
        NZ_CP021748 (2168102-2178507): Streptomyces alboflavus strain MDJK44 chromoso... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_6.svg">
        NZ_NNBL01000002 (781489-791888): Streptomyces sp. CB01635 scaffold2, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_7.svg">
        NZ_CP047140 (6109568-6119967): Streptomyces sp. Tu 2975 chromosome, complete ... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_8.svg">
        NZ_KN549153 (70322-80721): Streptomyces sp. M10 Scaffold8, whole genome shotg... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_9.svg">
        NZ_KZ195583 (5602913-5613312): Streptomyces sp. CS227 scaffold00001, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r227c1_10.svg">
        NC_020990 (5635345-5645744): Streptomyces albidoflavus, complete sequence (100% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r227c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r227c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r227c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-363"></div> <span class="help-tooltip" id="cb-known-help-363">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r227c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r227c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r227c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r227c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r227c1_3.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r227c1_4.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r227c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r227c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r227c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-364"></div> <span class="help-tooltip" id="cb-sub-help-364">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r227c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-365"></div> <span class="help-tooltip" id="MIBiG-help-365">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r227c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r227c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r227c1"></div>
    <div id="comparison-MIBiG-r227c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.82 (id:0.63, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
         
        
        <td>0.56</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.82 (id:0.62, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
         
        
        <td>0.55</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.49, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.49, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r227c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.82 (id:0.63, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
      <td>0.82</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.82 (id:0.62, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
      <td>0.82</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.49, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.49, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r227c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-366"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-366">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r227c1-pfam-details-svg" data-filename="c00227_NODE_92.._r227c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r227c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r228c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00228_NODE_19.. - Region 1 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-494"></div> <span class="help-tooltip" id="region_svg-help-494">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00228_NODE_19...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r228c1-svg" data-filename="c00228_NODE_19.._r228c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,567 nt. (total: 8,567 nt)
        
        <a class="cluster-rules-header" id="r228c1-rules-header" href="#r228c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r228c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r228c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-495"></div> <span class="help-tooltip" id="focus-panel-help-495">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r228c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r228c1-nrps_pks" data-name="r228c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r228c1-clusterblast" data-name="r228c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r228c1-knownclusterblast" data-name="r228c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r228c1-subclusterblast" data-name="r228c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r228c1-MIBiG-cluster-compare" data-name="r228c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r228c1-pfam-details" data-name="r228c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r228c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-367"></div> <span class="help-tooltip" id="nrps-domain-help-367">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r228c1-details-svg" data-filename="c00228_NODE_19.._r228c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r228c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r228c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-370"></div> <span class="help-tooltip" id="cb-general-help-370">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r228c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r228c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r228c1_1.svg">
        NZ_CP032221 (5367361-5496657): Rhodococcus rhodochrous strain EP4 chromosome (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r228c1_2.svg">
        NZ_CP012949 (5818422-5994901): Streptomyces ambofaciens strain DSM 40697 chro... (5% of genes show similarity), NRPS,T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r228c1_3.svg">
        NZ_CP029190 (2564820-2621797): Streptomyces venezuelae strain ATCC 21782 chro... (5% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r228c1_4.svg">
        NZ_CP045704 (2847124-2952068): Streptomyces rutgersensis strain NBH77 chromos... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r228c1_5.svg">
        NZ_CP012382 (5967338-&gt;6143326): Streptomyces ambofaciens ATCC 23877 chromosom... (4% of genes show similarity), NRPS,T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r228c1_6.svg">
        NZ_AP018316 (3906381-4033519): Dolichospermum compactum NIES-806 (5% of genes show similarity), NRPS,T1PKS,betalactone,hglE-KS,microviridin
      </option>
      
      <option value = "svg/clusterblast_r228c1_7.svg">
        NZ_MPLE01000004 (1684525-1779089): Streptomyces viridifaciens strain DSM 4023... (5% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r228c1_8.svg">
        NZ_CP023698 (1837342-1933027): Streptomyces viridifaciens strain ATCC 11989 c... (6% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r228c1_9.svg">
        NC_014659 (3780515-3880782): Rhodococcus hoagii 103S, complete genome (4% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r228c1_10.svg">
        NZ_CP041647 (2840583-2940822): Rhodococcus hoagii strain WY chromosome, compl... (4% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r228c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r228c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r228c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-371"></div> <span class="help-tooltip" id="cb-known-help-371">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r228c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r228c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r228c1_1.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r228c1_2.svg">
        kutzneride 2
      </option>
      
    </select>
    <button id="knownclusterblast-r228c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r228c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r228c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-372"></div> <span class="help-tooltip" id="cb-sub-help-372">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r228c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-373"></div> <span class="help-tooltip" id="MIBiG-help-373">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r228c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r228c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r228c1"></div>
    <div id="comparison-MIBiG-r228c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001748.1: 0-18705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001748/index.html#r1c1" target="_new">BGC0001748.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>NRP, Polyketide</td>
        
        <td>pseudospumigin A, pseudospumigin B, pseudospumigin C, pseudospumigin D, pseudospumigin E, pseudospumigin F</td>
        
        
        <td>Nostoc sp. CENA543</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:1.00, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:1.00, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>NRP, Polyketide</td>
        
        <td>antalid</td>
        
        
        <td>Polyangium spumosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001341.1: 260464-281186">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001341/index.html#r1c1" target="_new">BGC0001341.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.59</td>
        <td>NRP</td>
        
        <td>valinomycin</td>
        
        
        <td>Rothia nasimurium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.56</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000320.1: 277-35864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000320/index.html#r1c1" target="_new">BGC0000320.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:1.00, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:1.00, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.55</td>
        <td>NRP</td>
        
        <td>cereulide</td>
        
        
        <td>Bacillus cereus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001846.1: 7605326-7645900">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001846/index.html#r1c1" target="_new">BGC0001846.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.53</td>
        <td>Saccharide, NRP</td>
        
        <td>valinomycin, montanastatin</td>
        
        
        <td>Streptomyces sp. CBMAI 2042</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001657.1: 1563-71744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001657/index.html#r1c1" target="_new">BGC0001657.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.50</td>
        <td>NRP</td>
        
        <td>lysocin</td>
        
        
        <td>Lysobacter sp. RH2180-5</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001630.1: 45-36789">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001630/index.html#r1c1" target="_new">BGC0001630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.50</td>
        <td>NRP, Polyketide</td>
        
        <td>kasumigamide</td>
        
        
        <td>uncultured Candidatus Entotheonella sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000297.1: 1263-31426">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000297/index.html#r1c1" target="_new">BGC0000297.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.49</td>
        <td>NRP, Polyketide, Saccharide</td>
        
        <td>aeruginoside 126B, aeruginoside 126A</td>
        
        
        <td>Planktothrix agardhii NIVA-CYA 126</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r228c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001748.1: 0-18705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001748/index.html#r1c1" target="_new">BGC0001748.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP, Polyketide</td>
      <td>pseudospumigin A, pseudospumigin B, pseudospumigin C, pseudospumigin D, pseudospumigin E, pseudospumigin F</td>
      <td>Nostoc sp. CENA543</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:1.00, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP, Polyketide</td>
      <td>antalid</td>
      <td>Polyangium spumosum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001341.1: 260464-281186">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001341/index.html#r1c1" target="_new">BGC0001341.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>valinomycin</td>
      <td>Rothia nasimurium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000320.1: 277-35864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000320/index.html#r1c1" target="_new">BGC0000320.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:1.00, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>cereulide</td>
      <td>Bacillus cereus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001846.1: 7605326-7645900">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001846/index.html#r1c1" target="_new">BGC0001846.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Saccharide, NRP</td>
      <td>valinomycin, montanastatin</td>
      <td>Streptomyces sp. CBMAI 2042</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001657.1: 1563-71744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001657/index.html#r1c1" target="_new">BGC0001657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>lysocin</td>
      <td>Lysobacter sp. RH2180-5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001630.1: 45-36789">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001630/index.html#r1c1" target="_new">BGC0001630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>kasumigamide</td>
      <td>uncultured Candidatus Entotheonella sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000297.1: 1263-31426">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000297/index.html#r1c1" target="_new">BGC0000297.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:1.00, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide, Saccharide</td>
      <td>aeruginoside 126B, aeruginoside 126A</td>
      <td>Planktothrix agardhii NIVA-CYA 126</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r228c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-374"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-374">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r228c1-pfam-details-svg" data-filename="c00228_NODE_19.._r228c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r228c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r228c1-nrps_pks_products" data-name="r228c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r228c1-nrps_pks" data-name="r228c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r228c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-368"></div> <span class="help-tooltip" id="nrps-structures-help-368">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 8567:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r228c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r228c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r228c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-369"></div> <span class="help-tooltip" id="nrps-monomers-help-369">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_4245</span></strong>: X
      <div class="collapser collapser-target-cds0_4245 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (465..855)</strong>: X
          <div class="collapser collapser-target-cds0_4245 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds0_4245-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVFSVGpfyK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>d-lyserg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds4786_8566</span></strong>: X
      <div class="collapser collapser-target-cds4786_8566 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (76..519)</strong>: X
          <div class="collapser collapser-target-cds4786_8566 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds4786_8566-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">vtyWVGAsGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tcl</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r238c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00238_NODE_56.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-496"></div> <span class="help-tooltip" id="region_svg-help-496">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00238_NODE_56...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r238c1-svg" data-filename="c00238_NODE_56.._r238c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 35,321 - 45,695 nt. (total: 10,375 nt)
        
        <a class="cluster-rules-header" id="r238c1-rules-header" href="#r238c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r238c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r238c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-497"></div> <span class="help-tooltip" id="focus-panel-help-497">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r238c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r238c1-clusterblast" data-name="r238c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r238c1-knownclusterblast" data-name="r238c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r238c1-subclusterblast" data-name="r238c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r238c1-MIBiG-cluster-compare" data-name="r238c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r238c1-pfam-details" data-name="r238c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r238c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-375"></div> <span class="help-tooltip" id="cb-general-help-375">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r238c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r238c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r238c1_1.svg">
        NZ_CP020700 (307-229249): Streptomyces tsukubensis strain NRRL 18488 chromoso... (7% of genes show similarity), NRPS,T1PKS,butyrolactone,ectoine,other
      </option>
      
      <option value = "svg/clusterblast_r238c1_2.svg">
        NZ_CP042266 (12581-225174): Streptomyces qinzhouensis strain SSL-25 chromosom... (7% of genes show similarity), NRPS,T1PKS,T3PKS,ectoine,melanin,other,phenazine
      </option>
      
      <option value = "svg/clusterblast_r238c1_3.svg">
        NZ_CP042266 (8021364-8134460): Streptomyces qinzhouensis strain SSL-25 chromo... (10% of genes show similarity), NRPS,T1PKS,T3PKS,ectoine,other,phenazine
      </option>
      
      <option value = "svg/clusterblast_r238c1_4.svg">
        NZ_SLWS01000011 (20774-31149): Actinocrispum wychmicini strain DSM 45934 Ga03... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r238c1_5.svg">
        NZ_CP023701 (73996-180562): Streptomyces subrutilus strain ATCC 27467 chromos... (8% of genes show similarity), NRPS,butyrolactone,ectoine,terpene
      </option>
      
      <option value = "svg/clusterblast_r238c1_6.svg">
        NZ_CP020039 (73131-126275): Streptomyces sp. 3211 isolate 3 chromosome, compl... (16% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r238c1_7.svg">
        NZ_CP020555 (87136-140489): Streptomyces sp. Sge12 chromosome, complete genome (16% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r238c1_8.svg">
        NZ_CDME01000008 (562787-573162): Kibdelosporangium sp. MJ126-NF4, whole genom... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r238c1_9.svg">
        NZ_CP012752 (9426407-9436782): Kibdelosporangium phytohabitans strain KLBMP11... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r238c1_10.svg">
        NZ_FWXV01000003 (468359-478734): Kibdelosporangium aridum strain DSM 43828, w... (66% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r238c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r238c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r238c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-376"></div> <span class="help-tooltip" id="cb-known-help-376">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r238c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r238c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r238c1_1.svg">
        kosinostatin
      </option>
      
      <option value = "svg/knownclusterblast_r238c1_2.svg">
        showdomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r238c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r238c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r238c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-377"></div> <span class="help-tooltip" id="cb-sub-help-377">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r238c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-378"></div> <span class="help-tooltip" id="MIBiG-help-378">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r238c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r238c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r238c1"></div>
    <div id="comparison-MIBiG-r238c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.47, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001896.1: 7184700-7193029">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001896/index.html#r1c1" target="_new">BGC0001896.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>carbazomycin B</td>
        
        
        <td>Streptomyces luteoverticillatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r238c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.47, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001896.1: 7184700-7193029">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001896/index.html#r1c1" target="_new">BGC0001896.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>carbazomycin B</td>
      <td>Streptomyces luteoverticillatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r238c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-379"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-379">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r238c1-pfam-details-svg" data-filename="c00238_NODE_56.._r238c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r238c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r240c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00240_NODE_10.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-498"></div> <span class="help-tooltip" id="region_svg-help-498">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00240_NODE_10...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r240c1-svg" data-filename="c00240_NODE_10.._r240c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1,040 - 32,675 nt. (total: 31,636 nt)
        
        <a class="cluster-rules-header" id="r240c1-rules-header" href="#r240c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r240c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r240c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-499"></div> <span class="help-tooltip" id="focus-panel-help-499">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r240c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r240c1-nrps_pks" data-name="r240c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r240c1-clusterblast" data-name="r240c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r240c1-knownclusterblast" data-name="r240c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r240c1-subclusterblast" data-name="r240c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r240c1-MIBiG-cluster-compare" data-name="r240c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r240c1-pfam-details" data-name="r240c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r240c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-380"></div> <span class="help-tooltip" id="nrps-domain-help-380">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r240c1-details-svg" data-filename="c00240_NODE_10.._r240c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r240c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r240c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-383"></div> <span class="help-tooltip" id="cb-general-help-383">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r240c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r240c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r240c1_1.svg">
        NZ_CP048641 (1021665-1243764): Streptomyces aureoverticillatus strain HN6 chr... (29% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_2.svg">
        NZ_CP040916 (7733849-7879172): Streptomyces spectabilis strain NRRL 2792 chro... (52% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_3.svg">
        NZ_CP023690 (8047123-8193590): Streptomyces spectabilis strain ATCC 27465 chr... (52% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_4.svg">
        NZ_CP030771 (8015578-8162299): Streptomyces sp. YIM 121038 chromosome, comple... (52% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_5.svg">
        NZ_CP021748 (8263860-8399611): Streptomyces alboflavus strain MDJK44 chromoso... (47% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_6.svg">
        NZ_CP033581 (898663-1139331): Streptomyces sp. ADI95-16 chromosome, complete ... (25% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_7.svg">
        NZ_LT607413 (5451020-5785033): Micromonospora echinospora strain DSM 43816 ch... (15% of genes show similarity), NRPS,T1PKS,nucleoside
      </option>
      
      <option value = "svg/clusterblast_r240c1_8.svg">
        NZ_CP029189 (1020283-1209544): Streptomyces venezuelae strain ATCC 21018 chro... (36% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_9.svg">
        NZ_LWLD01000001 (218189-407586): Streptomyces sp. CB03578 scaffold1, whole ge... (38% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r240c1_10.svg">
        NZ_CP011664 (1057905-1247427): Streptomyces sp. Mg1 chromosome 1, complete se... (36% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r240c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r240c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r240c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-384"></div> <span class="help-tooltip" id="cb-known-help-384">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r240c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r240c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_1.svg">
        ECO-02301
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_2.svg">
        brasilinolide A / brasilinolide B / brasilinolide C
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_3.svg">
        mediomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_4.svg">
        mediomiycin A
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_5.svg">
        funisamine
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_6.svg">
        oligomycin
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_7.svg">
        lydicamycin
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_8.svg">
        ibomycin
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_9.svg">
        primycin
      </option>
      
      <option value = "svg/knownclusterblast_r240c1_10.svg">
        nystatin-like Pseudonocardia polyene
      </option>
      
    </select>
    <button id="knownclusterblast-r240c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r240c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r240c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-385"></div> <span class="help-tooltip" id="cb-sub-help-385">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r240c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-386"></div> <span class="help-tooltip" id="MIBiG-help-386">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r240c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r240c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r240c1"></div>
    <div id="comparison-MIBiG-r240c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000484.1: 184-6641">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000484/index.html#r1c1" target="_new">BGC0000484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>glycocin F</td>
        
        
        <td>Lactobacillus plantarum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
        
        
        <td>Nannocystis pusilla</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001793.1: 538-21374">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001793/index.html#r1c1" target="_new">BGC0001793.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>swainsonine</td>
        
        
        <td>Chaetothyriaceae sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>swainsonine</td>
        
        
        <td>Alternaria oxytropis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043.1: 0-13097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>curacin A</td>
        
        
        <td>Moorea producens 19L</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000057.1: 122622-149174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000057/index.html#r1c1" target="_new">BGC0000057.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>F9775A, F9775B, orsellinic acid</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r240c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000484.1: 184-6641">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000484/index.html#r1c1" target="_new">BGC0000484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>glycocin F</td>
      <td>Lactobacillus plantarum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
      <td>Nannocystis pusilla</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001793.1: 538-21374">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001793/index.html#r1c1" target="_new">BGC0001793.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>swainsonine</td>
      <td>Chaetothyriaceae sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>swainsonine</td>
      <td>Alternaria oxytropis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043.1: 0-13097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>curacin A</td>
      <td>Moorea producens 19L</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000057.1: 122622-149174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000057/index.html#r1c1" target="_new">BGC0000057.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>F9775A, F9775B, orsellinic acid</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r240c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-387"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-387">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r240c1-pfam-details-svg" data-filename="c00240_NODE_10.._r240c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r240c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r240c1-nrps_pks_products" data-name="r240c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r240c1-nrps_pks" data-name="r240c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r240c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-381"></div> <span class="help-tooltip" id="nrps-structures-help-381">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 1039 - 32675:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r240c1-1" class="smiles-canvas" data-smiles="C(C)C(O)CC(O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-ohmal - ohmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ohmal,ohmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ohmal*,ohmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r240c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(O)CC(O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r240c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-382"></div> <span class="help-tooltip" id="nrps-monomers-help-382">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds21039_32673</span></strong>: mmal - mal
      <div class="collapser collapser-target-cds21039_32673 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (713..1005)</strong>: mmal
          <div class="collapser collapser-target-cds21039_32673 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds21039_32673-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds21039_32673-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 214.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 195.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 171.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 157.7</dt>

  <dd></dd><dt>Propionyl-CoA: 119.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 112.8</dt>

  <dd></dd><dt>CHC-CoA: 108.2</dt>

  <dd></dd><dt>Malonyl-CoA: 95.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 93.7</dt>

  <dd></dd><dt>Acetyl-CoA: 73.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 59.0</dt>

  <dd></dd><dt>fatty_acid: 43.4</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 37.4</dt>

  <dd></dd><dt>inactive: 16.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2265..2556)</strong>: mal
          <div class="collapser collapser-target-cds21039_32673 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds21039_32673-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds21039_32673-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 202.6</dt>

  <dd></dd><dt>inactive: 112.6</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 39.6</dt>

  <dd></dd><dt>Propionyl-CoA: 22.0</dt>

  <dd></dd><dt>Acetyl-CoA: 20.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 17.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 8.6</dt>

  <dd></dd><dt>fatty_acid: 7.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -4.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>